---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 200 word limit for abstract -->
<!-- 6 page limit w/o incurring costs -->

```{r, include=FALSE}
# set wd
setwd('..')

# load .rda files
load('results/results.rda')

# load packages
source('R/analysis/00-initialization.R')

# set options
options(error=NULL)
```

## Introduction
Biodiversity is under threat. Over the last few decades, many species have gone extinct [@r270; @r272]. Today, thousands of species are listed as endangered or near threatened in the International Union for Conservation of Nature (IUCN) red list (\url{www.iucnredlist.org}). In the future, species are expected to face even more threats [eg. climate change; @r236; @r243]. In response to this crisis, the overarching aim of conservation is to maximize the long-term persistence of biodiversity [@r255; @r67].

The act of setting aside areas for preservation is the hallmark of modern conservation [@r440]. Protected areas buffer species from anthropogenic impacts and provide places for biodiversity processes to persist (eg. nutrient cycling and water purification). Additionally, protected areas set the stage for direct management actions (eg. habitat rehabilitation, translocation, and invasive species management). Since the resources available for conservation are limited, the plans for protected areas (prioritizations) are often generated using spatially explicit biological data (eg. species distribution data) and decision support tools [eg. $\texttt{MARXAN}$; @r22; $\texttt{Zonation}$; @r429] that identify cost-effective solutions. 

Protected areas must preserve intra-specific genetic variation to maximize species recovery [@r67; @r3; @r61]. This variation can be classified as either adaptive or neutral [reviewed in @r????]. Adaptive genetic variation is associated with loci currently under selection, whereas neutral genetic variation is associated with loci that are not. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral variation reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can secure different genetic lineages and avoid the adverse effects of low genetic diversity [eg. inbreeding depression; @r67]. However, genetic data requires considerable investment to obtain and analyze, and as a consequence, conservation planners do not typically use genetic data to inform prioritizations [@r61]. Thus many prioritizations may fail to capture species' intra-specific genetic variation, and ultimately fail to deliver conservation outcomes.

Recently, conservation scientists have begun investigating the use of surrogates for genetic data to generate more effective prioritizations without the overheads associated with genetic data. Since adaptive genetic variation is driven by selection pressures, environmental variables have been explored as potential surrogates for genetic variation [eg. @r247; @r2]. On the other hand, neutral variation arises from a reduction in gene flow, and so variables that predict reduction in gene flow between areas have been explored. For instance, the isolation by distance theory [IBD; @r259] predicts that populations further apart will have lower gene flow, and in turn have greater neutral variation among them. The use of distance-based surrogates has been found to improve prioritizations in an island network [@r200]. However the effectiveness of distance-based surrogates remains untested in systems where connectivity is complicated by additional factors (eg. elevation in terrestrial systems). Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates needs to be verified. Otherwise, conservation planners may waste precious resources on securing places that do little to benefit biodiversity.

Our overarching aim is to investigate the effectiveness of environmental and geographic surrogates for adaptive neutral genetic variation in the context of conservation planning. First, we aim to test the effectiveness of these surrogates under the simplest of conditions by using a set of single-species prioritizations. Second, we aim to test the effectiveness of these surrogates in a more realistic multi-species context, and see how they compare to the single-species prioritizations. Third, we aim to understand the relationship between the proportion of surrogate variation secured in a multi-species prioritization and the proportion of genetic variation secured in a prioritization. If these surrogates can be shown to markedly improve the proportion of genetic variation secured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
To address the aims of the study, we utilized species distribution and genomic data collected by the IntraBioDiv project [@r451; see @r452 for methodological details]. This dataset was well suited for this study, because it provides spatially explicit genomic data for a multitude of species across a broad geographic area and over large environmental gradients. As part of this project, data for 27 alpine plant species were collected across the European Alps according to a grid (22.3 km $\times$ 25 km; Figure S1). Briefly, in every second cell, plant samples were collected from three individuals for each species present inside the cell. Cells that did not contain any plant samples were omitted from our analysis. Samples were genotyped using amplified fragment length polymorphisms [AFLP; @r453]. Matrices denoting the presence/absence of polymorphisms at loci were constructed independently for each species. Thus for each second grid cell, for each of the species in the grid cell, the dataset contains information describing the genomic variation of individuals inside the grid cell. We used these grid cells as planning units to develop prioritizations to investigate the effectiveness of surrogates for the (AFLP) genetic variation. Most of our analyses described hereafter were performed using the \texttt{R} software package [version `r paste0(version$major, '.', version$minor)`; @r192].

### Surrogate data
We explored the effectiveness of climatic and geographic surrogates. To describe variation in the location of each grid cell, we projected the grid into an equi-distant coordinate system ($\texttt{ESRI:102031}$) and used the centroid of each cell. To describe the climatic variation among each cell, we obtained 21 bioclimatic variables [approx. 1 km resolution; @r33]. These layers and the grid cells were projected into an equal-area coordinate system ($\texttt{ESRI:102014}$). The bioclimatic variables were clipped to the extent of the grid and subject to a principle components analysis (PCA; Table S1). The first two principle components (PCs) cumulatively explained `r round(pca.DF[[4]][2],1)` % of the variation and were used to generate two new layers. We calculated the average value for each PC layer in each grid cell, and used these values to characterize the main themes of climatic variation among the grid cells (Figure S2).

### Outlier locus detection
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first identified which of the sampled loci for each species were under selection. Specifically, we used outlier locus detection methods [evaluated in @r454] to avoid circularity issues. The basic premise underpinning these analyses is that neutral loci are expected to exhibit a specific level of variation among populations ($F_{st}$), and loci that deviate from this expectation should be under selection. In an earlier study, Bothwell _et al._ [-@r455] applied such outlier detection methods to one of the species in this dataset (_Getiana nivalis_). Here, we adopted a similar methodology and applied it to each of the 27 species in the dataset.

First, we grouped individuals into genetic lineages for each species. The purpose of this step was to avoid misclassifying loci as adaptive. We used discriminant analyses of principle components [DAPC; @r467] to assign individuals to populations [implemented in $\texttt{adegenet}$ R package; @r468]. Briefly, for each species, we transformed the binary loci data using a principle components analysis (PCA) and obtained the principle components required to explain at least `r adegenet.params.LST[[MODE]]$perc.pca` % variation. We subject these components to multiple \textit{k}-means clustering models with varying numbers of assumed populations (\textit{k} = 1--`r adegenet.params.LST[[MODE]]$max.n.clust`; `r adegenet.params.LST[[MODE]]$n.iter` iterations; `r adegenet.params.LST[[MODE]]$n.start` random starts). We then  applied Ward's clustering method to the Bayesian Information Criterion (BIC) statistics associated with each model to partition the models into two groups, and selected the model before the first group switch (as recommended in the \texttt{adegenet} manual; Figure S4). The populations memberships associated with these models were used for subsequent analysis (Figure S5).

We identified loci under selection for each species (Table S2). For each species, we ran $\texttt{Bayescan}$ [version 2.1; @r456] using the population memberships identified in the previous analysis (1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations). We omitted loci where the global frequency of the minor allele was $\geq 0.05$ to avoid false-positives (following guidelines in the $\texttt{Bayescan}$ manual). To ensure convergence, we ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate [FDR; $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species (Figures S6--S32). For each species, for each type of genetic variation detected, we used Gower distances [@r464] to express the genomic differences between individuals [using the $\texttt{cluster}$ R package to accommodate sparsity; @r457]. These distances were subject to multiple non-metric multi-dimensional scaling analyses [NMDS; implemented in the $\texttt{vegan}$ R package; @r458] using the minimum number of dimensions required to achieve an adequate stress value  ($\leq$ `r mds.params.LST[[MODE]]$max.stress`; `r mds.params.LST[[MODE]]$trymax` random starts for all models; Table S3). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the samples in each grid cell to characterize their the genetic variation of individuals in the cells.

### Prioritizations
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve a proportion of the species' range as well as a proportion of the species' variation using an attribute space [_sensu_ @r465]. We treated the genomic and surrogate variables as distinct attribute spaces. We set the demand points for each species and its attribute space using the values associated with the planning units. Although Hanson _et al._ [-@r465] recommend against using the same coordinates for demand points and planning units for real-word conservation planning exercises, we did this to provide a more robust test and reduce uncertainty in the analysis. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463]. 

To understand how prioritizations generated using surrogate-based targets might perform under typical conservation planning scenarios, we generated a set of single-species and multi-species prioritizations. All of these prioritizations used `r (rapr.params.LST[[MODE]]$amount.target*100)` % amount-based targets to ensure an adequate proportion of habitat was secured for each species [@r197; @r255; @r471]. We generated a set of single-species prioritizations for each species: `r rapr.params.LST[[MODE]]$single.species.amount.replicates` prioritizations using only the amount-based targets, a single prioritization using the amount-based target and `r (rapr.params.LST[[MODE]]$surrogate.target*100)` % surrogate-based targets, and a single prioritization using the amount-based target and `r (rapr.params.LST[[MODE]]$genetic.target*100)` % genetic-based targets. We also generated three multi-species prioritizations using the three unique combinations of targets used to generate the single-species prioritizations. Note that we obtained multiple solutions for the amount-based single-species prioritizations because, unlike the other prioritizations, these targets yielded many optimal solutions with varying levels effectiveness.

To understand the relationship between the surrogate-based targets and the proportion of genetic variation secured, we generated two sets of multi-species prioritizations with increasing surrogate-based targets between `r (min(rapr.params.LST[[MODE]]$correlation.surrogate.targets)*100)` % and `r (max(rapr.params.LST[[MODE]]$correlation.surrogate.targets)*100)` % in increments of `r diff(rapr.params.LST[[MODE]]$correlation.surrogate.targets[1:2])*100` %. The first set of prioritizations contained prioritizations with varying environmental surrogate-based targets with no specified geographic targets. The second contained prioritizations with varying geographic surrogate-based targets and no environmental targets. None of these prioritizations used amount-based targets. 

## Statistical analysis
We tested for differences in the effectiveness of prioritizations representing typical scenarios using generalized linear mixed-effects models [GLMMs; using the $\texttt{lme4}$ R package; @r469]. We fit two full models to the proportion of genomic variation secured in the single- and multi-species prioritizations using a logit link. Both models contained a predictor variable indicating if the prioritization was generated using amount, surrogate, or genetic targets, a second predictor variable indicating if the genetic variation was adaptive or neutral, and the interaction between these variables. Additionally, both models contained an intercept-only random effect to ensure that comparisons were made within species. Each full model was subject to a step-wise term deletion routine using likelihood ratio tests (LRT) to assess term significance (terms were retained when $P < 0.05$). Finally, we conducted a post-hoc analysis using Tukey constrasts with a Bonferroni correction [using the $\texttt{multcomp}$ R package; @r470].

We investigated the relationship between surrogate-based targets and the genetic variation secured in a prioritization using nonlinear mixed-effects models (NLMMs; using the $\texttt{lme4}$ R package). Two models were fit to the adaptive and neutral genetic variation secured using a negative exponential formulation to estimate the rate of the accumulation of genetic variation ($y = 0 + (0 - 1) \times exp(-r \times x)$; $r$ is the accumulate rate). Both models contained intercept and slope random effects to accommodate variation between species.

## Results
<!-- Single species prioritizations -->
Surrogates-targets improved the performance of single-species prioritizations (Figure 1). There were significant differences among the performance of prioritizations depending on the targets used to generate them and if they were assessed according to adaptive or neutral genetic variation (${\chi^{2}}_{`r single.spp.AOV1[['Df']][1]`,`r single.spp.AOV1[['Df']][2]`} = `r round(single.spp.AOV1[['Chisq']][2],2)`$, $P `r pretty.pval(single.spp.AOV1[['Pr(>Chisq)']][2])`$). Specifically, the amount-based prioritization captured much less adaptive and neutral genetic variation than either the surrogate-based ($Z = `r round(single.spp.MCP$test$tstat[['Surrogate.Adaptive variation - Amount.Adaptive variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Surrogate.Adaptive variation - Amount.Adaptive variation']])`$; $Z = `r round(single.spp.MCP$test$tstat[['Surrogate.Neutral variation - Amount.Neutral variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Surrogate.Neutral variation - Amount.Neutral variation']])`$) or genetic-based ($Z = `r round(single.spp.MCP$test$tstat[['Genetic.Adaptive variation - Amount.Adaptive variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Genetic.Adaptive variation - Amount.Adaptive variation']])`$; $Z = `r round(single.spp.MCP$test$tstat[['Genetic.Neutral variation - Amount.Neutral variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Genetic.Neutral variation - Amount.Neutral variation']])`$) prioritizations. No differences were detected in the proportion of adaptive ($Z = `r round(single.spp.MCP$test$tstat[['Surrogate.Adaptive variation - Genetic.Adaptive variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Surrogate.Adaptive variation - Genetic.Adaptive variation']])`$) or neutral genetic variation ($Z = `r round(single.spp.MCP$test$tstat[['Surrogate.Neutral variation - Genetic.Neutral variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Surrogate.Neutral variation - Genetic.Neutral variation']])`$) secured by the the surrogate-based and genetic-based prioritizations. However, these patterns were not reflected in the multi-species prioritizations.

<!-- Multi-species prioritizations -->
Surrogate-targets were not found to improve the performance of multi-species prioritizations (Figure 2; Figure 3). No significant differences in the effectiveness of prioritizations were found depending on whether they were generated using different targets (${\chi^{2}}_{`r multi.spp.AOV3[['Df']][1]`,`r multi.spp.AOV3[['Df']][2]`} = `r round(multi.spp.AOV3[['Chisq']][2],2)`$; $P `r pretty.pval(multi.spp.AOV3[['Pr(>Chisq)']][2])`$), if they were assessed using adaptive or neutral genetic variation (${\chi^{2}}_{`r multi.spp.AOV2[['Df']][1]`,`r multi.spp.AOV2[['Df']][2]`} = `r round(multi.spp.AOV2[['Chisq']][2],2)`$; $P `r pretty.pval(multi.spp.AOV2[['Pr(>Chisq)']][2])`$), nor the interaction between these variables (${\chi^{2}}_{`r multi.spp.AOV1[['Df']][1]`,`r multi.spp.AOV1[['Df']][2]`} = `r round(multi.spp.AOV1[['Chisq']][2],2)`$; $P  `r pretty.pval(multi.spp.AOV1[['Pr(>Chisq)']][2])`$).

<!-- Correlation analysis -->
The surrogate variables showed a greater utility for adaptive genetic variation than neutral variation (Figure 4). The accumulation rate ($r$) was `r round(summary(env.correlation.NLMM)$coefficients[1],2)` $\pm$ `r round(summary(env.correlation.NLMM)$coefficients[2],2)` S.E. for the proportion of adaptive variation captured in a prioritization using environmental surrogates. On the other hand, the rate ($r$) was `r round(summary(geo.correlation.NLMM)$coefficients[1],2)` $\pm$ `r round(summary(geo.correlation.NLMM)$coefficients[2],2)` S.E. for neutral variation using geographic surrogates. Thus minor increases in the target for either surrogate results in major increases in the benefit of the prioritization. However, the coefficient was much larger for the adaptative genetic variation so high targets are not required to acheive substantial gains in adaptive genetic variaition.

## Discussion
### Introduction
* aim
	+ we aimed to investigate the benefits of using surrogates for genetic variation in conservation planning.
* innovation
	+ first to test the effectiveness of environmental and geographic surrogates in a conservation planning context
* key findings
	+ surrogate variables are beneficial in single-species contexts.
	+ but not in multi-species contexts when a comprehensive set of species is used.
* consequences
	+ highlights the benefit of using a comprehensive set of species
	+ supports literature on using comprehensive set of species

### Single + multi-species analysis

### Correlation analysis

### Limitations/Future directions
* spatial/surrogate data
	+ isolation by resistance: least-cost paths and circuit theory distances are becoming more population for modeling connectivity, and may offer better surrogates than linear geographic distances. However, the scaling for these variables requires an understanding of the often non-linear relationship, which is not available to conservation planners
	+ cost data: here we used no cost data to provide a rigorous test for the benefits of surrogates in conservation planning. however in real conservation planning situations, prioritizations are largely driven by cost. future studies should look at including cost data to see if surrogates provide any benefit when including realistic cost data.
	
* genetic data
	+ AFLP data: next gen data would be better however the aim of a prioritization is to capture the main gradients of genetic variation--not all of the variation.
	+ adegenet: whilst other software packages are more popular (eg. Structure), we used adegenet because information theory is more reliable than an ad-hoc second derivative process for comparing models. Additionally, it is computationally infeasible to analyze run 27 species in Structure in a suitable period of time.

### Concluding remarks
* take-home messages
	+ surrogates are effective for single-species prioritizations in conservation planning
	+ however, surrogates don't provide any noticeable benefits when using a comprehensive set of species in a multi-species prioritization
	+ conservation planners are encouraged to use comprehensive sets of species

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References

