---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
setwd('..')

# load .rda file
session::restore.session('results/results.rda')

# set options
options(error=NULL)
```


<!--  title page -->
Short running title: Genetic surrogates in conservation \newline
Word count: XXXXX \newline
Number of references: XXXX \newline
Number of figures, table and text boxes: 4 \newline

\newpage

<!-- add line numbers -->
\begin{linenumbers}

## Introduction
<!--  main text -->
The act of setting aside areas for preservation spearheads modern conservation efforts [@r440]. They buffer species from anthropogenic impacts and provide places for populations of species to persist. Intra-specific genetic variation underpins species persistence, and there has been increasing interest in understanding how to design protected areas networks in a way that maximizes this variation [@r67; @r3; @r61]. Broadly speaking, genetic variation can be classified as either adaptive or neutral [reviewed in @r????]. Adaptive genetic variation is associated with loci currently under selection. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral genetic variation is associated with loci that are not under natural selection. Neutral variation often reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can secure different genetic lineages and avoid the adverse effects of low genetic diversity [eg. inbreeding depression and reduced capacity to respond to environmental change; @r67]. However, genetic data requires considerable investment to obtain and analyze. As a consequence, conservation planners do not typically use genetic data to inform prioritizations [@r61]. Thus many prioritizations may fail to capture species' intra-specific genetic variation, and ultimately fail to deliver long-term conservation outcomes.

Recently, conservation scientists have been investigating surrogates for genetic variation to generate better prioritizations without the overheads associated with genetic data. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been used to guide reserve selection [eg. habitat type and local climate; @r247; @r2]. These variables are often used to detect adaptive variation [eg. @r475; @r455]. However, their effectiveness as surrogates in conservation planning remains unverified. On the other hand, neutral genetic variation arises from a reduction in gene flow between individuals. Thus surrogates for neutral variation are based on variables that predict the level of connectivity between planning units. For instance, the isolation by distance theory [IBD; @259] predicts populations further apart will have less connectivity,  result in reduced gene flow, and in turn share less neutral genetic variation. Geographic distance-based surrogates have been found to improve prioritizations in an island network [@200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain; eg. @r474]--the systems commonly featured in real-world conservation planning exercises. Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates needs to be verified. Otherwise, conservation planners may waste precious resources on places that do little to benefit biodiversity.. 

Here we investigate the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation in the context of conservation planning. First, we examine the relationship between surrogates and gene variation, and verify that these variables can be used as surrogates. Second, we determine if the inclusion of surrogate-based targets in a variety of conservation planning scenarios with increasingly levels of realism can result in prioritizations that secure more genetic variation. If these surrogates can be shown to improve the intra-specific genetic variation captured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
We used species distribution and genomic data collected by the IntraBioDiv project in the European Alps [@r451; see @r452 and @r478 for methodological details]. This dataset was particularly well suited for this study, because it provides spatially explicit genomic data for multiple species with vastly different demographic characteristics and evolutionary histories across a broad geographic area with large environmental gradients. As part of this project, data was collected for 27 alpine plant species using a 20' longitude by 21' latitude grid (approx. 21' 22.3 km $\times$ 25 km; Figure S1). They visited every second grid cell, and if a species was detected in a grid cell, samples were collected from three individuals. They genotyped samples using amplified fragment length polymorphisms [AFLP; @r453], and constructed matrices denoting the presence/absence of polymorphisms at loci for each species. Thus for each second grid cell, for each of the species present in the grid cell, the dataset contains information describing the genomic variation of individuals in that grid cell. We used these grid cells as planning units to develop prioritizations investigating the effectiveness of surrogates for the (AFLP) genetic variation. Note that we omitted grid cells that did not contain any individuals from our analysis. The majority of our analyses described hereafter were performed using the \texttt{R} software package [version `r paste0(version$major, '.', version$minor)`; @r192].

### Landscape data
We aimed to investigate the effectiveness of climatic and geographic surrogates. To describe the location of each grid cell, we projected the grid into an equi-distant coordinate system ($\texttt{ESRI:102031}$), calculated their centroids, and extracted its two-dimensional coordinates. To describe the climatic variation among each cell, we obtained 19 bioclimatic variables [approx. 1 km resolution; @r33]. These layers and the grid cells were projected into an equal-area coordinate system ($\texttt{ESRI:102014}$). The bioclimatic variables were clipped to the extent of the grid and subject to a principal components analysis (PCA; using ArcMap version 10.2.2; Table S1). The first two principal components (PCs) cumulatively explained `r round(pca.DF[[4]][2],1)` % of the variation and were used to generate two new layers. We calculated the average value for each PC layer in each grid cell, and used these values to characterize the main themes of climatic variation among the grid cells (Figure S2). To examine if the influence of opportunity cost, we calculated the total human population density inside each planning unit (obtained at 1 km resolution from the Global Rural-Urban Mapping Project; \texttt{GRUMP V1}; @r472).

### Outlier locus detection
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first need to identify which of the sampled loci--if any--were under selection for each species. We used outlier locus detection methods [evaluated in @r454] to achieve this. The basic premise underpinning these analyses is that neutral loci are expected to exhibit a specific level of variation among populations ($F_{st}$), and loci that deviate from this expectation should be under selection. The advantage of these methods--in contrast with environmental association analyses [eg. r@475]--is that they do not use environmental data to identify loci under selection, which would have biased our analysis. In an earlier study, Bothwell _et al._ [-@r455] applied such outlier detection methods to one of the species in this dataset (_Gentiana nivalis_). Here, we adopted a similar methodology and applied it to each of the 27 species in the dataset.

First, we grouped individuals into genetic lineages for each species. The purpose of this step was to avoid misclassifying loci as adaptive. For each species, we used non-metric multi-dimensional scaling [NMDS; implemented in the $\texttt{vegan R}$ package] to transform the binary loci data into continuous variables (Table S2). We used Gower distances [@r464] to express the genomic differences between individuals [using the $\texttt{cluster R}$ package to accommodate sparsity; @r457]. We ran successive scaling analyses with increasing dimensionality until ordinations were obtained with an adequate stress value ($\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for all models). We then fit a suite of Gaussian mixture to the ordinated values to estimate the probability that individuals belonged to different populations [using the default priors and covariance structures implemented in the $\texttt{mclust} R$ package; @r477; @r476]. For each parametrization, we fit a set of models assuming varying numbers of total populations (\textit{G}= 1--`r max(mclust.params.LST[[MODE]]$G)`). We identified the most parsimonious models (lowest Bayesian Information Criterion $BIC$ statistic; Figures S4), and used these to group individuals into populations (Figure S5).

We identified loci under selection (Table S3). For each species, we ran $\texttt{Bayescan}$ [version 2.1; @r456] using the population memberships identified in the previous analysis (1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations). To avoid false-positive, we omitted loci where the global frequency of the minor allele was $\geq 0.05$ (following guidelines in the $\texttt{Bayescan}$ manual). We ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate to ensure convergence [FDR $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species (Figures S6--S32). For each species, we partitioned adaptive and neutral loci into separate datasets, and reran the nonmetric multidimensional scaling analysis (NMDS) to derive continuous variables that described the main gradients of adaptive and neutral genetic variation separately (Table S2). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the individuals in each grid cell to summarize the typical genetic characteristics of individuals in the cells.

### Prioritizations
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve a proportion of the species' range as well as a proportion of the species' variation using an attribute space [_sensu_ @r465]. We treated the genomic and surrogate variables as distinct attribute spaces. We set the demand points for each species and its attribute space using the values associated with the planning units. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

To understand the relationship between the surrogate-based targets used to generate a prioritizations and the amount of genetic variation secured, we generated a collection of single-species prioritizations. For each species, we generated two sets of prioritizations using increasing environmental and geographic-based targets (respectively). Prioritizations were generated using surrogate-based targets ranging from `r (min(rapr.params.LST[[MODE]]$correlation.surrogate.targets)*100)` % and `r (max(rapr.params.LST[[MODE]]$correlation.surrogate.targets)*100)` % in increments of `r diff(rapr.params.LST[[MODE]]$correlation.surrogate.targets[1:2])*100` %. None of these prioritizations contained any amount-based targets. 

To determine if the use of surrogate-based targets can actually improve prioritizations in real-world conservation planning scenarios, we used three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritizations for each species. Most large-scale prioritizations involve a comprehensive set of species, and in turn, may secure a greater proportion of intra-specific genetic variation. Our second scenario involved generating multi-species prioritizations. Finally, most large-scale prioritizations also explicitly consider opportunity cost to ensure prioritizations are cost-effective. Our third scenario involved generating multi-species prioritizations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritizations. We generated prioritizations using only amount-based, both amount- and surrogate-based targets, and both amount- and genetic-based targets. The purely amount-based prioritizations represented those typically generated in conservation planning exercises and serve as our baseline. We used `r (rapr.params.LST[[MODE]]$amount.target*100)` % amount-based targets in all prioritizations to ensure an adequate proportion of habitat was secured for each species [@rXXX]. We set surrogate- and genetic-based targets as `r (rapr.params.LST[[MODE]]$surrogate.target*100)` % to ensure that most of the repsective variation was secured for each species. To accomodate variation in the high number of optimal solutions for the single-species amount-based prioritizations, we generated `r (rapr.params.LST[[MODE]]$amount.replicates)` replicate prioritizations. Since all other contexts were likely to be associated with a single optimal solution, we used only a single replicate.

## Statistical analysis
We investigated the effectiveness of prioritizations generated using surrogate-based targets using generalized linear mixed-effects models [GLMMs; using the $\texttt{lme4}$ R package; @r469]. All models were fit to the proportion of genetic variation that prioritizations secured using a logit link. To understand the relationship between the surrogate-based target used to generate a prioritization and how much genetic variation the resulting prioritization secured, we fit a full model containing a continuous quadratic predictor variable for the surrogate target used to generate the prioritization, a categorical variable indicating the type of genetic variation, and a variable for the interaction between the previous variables. This model contained random effects for both the intercept and the predictor variables to accommodate variation between species. To determine if surrogate-based targets improved the effectiveness of prioritizations under different scenarios, we fit a full model containing categorical variables indicating the scenario used to generate the prioritization, the combination of targets used to generate the prioritizations, and the type of genetic variation measured. This model contained intercept-only random effects. Both models were subject to step-wise term deletion routines using likelihood ratio tests (LRT) to assess significance (terms were retained when $P < 0.05$). Finally, we conducted a post-hoc analysis using Tukey contrasts with a Bonferroni correction [using the $\texttt{multcomp}$ R package; @r470].

## Results
<!-- Surrogacy analysis -->
Prioritizations generated using greater environmental and geographic surrogate-based targets were found to secure greater proportions of adaptive and neutral genetic variation (Figure 1). The proportion of genetic variation that these prioritizations secured was substantially more than the genetic variation secured by randomly generated prioritizations (`r xxx`). Generally, prioritizations needed to capture at least `r xxx` % of the environmental and geographic surrogates to secure 90 % of the species' genetic variation. Additionally, we detected differences in the effectiveness of the environmental and geographic surrogates (`r xxxx`). Increases in environmental surrogate-based targets resulted prioritizations that secured more adaptive genetic variation, than geographic surrogate-based targets for neutral variation. These results demonstrate that environmental and geographic variables are effective surrogates for genetic variation.

<!-- Scenario analysis -->
The addition of environmental and geographic surrogate-based targets to amount-based targets was not found to substantially improve the effectiveness of prioritizations under any of the scenarios (Figure 2). No significant differences were detected between prioritizations generated using different combinations of targets (${\chi^{2}}_{`r `,`r `} = `r `$, $P `r `$), or under any of the different scenarios (${\chi^{2}}_{`r `,`r `} = `r `$, $P `r `$). The only difference was that the inclusion of surrogate-based targets tended to result in prioritizations that secured slightly more adaptive than neutral genetic variation. Under the simplest of circumstances, the single-species amount-based prioritizations secured most of the species' adaptive (mean `r ` \pm `r `) and neutral (mean `r ` \pm `r `) genetic variation. These proportions were not significantly different to those associated with prioritizations generated using additional surrogate-based targets ($Z = `r `$, $P `r `$) or genetic-based targets ($Z = `r `$, $P `r `$). We found similar patterns in the multi-species scenario () and the multi-species with opportunity cost scenario (). Although the prioritizations had similar levels of performance, there were different in the their spatial distributions (Figure 3). In particular, the inclusion of cost data in the multi-species prioritizations had the effect of concentrating conservation effort towards areas with higher elevation. Overall, these results suggest that while environmental and geographic surrogate variables are effective surrogates, they are not needed to generate prioritizations that secure most of the intra-specific genetic variation.

## Discussion
We aimed to investigate the effectiveness of environmental and geographic surrogates for genetic variation in conservation planning. This study is the first to empirically test this using a realistic system. We found that the inclusion of environmental and geographic surrogates in single-species prioritizations significantly improved the proportion of intra-specific genetic variation they secured. But we also found that the same surrogates did little to improve the multi-species prioritizations. Our results suggest that conservation planners can use a comprehensive set of species to secure a sufficient proportion of their intra-specific genetic variation. 

### Single + multi-species analysis
* summarize results

* compare + contrast with other papers

* implication of results

### Correlation analysis
* summarize results

* compare + contrast with other papers

* implication of results

### Limitations/Future directions
We wish to alert practitioners to several limitations associated with our analysis. First, the size of the planning units we used (23 \times 25 km) is much larger than typically used in conservation planning scenarios (approx. 1--5 km^2). We used this resolution because the genetic data was collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis because the surrogate variables are spatially auto-correlated. Second, we used geographic distances as surrogates for connectivity between planning units. Although connectivity between areas can be modeled using isolation by resistance measures [@r473] derived from landscape data [eg. @r474], they are unlikely to be of use in broad-scale conservation exercises since they need to be tuned using species-specific parameters. Third, we used AFLP data to characterize the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @rXXX], our methodology would still have utilized only the main gradients of the genetic variation to generate prioritizations in a feasible amount of time. Overall, the limitations associated with this study are relatively minor.

### Concluding remarks
The overarching aim of our study was to investigate the effectiveness of surrogates in conservation planning. We found that environmental and geographic surrogates are effective. However, they seem to be largely unnecessary in conservation planning scenarios because a comprehensive set of species can be used to achieve a similar result. Conservation planners are therefore urged to use a comprehensive set of species in prioritizations--one of the fundamental principals of conservation planning.

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References

\end{linenumbers}