---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
setwd('..')

# load .rda file
session::restore.session('results/results.rda')

# set options
options(error=NULL)
```


<!--  title page -->
Short running title: Genetic surrogates in conservation \newline
Word count: XXXXX \newline
Number of references: XXXX \newline
Number of figures, table and text boxes: 4 \newline

\newpage

<!-- add line numbers -->
\begin{linenumbers}

## Introduction
<!--  main text -->
Biodiversity is under threat. In recent history, many species have gone extinct [@r270; @r272]. Today, thousands of species are listed as endangered or near threatened in the International Union for Conservation of Nature (IUCN) red list (\url{www.iucnredlist.org}). In the near future, species are expected to face even more threats [eg. climate change; @r236; @r243]. In response to this crisis, the overarching aim of conservation is to maximize the long-term persistence of biodiversity [@r255; @r67].

Protected areas are one of the hallmarks of modern conservation [@r440]. They buffer species from anthropogenic impacts and provide places for biodiversity processes to persist (eg. nutrient cycling and water purification). Protected areas also set the stage for direct management actions to enhance species recovery (eg. habitat rehabilitation, translocation, and invasive species management). But the resources available for conservation are limited, and so protected areas must be cost-effective. As a consequence, decision support tools [eg. $\texttt{MARXAN}$; @r22; $\texttt{Zonation}$; @r429] are used to identify cost-effective plans for protected areas (prioritizations). These tools typically estimate the benefit that a given species receives from a given prioritization based on the proportion of its range that is secured (eg. using targets). However, the true benefit of a given prioritization is dependent on many additional factors that are not used to inform the reserve selection process.

Protected areas must preserve intra-specific genetic variation to maximize species recovery [@r67; @r3; @r61]. This variation can be classified as either adaptive or neutral [reviewed in @r????]. Adaptive genetic variation is associated with loci currently under selection. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral genetic variation is associated with loci that are not under natural selection. Neutral variation often reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can secure different genetic lineages and avoid the adverse effects of low genetic diversity [eg. inbreeding depression; @r67]. However, genetic data requires considerable investment to obtain and analyze. As a consequence, conservation planners do not typically use genetic data to inform prioritizations [@r61]. Thus many prioritizations may fail to capture species' intra-specific genetic variation, and ultimately fail to deliver conservation outcomes.

In light of this problem, conservation scientists have been investigating surrogates for genetic variation to generate better prioritizations--but without the overheads associated with genetic data. Since adaptive genetic variation is ultimately driven by selection pressures, conservation planners have used the distribution of likely selection pressures. For instance, environmental variation such as local habitat type [@r247] and climate [@r2] have been used to guide the reserve selection process. While such variables have been used to detect and verify adaptive genetic variation [eg. @r475; @r455], the effectiveness of prioritizations based on such surrogates remains unverified. 

On the other hand, neutral genetic variation arises a reduction in gene flow between individuals. By modeling the connectivity between areas, a function of gene flow, conservation scientists have explored the effectiveness of surrogates for neutral genetic variation. For instance, the isolation by distance theory [IBD; @r259] predicts that populations further apart will have reduced connectivity, and in turn exchange less genetic material. The use of geographic distance-based surrogates has been found to improve prioritizations in an island network [@r200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain; eg. @r474]--the systems commonly featured in real-world conservation planning exercises. Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates needs to be verified. Otherwise, if these surrogates are ineffective, conservation planners may waste precious resources on places with little benefit. 

Our overarching aim was to investigate the effectiveness of environmental and geographic surrogates for adaptive neutral genetic variation in the context of conservation planning. First, we aimed to understand the relationship between the surrogate targets used to generate a prioritization and the proportion of genetic variation secured in the prioritization. Second, we aimed to determine if the addition of surrogate-based targets in more realistic conservation planning scenarios would result in prioritizations that secured substantially more genetic variation. If these surrogates can be shown to improve the intra-specific genetic variation captured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
To address the aims of the study, we utilized species distribution and genomic data collected by the IntraBioDiv project [@r451; see @r452 for methodological details]. This dataset was particularly well suited; it provides spatially explicit genomic data for a multitude of species across a broad geographic area over large environmental gradients. As part of this project, data were collected for 27 alpine plant species were across the European Alps according to a grid (22.3 km $\times$ 25 km; Figure S1). Briefly, in every second cell, plant samples were collected from three individuals for each species present inside the cell. Cells that did not contain any plant samples were omitted from our analysis. Samples were genotyped using amplified fragment length polymorphisms [AFLP; @r453]. Matrices denoting the presence/absence of polymorphisms at loci were constructed independently for each species. Thus for each second grid cell, for each of the species in the grid cell, the dataset contains information describing the genomic variation of individuals inside the grid cell. We used these grid cells as planning units to develop prioritizations to investigate the effectiveness of surrogates for the (AFLP) genetic variation. Most of our analyses described hereafter were performed using the \texttt{R} software package [version `r paste0(version$major, '.', version$minor)`; @r192].

### Landscape data
We aimed to investigate the effectiveness of climatic and geographic surrogates. To describe the location of each grid cell, we projected the grid into an equi-distant coordinate system ($\texttt{ESRI:102031}$), calculated their centroids, and extracted its two-dimensional coordinates. To describe the climatic variation among each cell, we obtained 19 bioclimatic variables [approx. 1 km resolution; @r33]. These layers and the grid cells were projected into an equal-area coordinate system ($\texttt{ESRI:102014}$). The bioclimatic variables were clipped to the extent of the grid and subject to a principle components analysis (PCA; Table S1). The first two principle components (PCs) cumulatively explained `r round(pca.DF[[4]][2],1)` % of the variation and were used to generate two new layers. We calculated the average value for each PC layer in each grid cell, and used these values to characterize the main themes of climatic variation among the grid cells (Figure S2). To examine if the influence of opportunity cost, we calculated the total human population density inside each planning unit (obtained at 1 km resolution from the Global Rural-Urban Mapping Project; GRUMP V1; @r472).

### Outlier locus detection
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first need to identify which of the sampled loci--if any--were under selection for each species. We used outlier locus detection methods [evaluated in @r454] to achieve this. The basic premise underpinning these analyses is that neutral loci are expected to exhibit a specific level of variation among populations ($F_{st}$), and loci that deviate from this expectation should be under selection. The advantage of these methods--in contrast with environmental association analyses [eg. r@475]--is that they do not use environmental data to identify loci under selection, which would have biased our analysis. In an earlier study, Bothwell _et al._ [-@r455] applied such outlier detection methods to one of the species in this dataset (_Getiana nivalis_). Here, we adopted a similar methodology and applied it to each of the 27 species in the dataset.

First, we grouped individuals into genetic lineages for each species. The purpose of this step was to avoid misclassifying loci as adaptive. We used discriminant analyses of principle components [DAPC; @r467] to assign individuals to populations [implemented in $\texttt{adegenet}$ R package; @r468]. Briefly, for each species, we transformed the binary loci data using a principle components analysis (PCA) and obtained the principle components required to explain at least `r adegenet.params.LST[[MODE]]$perc.pca` % variation. We subject these components to multiple \textit{k}-means clustering models with varying numbers of assumed populations (\textit{k} = 1--`r adegenet.params.LST[[MODE]]$max.n.clust`; `r adegenet.params.LST[[MODE]]$n.iter` iterations; `r adegenet.params.LST[[MODE]]$n.start` random starts). We then  applied Ward's clustering method to the Bayesian Information Criterion (BIC) statistics associated with each model to partition the models into two groups, and selected the model before the first group switch (as recommended in the \texttt{adegenet} manual; Figure S4). The populations memberships associated with these models were used for subsequent analysis (Figure S5).

We identified loci under selection for each species (Table S2). For each species, we ran $\texttt{Bayescan}$ [version 2.1; @r456] using the population memberships identified in the previous analysis (1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations). We omitted loci where the global frequency of the minor allele was $\geq 0.05$ to avoid false-positives (following guidelines in the $\texttt{Bayescan}$ manual). To ensure convergence, we ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate [FDR; $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species (Figures S6--S32). For each species, for each type of genetic variation detected, we used Gower distances [@r464] to express the genomic differences between individuals [using the $\texttt{cluster}$ R package to accommodate sparsity; @r457]. These distances were subject to multiple non-metric multi-dimensional scaling analyses [NMDS; implemented in the $\texttt{vegan}$ R package; @r458] using the minimum number of dimensions required to achieve an adequate stress value  ($\leq$ `r mds.params.LST[[MODE]]$max.stress`; `r mds.params.LST[[MODE]]$trymax` random starts for all models; Table S3). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the samples in each grid cell to characterize their the genetic variation of individuals in the cells.

### Prioritizations
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve a proportion of the species' range as well as a proportion of the species' variation using an attribute space [_sensu_ @r465]. We treated the genomic and surrogate variables as distinct attribute spaces. We set the demand points for each species and its attribute space using the values associated with the planning units. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

To understand the relationship between the surrogate-based targets used to generate a prioritizations and the amount of genetic variation secured, we generated a collection of single-species prioritizations. For each species, we generated two sets of prioritizations using increasing environmental and geographic-based targets (respectively). Prioritizations were generated using surrogate-based targets ranging from `r (min(rapr.params.LST[[MODE]]$correlation.surrogate.targets)*100)` % and `r (max(rapr.params.LST[[MODE]]$correlation.surrogate.targets)*100)` % in increments of `r diff(rapr.params.LST[[MODE]]$correlation.surrogate.targets[1:2])*100` %. None of these prioritizations contained any amount-based targets. 

To determine if the use of surrogate-based targets can actually improve prioritizations in real-world conservation planning scenarios, we used three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritizations for each species. Most large-scale prioritizations involve a comprehensive set of species, and in turn, may secure a greater proportion of intra-specific genetic variation. Our second scenario involved generating multi-species prioritizations. Finally, most large-scale prioritizations also explicitly consider opportunity cost to ensure prioritizations are cost-effective. Our third scenario involved generating multi-species prioritizations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritizations. We generated prioritizations using only amount-based, both amount- and surrogate-based targets, and both amount- and genetic-based targets. The purely amount-based prioritizations represented those typically generated in conservation planning exercises and serve as our baseline. We used `r (rapr.params.LST[[MODE]]$amount.target*100)` % amount-based targets in all prioritizations to ensure an adequate proportion of habitat was secured for each species [@rXXX]. We set surrogate- and genetic-based targets as `r (rapr.params.LST[[MODE]]$surrogate.target*100)` % to ensure that most of the repsective variation was secured for each species. To accomodate variation in the high number of optimal solutions for the single-species amount-based prioritizations, we generated `r (rapr.parmas.LST[[MODE]]$amount.replicates)` replicate prioritizations. Since all other contexts were likely to be associated with a single optimal solution, we used only a single replicate.

## Statistical analysis
We investigated the relationship between surrogate-based targets and the genetic variation secured in a prioritization using nonlinear mixed-effects models [NLMMs; using the $\texttt{lme4}$ R package; @r469]. A model was fit to the genetic variation secured using a negative exponential formulation to estimate the rate of the accumulation of genetic variation ($y = a - (a - 0) \times exp(-r \times x)$; where $a$ is the asymptote and $r$ is the accumulate rate). An interaction term was included to estimate parameters separately for adaptive and neutral variation. The models accommodated species-level variation using random effects for both parameters.

We tested for differences in the effectiveness of prioritizations generated under different scenarios and targets using generalized linear mixed-effects models [GLMMs; using the $\texttt{lme4}$ R package]. We fit a full model to the proportion of genomic variation secured in all prioritizations using a logit link. Both models contained variables indicating the scenario used to generate the prioritization, the targets used to generate the prioritizations, and the type of genetic variation measured. The model was subject to a step-wise term deletion routine using likelihood ratio tests (LRT) to assess term significance (terms were retained when $P < 0.05$). All models contained an intercept-only random effect to ensure that comparisons were made within species. Finally, we conducted a post-hoc analysis using Tukey contrasts with a Bonferroni correction [using the $\texttt{multcomp}$ R package; @r470].

## Results
<!-- Single species prioritizations -->
Surrogates-targets improved the performance of single-species prioritizations (Figure 1). There were significant differences among the performance of prioritizations depending on the targets used to generate them and if they were assessed according to adaptive or neutral genetic variation (${\chi^{2}}_{`r single.spp.AOV1[['Df']][1]`,`r single.spp.AOV1[['Df']][2]`} = `r round(single.spp.AOV1[['Chisq']][2],2)`$, $P `r pretty.pval(single.spp.AOV1[['Pr(>Chisq)']][2])`$). Specifically, the amount-based prioritization captured much less adaptive and neutral genetic variation than either the surrogate-based ($Z = `r round(single.spp.MCP$test$tstat[['Surrogate.Adaptive variation - Amount.Adaptive variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Surrogate.Adaptive variation - Amount.Adaptive variation']])`$; $Z = `r round(single.spp.MCP$test$tstat[['Surrogate.Neutral variation - Amount.Neutral variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Surrogate.Neutral variation - Amount.Neutral variation']])`$) or genetic-based ($Z = `r round(single.spp.MCP$test$tstat[['Genetic.Adaptive variation - Amount.Adaptive variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Genetic.Adaptive variation - Amount.Adaptive variation']])`$; $Z = `r round(single.spp.MCP$test$tstat[['Genetic.Neutral variation - Amount.Neutral variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Genetic.Neutral variation - Amount.Neutral variation']])`$) prioritizations. No differences were detected in the proportion of adaptive ($Z = `r round(single.spp.MCP$test$tstat[['Genetic.Adaptive variation - Surrogate.Adaptive variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Genetic.Adaptive variation - Surrogate.Adaptive variation']])`$) or neutral genetic variation ($Z = `r round(single.spp.MCP$test$tstat[['Genetic.Neutral variation - Surrogate.Neutral variation']],2)`$, $P `r pretty.pval(single.spp.MCP$test$pvalue[['Genetic.Neutral variation - Surrogate.Neutral variation']])`$) secured by the the surrogate-based and genetic-based prioritizations. However, these patterns were not reflected in the multi-species prioritizations.

<!-- Multi-species prioritizations -->
Environmental and geographic surrogates were not found to improve the performance of multi-species prioritizations (Figure 2; Figure 3). No significant differences in the effectiveness of prioritizations were found depending on whether they were generated using different targets (${\chi^{2}}_{`r multi.spp.AOV3[['Df']][1]`,`r multi.spp.AOV3[['Df']][2]`} = `r round(multi.spp.AOV3[['Chisq']][2],2)`$; $P `r pretty.pval(multi.spp.AOV3[['Pr(>Chisq)']][2])`$), if they were assessed using adaptive or neutral genetic variation (${\chi^{2}}_{`r multi.spp.AOV2[['Df']][1]`,`r multi.spp.AOV2[['Df']][2]`} = `r round(multi.spp.AOV2[['Chisq']][2],2)`$; $P `r pretty.pval(multi.spp.AOV2[['Pr(>Chisq)']][2])`$), nor the interaction between these variables (${\chi^{2}}_{`r multi.spp.AOV1[['Df']][1]`,`r multi.spp.AOV1[['Df']][2]`} = `r round(multi.spp.AOV1[['Chisq']][2],2)`$; $P  `r pretty.pval(multi.spp.AOV1[['Pr(>Chisq)']][2])`$).

<!-- Correlation analysis -->
The surrogate variables showed a greater utility for neutral genetic variation than adaptive variation (Figure 4). The accumulation rate ($r$) was `r round(summary(env.correlation.NLMM)$coefficients[1],2)` $\pm$ `r round(summary(env.correlation.NLMM)$coefficients[2],2)` S.E. for the proportion of adaptive variation captured in a prioritization using environmental surrogates. On the other hand, the rate ($r$) was `r round(summary(geo.correlation.NLMM)$coefficients[1],2)` $\pm$ `r round(summary(geo.correlation.NLMM)$coefficients[2],2)` S.E. for neutral variation using geographic surrogates. Thus minor increases in the target for either surrogate results in major increases in the benefit of the prioritization. However, the coefficient was much larger for neutral genetic variation so minor increases in the target can achieve substantial gains.

## Discussion
We aimed to investigate the effectiveness of environmental and geographic surrogates for genetic variation in conservation planning. This study is the first to empirically test this using a realistic system. We found that the inclusion of environmental and geographic surrogates in single-species prioritizations significantly improved the proportion of intra-specific genetic variation they secured. But we also found that the same surrogates did little to improve the multi-species prioritizations. Our results suggest that conservation planners can use a comprehensive set of species to secure a sufficient proportion of their intra-specific genetic variation. 

### Single + multi-species analysis
* summarize results

* compare + contrast with other papers

* implication of results

### Correlation analysis
* summarize results

* compare + contrast with other papers

* implication of results

### Limitations/Future directions
We wish to alert practitioners to several limitations associated with our analysis. First, the size of the planning units we used (23 \times 25 km) is much larger than typically used in conservation planning scenarios (approx. 1--5 km^2). We used this resolution because the genetic data was collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis because the surrogate variables are spatially auto-correlated. Second, we used geographic distances as surrogates for connectivity between planning units. Although connectivity between areas can be modeled using isolation by resistance measures [@r473] derived from landscape data [eg. @r474], they are unlikely to be of use in broad-scale conservation exercises since they need to be tuned using species-specific parameters. Third, we used AFLP data to characterize the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @rXXX], our methodology would still have utilized only the main gradients of the genetic variation to generate prioritizations in a feasible amount of time. Overall, the limitations associated with this study are relatively minor.

### Concluding remarks
The overarching aim of our study was to investigate the effectiveness of surrogates in conservation planning. We found that environmental and geographic surrogates are effective. However, they seem to be largely unnecessary in conservation planning scenarios because a comprehensive set of species can be used to achieve a similar result. Conservation planners are therefore urged to use a comprehensive set of species in prioritizations--one of the fundamental principles of conservation planning.

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References

\end{linenumbers}