---
title: "Are environmental and geographic effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 200 word limit for abstract -->
<!-- 6 page limit w/o incurring costs -->

```{r, include=FALSE}
# set wd
setwd('..')

# load .rda files
load('results/results.rda')

# load packages
source('R/analysis/00-initialization.R')

# set options
options(error=NULL)
```

## Introduction

<!--
paragraph 1
topic sentence: biodiversity is in crisis (@rXX, @rXX).
	+ in recent years many species have gone extinct (@rXX).
	+ currently there are thousands of species listed as endangered or near threatened in the International Union for Conservation of Nature (IUCN) red list (www.iucnredlist.org).
	+ futhermore, climate change is expected cause even greater loss to global biodiversity (@rXXX).
linking sentence: in response to this crisis, the overarching aim of conservation is to ensure the persistance of biodiversity patterns and processes.
-->

Biodiversity is under threat. In recent years many species have gone extinct [@r272; @r270]. currently there are thousands of species listed as endangered or near threatened in the International Union for Conservation of Nature (IUCN) red list (www.iucnredlist.org). Futhermore, climate change is expected cause even greater loss to global biodiversity [@r236]. In response to this crisis, the overarching aim of conservation is to maximise the long-term persistance of conservation [@r255].

<!--
paragraph 2
topic sentence: the act of setting aside areas for the preservation of biodiversity is one the hallmarks of conservation.
	+ protected areas buffer species from anthropogenic impacts and provide areas for biodiversity processes to persist (eg. nutrient cycling and water purification).
	+ additionally, protected areas set the stage for direct management actions (eg. habitat rehabitilation, translocation, and invasive species management).
	+ since the resources avaliable for conservation are limited, the plans for protected areas (prioritizations) are often generated using biodiversity data (eg. species distribution data; @rXXX) , expressing the conservation planning problem as an optimisation problem, and solving it.
linking sentence:
-->

The act of setting aside areas for the preservation of biodiversity is one the hallmarks of modern conservation [@r440]. Protected areas buffer species from anthropogenic impacts and provide areas for biodiversity processes to persist (eg. nutrient cycling and water purification). Additionally, protected areas set the stage for direct management actions (eg. habitat rehabitilation, translocation, and invasive species management). Since the resources avaliable for conservation are limited, the plans for protected areas (prioritizations) are often generated by using biodiversity data and decision support tools [eg. \texttt{MARXAN}, @r22; \texttt{Zonation}, @r429] to identify cost-effective prioritizations. 

<!--
paragraph 3
topic sentence: intra-specific genetic variation is important for species persistance.
	+ Genetic variation is classified as either adaptive or neutral. 
	+ Adaptive genetic variation refers to loci currently under selectiom.
	+ Neutral genetic variation to loci that are not under selection, and reflects the evolutionary history of populations.
	+ By preserving existing patterns of adaptive genetic variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost.
	+ In constrast, by preserving existing patterns of neutral genetic variation, protected areas can secure populations belonging to genetic lineages and avoid inbreeding depresssion.
	+ Protected areas must preserve a representative sample of both of these types of genetic variation to fulfil the overarching aims of conservation.
linking sentence: However, genetic data is costly to obtain and requires expert knowledge to process for use in conservation planning. As a consequence, genetic data is not used to inform large-scale prioritisations, and so these prioritisations may fail to preserve the intra-specific genetic variation for the species they aim to preserve.
-->

<!--
paragraph 5
topic sentence: Cost-effective surrogate data could be used to identify prioritizations that preserve intra-specific genetic variation.
+ Adaptive genetic variation is driven by selection pressures.
+ Conservation planners have explored the use of data on the distribution of likely selection pressures to generate prioritizations to secure adaptive genetic variation [eg. @rXXX; @rXXX; @rXXXX].
+ On the other hand, neutral genetic variation arises due to the break down of gene flow between populations.
+ Conservation planners have explored the use of variables that predict a reduction in gene flow between populations.
+ The isolation by distance theory [IBD; @rXX] predicts that populations are further apart will have lower gene flow.
+ The use of geographical surrogate data has been found to improve prioritisations in an island network [@rXX].
+ However this remains untested in system with in a terrestrial study system where connectivity is affected additional factors (eg. elevation).
linking sentence:
-->

<!--
paragraph 6
topic sentence: The aim of this study is to investigate the effectiveness of environmental and geographic surrogates for conservation planning.
	+ First, we aim to test the effectiveness of these surrogates under the simplest of conditions in a set of single-species prioritisations.
	+ Second, we aim to test the effectiveness of these surrogates in a more realistic multi-species context.
	+ Third, we aim to understand the relationship between the proportion of surrogate variation secured in a multi-species prioritisation and the prorpotion of genetic variation secured in a prioritization.
linking sentence:
-->

## Methods
### Study area
To address the aims of the study, we utilized species distribution and genomic data collected by the IntraBioDiv project [Figure S1; @r451; see @r452 for methodological details]. This dataset was well suited for this study, because it provides spatially explicit genomic data for a multitude of species across a broad geographic area and over large environmental gradients. As part of this project, data for 27 alpine plant species were collected across the European Alps according to a grid (22.3 km $\times$ 25 km). Briefly, in every second cell, plant samples were collected from three individuals for each species present inside the cell. Samples were genotyped using amplified fragment length polymorphisms [AFLP; @r453]. Matrices denoting the presence/absence of polymorphisms at loci were constructed independently for each species. Thus for each second grid cell, for each of the species in the grid cell, the dataset contains information describing the genomic variation of individuals inside the grid cell. We used these grid cells as planning units to develop prioritizations to investigate the effectiveness of surrogates for the (AFLP) genetic variation.

### Surrogate data
We explored the effectiveness of climatic and geographic surrogates. To describe variation in the location of each grid cell, we projected the grid into an equi-distant coordinate system (ESRI:102031) and used the centroid of each cell. To describe the climatic variation among each cell, we obtained twenty-one bioclimatic variables [approx. 1km resolution; @r33]. These layers and the grid cells were projected into an equal-area coordinate system (ESRI:102014). The bioclimatic variables were clipped to the extent of the grid and subject to a principle components analysis (PCA; Table S1). The first two principle components (PCs) cumulatively explained `r round(pca.DF[[4]][2],1)`% of the variation and were used to generate two new layers. We calculated the average value for each PC layer in each grid cell, and used these values to characterize the main themes of climatic variation among the grid cells (Figure S2).

### Outlier locus detection
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first identified which of the sampled loci for each species were under selection. We used outlier locus detection methods [evaluated in @r454] to avoid circularity issues. The basic premise underpinning these analyses is that neutral loci are expected to have a specific level of variation among populations ($F_{st}$), and loci that deviate from this expectation should be under selection. Thus loci can be identified as being under selection without using environmental data. In an earlier study, Bothwell _et al._ [-@r455] applied such outlier detection methods to one of the species in this dataset (_Getiana nivalis_). Here, we adopted a similar methodology and applied it to each of the 27 species in the dataset.

First, we grouped individuals into genetic lineages for each species (Figure S2). The purpose of this step was to avoid confounding variation between populations attributed to genetic history instead of selection pressures. We used discriminant analyses of principle components [DAPC; @r467] to assign individuals to populations [implemented in \texttt{adegenet} R package; @r468]. Briefly, for each species, we transformed the binary loci data using a principle components analysis (PCA) and obtained the principle components required to explain at least `r adegenet.params.LST[[MODE]]$perc.pca`% variation. We subject these components to multiple $k$-means clustering models with varying numbers of assumed populations (1--`r adegenet.params.LST[[MODE]]$max.n.clust`; `r adegenet.params.LST[[MODE]]$n.inter` iterations; `r adegenet.params.LST[[MODE]]$n.start` random starts). We then  applied Ward's clustering method to the Bayesian Information Criterion (BIC) statistics associated with each model to partition the models into two groups. We selected the model before the first group switch, and used the population memberships in that model for subsequent analysis.

We used $\texttt{Bayescan}$ [version 2.1; @r456] to identify loci under selection for each species (Table S2). For each species, $\texttt{Bayescan}$ was run using the population memberships identified in the previous analysis (1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations). Following guidelines in the $\texttt{Bayescan}$ manual, we omitted loci where the global frequency of the minor allele was $\geq 0.05$ to avoid false-positives. To ensure convergence, we ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate [FDR; $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if present) and neutral genetic variation for each species (Figures S4--S29). For each species, for each type of genetic variation present, we used Gower distances [@r464] to express the genomic differences between individuals [using the $\texttt{cluster}$ R package to accommodate sparsity; @r457]. These distances were subject to multiple non-metric multi-dimensional scaling analyses using the minimum number of dimensions required to acheive an adequate stress value  [`r mds.params.LST[[MODE]]$max.stress`; `r mds.params.LST[[MODE]]$trymax` random starts; Table S3; implemented in the \texttt{vegan} R package; @r458]. Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the samples in each grid cell to characterize the genomic variation among the grid cells.

### Prioritizations
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve a proportion of the species' range as well as a proportion of the species' variation [using an attribute space; _sensu_ @r465]. We treated the genomic and surrogate variables as distinct attribute spaces. We set the demand points for each species and its attribute space using the values associated with the planning units.

To fulfil the aims of the study, we generated three sets of prioritisations. First, we generated a set of single species prioritizations. For each species, we generated a prioritization using a `r (rapr.params.LST[[MODE]]$amount.target*100)`% amount-based target, a prioritization using a `r (rapr.params.LST[[MODE]]$amount.target*100)`% amount-based target and `r (rapr.params.LST[[MODE]]$surrogate.target*100)`% space-based targets for the surrogate variation, and a prioritization using a `r (rapr.params.LST[[MODE]]$amount.target*100)`% amount-based target and `r (rapr.params.LST[[MODE]]$genetic.target*100) `% space-based targets for the genomic variation. Next, we generated three multi-species prioritizations using the same sets of targets used in the single species prioritizations. Finally, to understand the relationship between the surrogate-based and genomic-based targets, we generated a suite of prioritizations using increasing space-based targets between `r (min(rapr.params.LST[[MODE]]$pareto.surrogate.targets)*100)`% and `r (max(rapr.params.LST[[MODE]]$pareto.surrogate.targets)*100)`% in increments of `r diff(rapr.params.LST[[MODE]]$pareto.surrogate.targets[1:2])*100`%. All of these prioritisations had no amount-based targets. For each prioritization we calculated the proportion of the adaptive (if present) and neutral genetic variation that it secured. We solved the reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)`% of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

## Statistical analysis
We tested for differences in the effectiveness of prioritisations at securing genomic variation using generalized linear mixed-effects models (GLMMs). These models were fit to the proportion of genomic variation explained using a logit link. They contained a predictor variable indicating if the prioritisation was generated using amount, surrogate, or genetic targets, a second predictor variable indicating if the genomic variation was adaptive or neutral, and the interaction between these variables. All models contained an intercept-only random effect to ensure that comparisons were made between species. These models were subject to stepwise term deletion routes using likelihood ratio tests (LRT) to assess term significance and identify the minimal adequate model (terms were retained when $P < 0.05$).

## Results
### Single species prioritizations


### Multi-species prioritisations


### Pareo-frontier analysis


## Discussion


## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References

