---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
setwd('..')

# load .rda file
try(session::restore.session('results/results.rda'))

# set options
options(error=NULL)
```

<!--  title page -->
Short running title: Genetic surrogates in conservation \newline
Word count: XXXXX \newline
Number of references: XXXX \newline
Number of figures, table and text boxes: 4 \newline
\newpage


## Introduction
<!--  main text -->
The act of setting aside areas for preservation spearheads modern conservation efforts [@r440]. They buffer species from anthropogenic impacts and provide places for populations of species to persist. Intra-specific genetic variation underpins species persistence, and there has been increasing interest in understanding how to design protected areas networks in a way that maximizes this variation [@r67; @r3; @r61]. Broadly speaking, genetic variation can be classified as either adaptive or neutral [reviewed in @r48]. Adaptive genetic variation is associated with loci that affect fitness. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral genetic variation is associated with loci that do not affect fitness. Neutral variation often reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can secure different genetic lineages and avoid the adverse effects of low genetic diversity [eg. inbreeding depression; @r67]. However, genetic data requires considerable investment to obtain and analyze. As a consequence, conservation planners do not typically use genetic data to inform prioritizations [@r61]. Thus prioritizations may fail to capture species' intra-specific genetic variation, and ultimately fail to deliver long-term conservation outcomes.

Recently, conservation scientists have been investigating surrogates for genetic variation to generate better prioritizations without the overheads associated with genetic data. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been used to guide reserve selection [eg. habitat conditions; @r247; @r2]. However, their effectiveness as surrogates in conservation planning remains unverified. On the other hand, neutral genetic variation arises from a reduction in gene flow between individuals. Surrogates for neutral variation are based on variables that predict the level of connectivity between planning units. For instance, the isolation by distance theory [IBD; @r259] predicts that populations further apart will have less connectivity, exchange less genetic material, and in turn have less genetic variation in common. Based on this theory, geographic distance-based surrogates have been found to improve prioritizations in an island network [@r200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain; eg. @r474]--the systems commonly featured in real-world conservation planning exercises. Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates needs to be verified. Otherwise, conservation planners may waste precious resources on places that do little to benefit biodiversity.

Here we investigate the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation in the context of conservation planning. First, we examine the relationship between surrogates and gene variation, and verify that these variables can be used as surrogates. Second, we determine if the inclusion of surrogate-based targets in a variety of conservation planning scenarios with increasingly levels of realism can result in prioritizations that secure more genetic variation. If these surrogates can be shown to improve the intra-specific genetic variation captured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
We used species distribution and genomic data collected by the IntraBioDiv project in the European Alps [@r451; see @r452 and @r478 for methodological details]. This data set was particularly well suited for our study, because it provides spatially explicit genomic data for multiple species with vastly different demographic characteristics and evolutionary histories across a broad geographic area with large environmental gradients. As part of this project, data was collected for 27 alpine plant species using a 20' longitude by 21' latitude grid (approx. 21' 22.3 km $\times$ 25 km; Figure S1). They visited every second grid cell, and if a species was detected in a cell, samples were collected from three individuals. They genotyped samples using amplified fragment length polymorphisms [AFLP; @r453], and constructed matrices denoting the presence/absence of polymorphisms at loci for each species. Thus for each second grid cell, for each of the species present in the grid cell, the data set contains information describing the genomic variation of individuals in the cell. We used these grid cells as planning units to develop prioritizations. To reduce computational effort, we only used grid cells that contained at least a single individual ($n = `r nrow(grid.DF)`$). The majority of our analyses were performed using the \texttt{R} software package [version `r paste0(version$major, '.', version$minor)`; @r192].

### Landscape data
We examined the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation (respectively). To describe the geographic location of each grid cell, we projected the grid into an equi-distant coordinate system ($\texttt{ESRI:102031}$), calculated their centroids, and extracted their two-dimensional coordinates. To describe the environmental characteristics of each grid cell, we obtained 19 bioclimatic variables [approx. 1 km resolution; @r33]. These layers and the grid cells were projected into an equal-area coordinate system ($\texttt{ESRI:102014}$). The bioclimatic variables were clipped to the extent of the grid and subject to a principal components analysis (PCA; using \texttt{ArcMap} version 10.2.2; Table S1). The first two principal components (PCs) cumulatively explained `r round(pca.DF[[4]][2],1)` % of the variation and were used to generate two new layers. We calculated the average value for each PC layer in each grid cell, and used these values to characterize the main themes of climatic variation among the grid cells (Figure S2). To examine if the influence of opportunity cost, we calculated the total human population density inside each planning unit (obtained at 1 km resolution from the Global Rural-Urban Mapping Project; \texttt{GRUMP V1}; @r472).

### Outlier locus detection
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first needed to identify which of the sampled loci--if any--were under selection. We used outlier locus detection methods [compared in @r454; reviewed in @r480] to achieve this. The basic premise underpinning these analyses is that neutral loci are expected to exhibit a specific level of variation among populations ($F_{st}$), and loci that deviate from this expectation should be under selection. The advantage of these methods--in contrast with environmental association analyses [eg. @r475]--is that they do not use environmental data to identify loci under selection, which would have biased our analysis. In an earlier study, Bothwell _et al._ [-@r455] applied such outlier detection methods to one of the species in this data set (_Gentiana nivalis_). Here, we adopted a similar methodology and applied it to each of the 27 species in the data set.

First, we grouped individuals into genetic lineages for each species (using a methodology similar to that implemented in \texttt{prabclust R} package; @r479). The purpose of this step was to avoid falsely classifying loci as adaptive. For each species, we ordinated the genomic data and fit models to the ordinations to fit to turn group individuals into populations. Specifically, we used non-metric multi-dimensional scaling [NMDS; implemented in the $\texttt{vegan R}$ package] using Gower distances [@r464; using the $\texttt{cluster R}$ package to accommodate sparsity; @r457] to transform the binary loci data into continuous variables (Table S2). To ensure that the ordinations described a sufficient amount of the genomic variation, we ran successive scaling analyses with increasing dimensionality until this was achieved (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for each analysis). We then fit a suite of Gaussian mixture to the ordinations to estimate the probability that individuals belonged to different populations [assuming different covariance structures implemented in the \texttt{mclust} R package; @r477; @r476]. For each parametrization, we fit a set of models assuming varying numbers of total populations (\textit{G}= 1--`r max(mclust.params.LST[[MODE]]$G)`). We identified the most parsimonious model for each species (lowest Bayesian Information Criterion $BIC$ statistic; Figures S4), and used them to group individuals to populations (Figure S5).

We identified the loci under selection (Table S3). Since outlier locus detection methods compare allele frequencies between populations, we were unable to examine species that contained a single population. For these species ($n = `r sum(sapply(spp.Mclust.LST, function(x) return(ncol(x$mclust$z)==1)))`$), we assumed that all loci were neutral. For species with more than a single population, we ran $\texttt{Bayescan}$ [version 2.1; @r456; 1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations]. To avoid false-positives, we omitted loci where the global frequency of the minor allele was $\geq 0.05$ (following guidelines in the user manual) and assumed they were neutral. Additionally, we omitted individuals from this analysis when their population membership was uncertain (maximum membership probability $\l `r mclust.params.LST[[MODE]]$probability.threshold`$). We ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate to ensure convergence [FDR $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species (Figures S6--S32). For each species, we partitioned adaptive and neutral loci into separate data sets, and reran the successive nonmetric multidimensional scaling analyses (using parameters described above) to derive continuous variables that described the main gradients of adaptive and neutral genetic variation separately (Table S2). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the individuals in each grid cell to express the typical genomic characteristics of individuals in the cells.

### Prioritizations
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve both a proportion of the species' range (using amount-based targets) and a proportion of the species' intra-specific variation [using space-based targets and attribute spaces; _sensu_ @r465]. We treated the environmental and geographic surrogate variables as separate attribute spaces. We also treated the adaptive (if detected) and neutral variables as separate attribute spaces for each species. We set the demand points for each species and each attribute space using the values associated with the planning units. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

To understand the relationship between the surrogate-based targets used to generate a prioritization and the proportion of genetic variation it secures, we generated a collection of single-species prioritizations. For each species, we first generated two sets of prioritizations using increasing environmental and geographic-based targets (separately). Prioritizations were generated using surrogate-based targets ranging from `r (min(rapr.params.LST[[MODE]][['surrogacy.analysis']]$surrogate.target)*100)` % and `r (max(rapr.params.LST[[MODE]][['surrogacy.analysis']]$surrogate.target)*100)` % in increments of `r diff(rapr.params.LST[[MODE]][['surrogacy.analysis']]$surrogate.target[1:2])*100` % ($n = `r rapr.params.LST[[MODE]][['surrogacy.analysis']]$surrogate.optimal.replicates`$ per target). To determine if these surrogate-based prioritizations performed better then random, we then generated another two sets of prioritizations by randomly selecting planning units that were occupied by the species ($n = `r rapr.params.LST[[MODE]][['surrogacy.analysis']]$surrogate.random.replicates`$ per surrogate target). To account for variation in the number of selected planning units in the target-based prioritizations, the number of selected of planning units in the randomly generated prioritizations were generated following their distribution. Randomly generated prioritizations associated with negative values--because they secured only a very small amount of genetic variation--were replaced with zeros to facilitate statistical analysis. 

To determine if the use of surrogate-based targets can actually improve prioritizations in real-world conservation planning scenarios, we investigated the effectiveness of surrogate-based targets using three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritizations for each species. However, most large-scale prioritizations involve a comprehensive set of species. This property may lead to prioritizations that secure a greater proportion of intra-specific genetic variation for each species by returning solutions that preserve the same species in multiple communities. To test for this effect, our second scenario involved generating multi-species prioritizations. Finally, most large-scale prioritizations also explicitly consider opportunity cost to ensure prioritizations are cost-effective. Our third scenario involved generating multi-species prioritizations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritizations. We generated prioritizations using only amount-based targets, both amount- and surrogate-based targets, and both amount- and genetic-based targets. The purely amount-based prioritizations serve as our baseline--they represent prioritizations typically generated in conservation planning exercises. We used `r (rapr.params.LST[[MODE]][['single.species']]$amount.target*100)` % amount-based targets in all prioritizations to ensure an adequate proportion of habitat was secured for each species. We set surrogate- and genetic-based targets as `r (rapr.params.LST[[MODE]][['single.species']]$amount.target*100)` % to ensure that most of the respective variation was secured for each species. To accomodate variation in the high number of optimal solutions for the single-species amount-based prioritizations, we generated `r (rapr.params.LST[[MODE]][['single.species']]$amount.replicates)` replicate prioritizations. Since all other contexts were likely to be associated with few optimal solutions, we generated only `r rapr.params.LST[[MODE]][['multi.species']]$genetic.replicates` replicates.

## Statistical analysis
We investigated the effectiveness of prioritizations generated using surrogate-based targets using generalized linear mixed-effects models [GLMMs; using a logit link and the $\texttt{lme4}$ R package; @r469]. To determine  if the environmental and geographic variables were effective surrogates, we fit a full model to the proportion of genetic variation that the first collection of prioritizations secured using categorical variables to indicate the surrogate-based targets used to generate them, the type of genetic variation (adaptive or neutral), the method using the generate the prioritization (optimal or random), and the interactions between these variables. To determine if surrogate-based targets improved the effectiveness of prioritizations under various scenarios, we fit a full model to the proportion of genetic variation that second collection of prioritizations secured using categorical variables to indicate the scenario used to generate the prioritization (single-species, multi-species, multi-species with cost), the combination of targets used to generate the prioritizations (amount-based targtes, amount and surrogate-based targets, amount and genetic-based targets), the type of genetic variation measured (adaptive or neutral), and the interactions between them. All model contained intercept- and slope-level random effects to accommodate variation between species. Both full models were subject to a step-wise term deletion routine using $\chi^{2}$ tests to assess significance (terms were retained when $P < 0.05$). Finally, we conducted post-hoc analysis on each full model using Tukey contrasts with a Bonferroni correction [using the $\texttt{multcomp}$ R package; @r470].

## Results
<!-- Surrogacy analysis -->
Prioritizations generated using greater environmental and geographic surrogate-based targets were found to secure greater proportions of adaptive and neutral genetic variation (Figure 1). The proportion of genetic variation that these prioritizations secured was substantially more than the genetic variation secured by randomly generated prioritizations (${\chi^{2}}_{`r NA`,`r NA`} = `r  NA`$, $P < `r NA`$). Generally, prioritizations needed to capture at least `r NA` % of the environmental and geographic surrogates to secure 90 % of the species' genetic variation. Additionally, we detected differences in the effectiveness of the environmental and geographic surrogates (${\chi^{2}}_{`r NA`,`r NA`} = `r  NA`$, $P < `r NA`$). Increases in environmental surrogate-based targets resulted prioritizations that secured more adaptive genetic variation, than geographic surrogate-based targets for neutral variation. These results demonstrate that environmental and geographic variables are effective surrogates for genetic variation for these plant species.

<!-- Scenario analysis -->
The addition of environmental and geographic surrogate-based targets to amount-based targets was not found to substantially improve the effectiveness of prioritizations under any of the scenarios (Figure 2). No significant differences were detected between prioritizations generated using different combinations of targets (${\chi^{2}}_{`r NA`,`r NA`} = `r  NA`$, $P < `r NA`$), or under any of the different scenarios (${\chi^{2}}_{`r NA`,`r NA`} = `r NA`$, $P < `r NA`$). The only difference was that the inclusion of surrogate-based targets tended to result in prioritizations that secured slightly more adaptive than neutral genetic variation. Under the simplest of circumstances, the single-species amount-based prioritizations secured most of the species' adaptive (mean `r NA` $\pm$ `r NA`) and neutral (mean `r NA` $\pm$ `r NA`) genetic variation. These proportions were not significantly different to those associated with prioritizations generated using additional surrogate-based targets ($Z = `r NA`$, $P < `r NA`$) or genetic-based targets ($Z = `r NA`$, $P < `r NA`$). We found similar patterns in the multi-species scenario ${\chi^{2}}_{`r NA`,`r NA`} = `r NA`$, $P < `r NA`$) and the multi-species with opportunity cost scenario ${\chi^{2}}_{`r NA`,`r NA`} = `r NA`$, $P < `r NA`$). Although the prioritizations had similar levels of performance, there were differences in their spatial distributions (Figure 3). In particular, the inclusion of cost data in the multi-species prioritizations had the effect of concentrating conservation effort towards areas with higher elevation. Overall, these results suggest that while environmental and geographic surrogate variables are effective surrogates, they are not needed to secure most of the intra-specific genetic variation.

## Discussion
We aimed to investigate the effectiveness of environmental and geographic surrogates for genetic variation in conservation planning. Our study is the first to test this in a realistic context. We found that environmental and geographic variables are effective surrogates for genetic variation in conservation planning. But we also found that just using amount-based targets--typically used in conservation planning--result in prioritizations that secure most of the adaptive and neutral intra-specific genetic variation. Our results support existing guidelines on using a comprehensive set of species in conservation planning.

### Single + multi-species analysis
* summarize results

* compare + contrast with other papers

* implication of results

### Correlation analysis
* summarize results

* compare + contrast with other papers

* implication of results

### Limitations/Future directions
We wish to alert practitioners to several limitations associated with our analysis. First, the size of the planning units we used (23 $\times$ 25 km) is much larger than typically used in conservation planning scenarios (approx. 1--5 km^2). We used this resolution because the genetic data was collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis because the surrogate variables are spatially auto-correlated. Second, we used geographic distances as surrogates for connectivity between planning units. Although connectivity between areas can be modeled using isolation by resistance measures [@r473] derived from landscape data [eg. @r474], they are unlikely to be of use in broad-scale conservation exercises since they need to be tuned using species-specific parameters. Third, we used AFLP data to characterize the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @r480], our methodology would still have utilized only the main gradients of the genetic variation to generate prioritizations in a feasible amount of time. Overall, we are confident that the limitations associated with our study are relatively minor.

### Concluding remarks
The overarching aim of our study was to investigate the effectiveness of surrogates in conservation planning. We found that environmental and geographic surrogates are effective. However, they seem to be largely unnecessary in conservation planning scenarios because a comprehensive set of species can be used to achieve a similar result. Conservation planners are therefore urged to use a comprehensive set of species in prioritizations--one of the fundamental principals of conservation planning.

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References
