---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
knitr::opts_knit$set(root.dir=normalizePath('../..'))
```

```{r,include=FALSE}
# load .rda file
try(session::restore.session('data/final/results.rda'))

# set options
options(error=NULL)

```

<!--  title page -->
Short running title: Genetic surrogates in conservation \newline
Word count: XXXXX \newline
Number of references: XXXX \newline
Number of figures, table and text boxes: 4 \newline
\newpage


## Introduction
<!--  main text -->
Protected areas spearhead conservation efforts [@r440]. These places buffer species from anthropogenic impacts and provide places for species to persist. However, the resources available for conservation are limited. Thus protected areas need to be sited in places that achieve conservation goals for minimal cost. To ensure that protected areas are cost-effective, plans for protected areas (prioritizations) are often generated using decision support tools to identify near-optimal solutions. Additionally, because reserves cannot preserve the entire range for a given species, the plans for multiple reserves are often designed simultaneously to ensure that the network of protected areas can facilitate biological processes (eg. gene flow between populations, seasonal migration). For instance, \texttt{Marxan} [@r22] uses amount-based targets to identify protected area networks that preserve a suitable amount of habitat for each species (eg. 100 km $^2$ of land occupied by the species needs to be secured in a reserve) and weights to penalize fragmented solutions. However, conservation planning exercises typically assume that all individuals in the same species are equivalent.

Intra-specific genetic variation is an important component for species persistence. As a consequence, there has been increasing interest in understanding how to design prioritizations that preserve and cultivate this variation [@r67; @r3; @r61]. Although the strength of natural selection is a continuous force; broadly speaking, genetic variation can be classified as either adaptive or neutral [reviewed in @r480]. Adaptive genetic variation is associated with loci that (significantly) affect fitness. Typically, such "adaptive" loci are detected due to anomalous patterns among individuals [eg. @r456; @r487; @r480; @r490; but see @r489 for alternative methods]. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral genetic variation is associated with loci that do not (significantly) affect fitness. Neutral variation often reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can secure different genetic lineages and avoid the adverse effects of low genetic diversity [eg. inbreeding depression; @r67]. Thus protected areas networks should preserve both adaptive and neutral patterns of genetic variation for species of conservation interest [@r67; @r3]. Despite this, genetic data are not often used to inform prioritizations because they require considerable investment to obtain and analyze [@r61; but see @r5 and @r157 for developments on using genetic data in conservation planning]. Thus prioritizations may fail to capture species' intra-specific genetic variation, and ultimately fail to deliver long-term conservation outcomes.

Recently, conservation scientists have been investigating surrogates to generate prioritizations that adequately preserve genetic variation--but without actually needing to utilize genetic data to do this. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been used to guide reserve selection [eg. environmental conditions; @r247; @r2]. By preserving a more representative sampled of environmental conditions across the species' ranges, they aimed to generate prioritizations that preserved more adaptive variation. However, the effectiveness of this approach remains unverified. On the other hand, neutral genetic variation arises from a reduction in gene flow between individuals. Surrogates for neutral variation have been based on variables that predict the level of connectivity between planning units. For instance, the isolation by distance theory [IBD; @r259] predicts that populations further apart will have less connectivity, exchange less genetic material, and in turn have less genetic variation in common. Based on this theory, geographic distance-based surrogates have been found to improve prioritizations in an island network [@r200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain; eg. @r474]--the very systems commonly featured in real-world conservation planning exercises [eg. @r197]. Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates needs to be verified. Otherwise, conservation planners may waste precious resources on protecting places that provide little benefit to biodiversity.

Here we investigate the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation in the context of conservation planning. First, we aim to verify if environmental and geographic variables correlate with broad-scale patterns in adaptive and neutral variation (respectively). Although these variables may correlate with genetic variation, they may not be effective surrogates if the level of correlation is not high enough or due to spatial auto-correlative effects. Second, we aim to verify that environmental and geographic variables are effective surrogates for adaptive and neutral variation (respectively). We expect that prioritizations generated using surrogate-based targets will secure more genetic variation than prioritizations generated using only amount-based targets with the same number of planning units. Third, we aim to determine if the addition of surrogate-based targets to conservation planning exercises can result in more effective prioritizations. We expect that prioritizations generated using amount- and surrogate-based targets will sufficiently preserve the genetic variation of more species than those generated using only amount-based targets. If these surrogates can improve the intra-specific genetic variation captured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
We used species distribution and genomic data collected by the IntraBioDiv project in the European Alps [@r451; see @r452 and @r478 for further explanation of data collection methods]. This data set contains genetic data for a diverse range of species that were collected using a standardized sampling regime. Additionally, the data set covers a large geographic domain that spans across a range of attitudinal gradients. As a consequence, there is large environmental variation among the grid cell cells which is desirable to assess effectiveness of environmental variables as surrogates for adaptive genetic variation. Thus this data set was particularly well suited for our study. 

The IntroBioDiv project data set contains data for 27 alpine plant species that was collected using a 20' longitude by 21' latitude grid (approx. 22.3 km $\times$ 25 km; Figure S1). Project members visited every second grid cell, and if a species was detected in a cell, samples were collected from three individuals. They genotyped samples using amplified fragment length polymorphisms [AFLP; @r453], and constructed matrices denoting the presence/absence of polymorphisms at loci for each species (mean `r round(mean(sapply(spp.StructureData.LST, structurer:::n.loci)),2)` $\pm$ `r round(sd(sapply(spp.StructureData.LST, structurer:::n.loci)),1)` SD markers genotyped per species; see Table 1 in @r478 for the number markers for each species). Although AFLPs are not ideal as genomic representatives (cf. single nucleotide polymorphisms; SNPs), no multi-species genomic data set with similar properties currently exists. Thus the data set contains information describing the genomic properties of individuals for each species present in every second grid cell.

This data set has previously been used to address a range of different questions. Many studies have used this data set to explore patterns of adaptive [@r484; @r451; @r475; @r455; @r485] and neutral genetic variation [@r478; @r451; @r486]. However, to our knowledge, only a single study has used this data set to explore the effectiveness of a surrogate in the context of conservation planning. Taberlet _et al._ [-@r486] investigated whether prioritizations generated using the spatial distribution of a comprehensive set species (> 300) could adequately secure the intra-specific genetic variation for 27 species. They found that the prioritization based on species distributions was substantially different to that based on genomic data. Their results suggest that prioritizations based on species distribution data alone will be unable to secure intra-specific genetic variation.

### Landscape data
We examined the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation (respectively). We used the grid cells used to sample genetic data as planning units to develop prioritizations. We used only planning units that contained at least a single individual ($n = `r nrow(grid.DF)`$) to reduce computational burden. To examine the potential effects of cost, we calculated the total human population density inside each planning unit [obtained at 1 km resolution from the Global Rural-Urban Mapping Project; \texttt{GRUMP V1}; @r472] and used this to represent acquisition cost. 

To describe the geographic location of each grid cell, we projected the grid into an equi-distant coordinate system ($\texttt{ESRI:102031}$), calculated their centroids, and extracted their two-dimensional coordinates. Thus we constructed two-dimensional geographic space--wherein each planning unit occupies a single point--as a potential surrogate for neutral genetic variation.

To describe the environmental characteristics of each grid cell, we used broad-scale climatic variables. We obtained data for 19 bioclimatic variables across the extent of the study area [approx. 1 km resolution; @r33]. We used this data set because it is commonly to map habitat suitability in conservation planning. These layers and the planning units were projected into an equal-area coordinate system ($\texttt{ESRI:102014}$). To reduce dimensionality, the 19 bioclimatic variables were clipped to the planning units and subject to a principal components analysis (PCA; using \texttt{ArcMap} version 10.2.2; Table S1). The first `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components (PCs) cumulatively explained `r round(pca.DF[[4]][surrogate.params.LST[[MODE]]$number.components],1)` % of the variation and were used to generate `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` new layers. We calculated the average value for each PC layer in each planning unit, and used these values to characterize climatic variation among the planning units (Figure S2). Thus we constructed a three-dimensional environmental space--wherein each planning unit occupies a single point--as a potential surrogate for adaptive variation.

### Adaptive and neutral genetic data
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first needed to identify which of the sampled loci--if any--were under selection. We used two outlier detection methods to achieve this. The basic premise underpinning such methods is that neutral loci are expected to exhibit a specific level of variation among populations, and loci that deviate from this expectation should be under selection [reviewed in @r480]. The advantage of these methods--in contrast with environmental association analyses [XXX]--is that they do not use environmental data to identify loci under selection, which would have biased our analysis. We omitted the species \textit{`r missing.species`} from all subsequent analysis because it was only associated with one genetic lineage and thus not compatible with one of the outlier detection methods. Loci identified as outliers in both analyses were treated as adaptive, and the remaining loci were treated as neutral.

First, we used multinominal-Dirichlet models implemented in \texttt{BayeScan} [version 2.1; @r456] to identify loci under selection. We adopted a similar methodology to Bothwell _et al._ [-@r455] and applied it to each of the species in the data set. To avoid falsely classifying loci as adaptive, we grouped individuals into genetic lineages for each species (Figures SXXX--SXXX). We used the number of genetic lineages in each species previously identified by Alvarez _et al._ [-@r478]. For each species, we used \texttt{Structure} [version 2.3.4; using same parameters as @r478 per run with `r structure.params.LST[[MODE]]$numruns` runs per species] and \texttt{ClumPP} [version 1.1.2; @r459] to group individuals into populations. We then ran $\texttt{Bayescan}$ for each species using these groupings [1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations; @r456]. To avoid false-positives, we omitted loci where the global frequency of the minor allele was $\geq 0.1$ (following guidelines in the user manual) and assumed they were neutral. Additionally, we omitted individuals from this analysis when their population membership was uncertain (maximum membership probability $< `r structure.params.LST[[MODE]]$probability.threshold`$). We ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate [FDR $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

Second, we used PCAs to identify outlier loci using methods implemented in the \texttt{pcadapt R} package [version 2.2; @r487]. For each of the 26 species, we ran a PCA and extracted the minimum number of components needed to secure `r pcadapt.params.LST[[MODE]]$min.variance.explained` % of the variation in loci. We then computed Mahalanobis distances [@r398] to detect outlier loci ($P < `r pcadapt.params.LST[[MODE]]$alpha.level`$). To enable comparisons between the two outlier detection analyses, we used the same individuals in this analysis as used in the $BayeScan$ analysis.

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species (Figures S6--S32). For each species, we discarded the population groupings, and partitioned adaptive and neutral loci into separate data sets. We used non-metric multi-dimensional scaling [NMDS; implemented in the \texttt{vegan R} pacakge; @r458] using Gower distances [@r464; using the \texttt{cluster R} package to accommodate sparsity; @r457] to derive continuous variables that described the main gradients of adaptive and neutral genetic variation for each species (Table S2). To ensure that the ordinations described a sufficient amount of the genomic variation, we ran successive scaling analyses with increasing dimensionality until this was achieved (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for each analysis). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the individuals in each grid cell to express the typical genomic characteristics of individuals in the cells.

The previous analyses resulted in an adaptive (if detected) and neutral genetic space for each species. For a given species, each planning unit occupied by the species was associated with a multi-dimensional point in the species' neutral genetic space. Planning units that were closer together in this space are occupied by individuals with similar neutral genetic loci. Similarly, if the species was associated with adaptive loci, each planning unit occupied by the species was associated with a multi-dimensional point in the species' adaptive genetic space. By spreading out conservation effort across a given genetic space for a given species, prioritizations can secure more genetic variation [see @r465 for discussion on attribute spaces].

### Prioritizations and statistical analyses
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve both a proportion of the species' range (using amount-based targets) and a proportion of variation found across the species' range [using space-based targets and attribute spaces; _sensu_ @r465]. We treated the environmental and geographic surrogate variables as separate attribute spaces. We also treated the adaptive (if detected) and neutral variables as separate attribute spaces for each species. We set the demand points for each species and each attribute space using the values associated with the planning units. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. Prioritizations associated with negative values--because they secured only a very small amount of genetic variation--were replaced with zeros to facilitate statistical analysis. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

To address the first aim--to determine if the environmental and geographic variables correlate with adaptive and neutral genetic variation--we fit maximum likelihood population-effects (MLPE) models [implemented in the \texttt{ResistanceGA R} package; @r488]. We computed differences between the grid cells in terms of their environmental conditions and  geographic position. Additionally, for each species, we computed the differences between the grid cells in terms of the adaptive and neutral genetic characteristics of individuals that inhabited them (if detected). All differences were calculated using Euclidean distances. We then fit a set of MLPE models to the differences in the adaptive genetic characteristics among the grid cells and the differences in the environmental conditions for each species associated with adaptive genetic variation. We also fit a set of similar models using the differences in neutral genetic characteristics and the geographic distances for each species. For each model we fit an intercept-only null model, and tested surrogate variables whether they explained substantially more variation using $\chi^2$. We then applied Bonferroni corrections to each set of models (separately). Note that this analysis does not take into account spatial auto-correlation. The subsequent analyses explicitly control for this effect.

To address the second aim--to determine if the environmental and geographic variables are effect surrogates for adaptive and neutral genetic variation--we generated a collection of single-species prioritizations. For each species, we first generated two sets of prioritizations using increasing environmental and geographic-based targets (separately). Prioritizations were generated using the following surrogate-based targets: `r paste(rapr.params.LST[[MODE]]$surrogacy.analysis$surrogate.target,collapse=', ')`. We then create a set of null expectations with which to compare the surrogate-based prioritizations. These null expectations represented possible solutions obtained using conventional conservation planning methods that just use amount (eg. Marxan). For each species, for each target, we generated `r rapr.params.LST[[MODE]]$surrogacy.analysis$random.replicates` prioritizations by randomly selecting planning units occupied by the species. The number of randomly selected planning units was set to equal the number of planning units in the optimal prioritizations.

To address the third aim--to determine if the use of surrogate-based targets can actually improve prioritizations in real-world conservation planning scenarios--we investigated the effectiveness of surrogate-based targets using three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritizations for each species. However, most large-scale prioritizations involve a comprehensive set of species. By forcing solutions to secure individuals in multiple communities, this may result in prioritizations that secure a greater proportion of intra-specific genetic variation for each species. To test for this effect, our second scenario involved generating multi-species prioritizations. Finally, most large-scale prioritizations also explicitly consider opportunity cost to ensure that prioritizations are cost-effective. Our third scenario involved generating multi-species prioritizations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritizations using (1) only amount-based targets, (2) both amount- and surrogate-based targets, and (3) both amount- and genetic-based targets. The purely amount-based prioritizations served as our baseline--representing the prioritizations typically generated in conservation planning exercises. We used `r (rapr.params.LST[[MODE]]$scenario.analysis$amount.target*100)` % amount-based targets in all prioritizations to ensure an adequate proportion of habitat was secured for each species. Based on the previous analysis, we set surrogate-based targets as `r (rapr.params.LST[[MODE]]$scenario.analysis$surrogate.target*100)` % and genetic-based targets as `r (rapr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` % to ensure that most of the variation was secured for each species. To accommodate variation in the high number of optimal solutions for the single-species amount-based prioritizations, we generated `r (rapr.params.LST[[MODE]]$scenario.analysis$single.species.amount.replicates)` replicate prioritizations.

We then counted the number of species that were sufficiently represented in terms of their adaptive and neutral genetic variation (genetic variation secured $\geq$ `r (rapr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` %) for each prioritization. We used a global $G$ test to assess whether the proportion of species that met their genetic target in a given prioritization depended on the combination of targets used to generate the planning units (amount-based targets, amount and surrogate-based targets, amount and genetic-based targets), the planning scenario (single-species, multi-species, multi-species with cost), or the type of genetic variation measured (adaptive or neutral). We also conducted a post-hoc analysis involving pairwise $G$ tests [using the \texttt{RVAideMemoire R} package; @r483].

## Results
<!-- Surrogacy analysis -->
The environmental and geographic-based variables were effective surrogates for adaptive and genetic variation (Figure 1). The prioritizations generated using greater surrogate-based targets secured more genetic variation (XXXX). In addition, prioritizations generated using environmental targets often secured more adaptive genetic variation than prioritizations generated using geographic-based targets did neutral genetic variation (XXXX). Only particularly high geographic targets ($\geq 0.95$ %) were able to secure a moderate proportion of neutral variation ($\geq XXXX$ %). Overall, there was a non-linear relationship between the surrogate-based targets used to generate the prioritizations and the proportion of genetic variation they secured.

<!-- Scenario analysis -->
To determine if the addition of environmental and geographic targets could improve the effectiveness of prioritizations in a more realistic context, we generated prioritizations according to three different scenarios (Figure 2). The prioritizations generated under the single-species scenario showed similar selection frequencies across the study area--regardless of the combination of targets used to generate them. On the other hand, the prioritizations generated under the multi-species scenario showed different selection frequencies depending on the targets used to generate them. The addition of surrogate- or genetic-based targets resulted in prioritizations that contained substantially more planning units. The prioritizations generated under the multi-species with cost scenario had different spatial distributions to the previous two scenarios. The prioritizations generated under this third scenario concentrated conservation effort towards areas with higher elevation.

The inclusion of surrogate-based targets caused an improvement in the proportion of neutral variation secured in a prioritization (Figure 3). Specifically, the proportion of genetic variation secured by the prioritizations differed depending on the type of genetic variation measured (XXXX) and combination of targets used to generate the prioritizations (XXXX). No significant differences were found among the single-species, multi-species, and multi-species with cost contexts (XXXX).

## Discussion
<!-- introduction -->
We aimed to investigate the effectiveness of broad-scale environmental and geographic variables as surrogates for adaptive and neutral genetic variation in conservation planning. Our study is the first to address this hypothesis using moderately realistic planning scenarios. Overall, we found that environmental and geographic variables are effective surrogates for a few species when developing prioritizations. Furthermore, when developing prioritizations under various planning scenarios, we found that the addition of environmental and geographic targets resulted in prioritizations that sufficiently represented the genetic variation of a greater number of species. Our results suggest that while the environmental and geographic variables have little benefit for most species, they can make a large difference for some species. 

<!-- consequences for conservation planning -->
+ Typically, conservation planning exercises do not consider intra-specific variation when designing prioritizations. 
+ our results show that for most species, a sufficient proportion of their intra-specific variation can be secured using suitable amount-based targets alone.
+ in particular, we found that using suitable amount-based targets in a multi-species context with comprehensive set of target species resulted in a prioritization that adequately represented the adaptive genetic variation for nearly all the target species (panel c; Figure 3).
+ however, for some species, such a prioritization secures only a small proportion of their genetic variation.
+ we found that for some species, prioritizations generated using amount-based targets only secured a small amount of their neutral genetic variation (panels b, d, e; Figure 3).
+ in planning exercises where the decision maker does not have access to genetic data on their target species across the extent of the study area--ie. nearly all real world planning scenarios--the decision maker will not know which species are poorly represented in terms of their inta-specific variation.
+ the poorly represented species may be those of particular significant or they may not.
+ the decision maker has to weigh the increased costs of a prioritization that includes environmental and geographic surrogates against the risk that a prioritization that does not include these surrogates fails to represent species of conservation significance.
+ here we found that prioritizations generated using environmental and geographic surrogates have only minor increased costs (compare panels c and d; panels e and f; Figure 3).
+ overall, these results suggest that although surrogates are unnecessary for most species, they can be highly beneficial for a few species, the increased costs associated with using these surrogates is small, so generally using surrogates is a good idea.

<!-- comparison to previous studies that have investigated genetic studies -->
+ Previous studies on this data set have found correlations between environmental and genetic variation.
+ Yet, for most species, prioritizations generated using environmental surrogates did not secure substantially more genetic variation than randomly generated prioritizations with the same number of planning units.
+ Here we discuss some potential reasons for this disagreement.
+ 

+ This apparent disagreement can be attributed to multiple reasons.


+ These results highlight the importance of using prioritizations to assess the effectiveness of potential surrogates for conservation and not simply measuring the strength of correlation between potential surrogates and genetic variation.






<!-- limitations/future directions -->
We wish to alert our readers to several limitations associated with our analysis. First, the size of the planning units we used (23 $\times$ 25 km) is much larger than typically used in conservation planning scenarios (between 1--10 km^2). We used this resolution because the genetic data was collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis because this process would have  artificially introduced additional spatial auto-correlation into the dataset. Second, we used geographic distances as surrogates for connectivity between planning units. Although connectivity between areas can be modeled using isolation by resistance measures [@r473] derived from landscape data [eg. @r474], they are unlikely to be of use in broad-scale conservation exercises since they need to be tuned using species-specific parameters. Third, we used AFLP data to characterize the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @r480], our methodology would still have utilized only the main gradients of the genetic variation in order to generate prioritizations in a feasible amount of time. Overall, we are confident that the limitations associated with our study are relatively minor.

### Concluding remarks
The overarching aim of our study was to investigate the effectiveness of surrogates in conservation planning. We found that environmental and geographic surrogates are effective. However, they seem to be largely unnecessary in conservation planning scenarios because a comprehensive set of species can be used to achieve a similar result. Conservation planners are therefore urged to use a comprehensive set of species in prioritizations--one of the fundamental principals of conservation planning.

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References
