---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: references.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
knitr::opts_knit$set(root.dir=normalizePath('../..'))
```

```{r,include=FALSE}
# load .rda file
try(session::restore.session('data/final/results.rda'))

# set options
options(error=NULL)
```

<!--  title page -->
Short running title: Genetic surrogates in conservation \newline
Word count: XXXXX \newline
Number of references: XXXX \newline
Number of figures: XXXX \newline
Number of tables: XXXX \newline
Number of text boxes: XXXX \newline
\newpage


## Introduction
<!--  main text -->
Protected areas spearhead conservation efforts [@r440]. These places buffer species from anthropogenic impacts and provide places for species to persist. However, the resources available for conservation are limited. Thus protected areas need to be sited in places that achieve conservation goals for minimal cost. To ensure that the plans for protected areas (prioritizations) are cost-effective, they are often generated using decision support tools to identify near-optimal solutions. These prioritizations often contain plans for a network of multiple protected areas so they can preserve biological processes (eg. connectivity between populations). For instance, \texttt{Marxan} [@r22] uses amount-based targets to identify protected area networks that preserve a suitable amount of habitat for each species (eg. 100 km $^2$ of land occupied by the species needs to be secured in a reserve) and weights to penalize fragmented solutions (ie. the \textit{BLM} parameter). However, conservation planning exercises typically assume that all individuals in the same species are equivalent.

Intra-specific genetic variation is important for long-term species persistence. As a consequence, there has been increasing interest in designing prioritizations that preserve and cultivate this variation [@r67; @r3; @r61]. Although the strength of natural selection is a continuous force; broadly speaking, genetic variation can be classified as either adaptive or neutral [reviewed in @r480]. Adaptive genetic variation is associated with loci that (significantly) affect fitness. Typically, such "adaptive" loci are detected due to anomalous patterns among individuals [eg. @r456; @r487; @r480; @r490; but see @r489 for alternative methods]. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral genetic variation is associated with loci that do not (significantly) affect fitness. Neutral variation often reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can safeguard against the adverse effects of low genetic diversity [eg. inbreeding depression; @r67]. Thus protected area networks should aim to preserve both adaptive and neutral patterns of genetic variation for species of conservation interest [@r67; @r3]. Despite this, genetic data are not often used to inform conservation planning exercises because such data require considerable investment to obtain and analyze [@r61; but see @r5 and @r157 for developments on using genetic data in conservation planning]--especially since most large-scale exercises involve hundreds of species. Thus prioritizations may fail to secure a sufficient proportion of the intra-specific genetic variation of  species, and ultimately fail to deliver long-term conservation outcomes.

Recently, conservation scientists have been investigating the use of surrogate data to generate prioritizations that adequately preserve intra-specific genetic variation--but without actually needing to utilize genetic data to acheive this. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been used to guide reserve selection as surrogates for adaptive genetic variation [eg. environmental conditions; @r247; @r2]. By preserving a more representative sample of environmental conditions across the species' ranges, these researchers aimed to generate prioritizations that would preserve more adaptive variation. However, the effectiveness of this approach remains unverified. On the other hand, neutral genetic variation arises from a reduction in gene flow between individuals. Surrogates for neutral variation have been based on variables that predict the level of connectivity between planning units. For instance, the isolation by distance theory [IBD; @r259] predicts that populations further apart will have less connectivity, exchange less genetic material, and in turn have less genetic variation in common. Based on this theory, geographic distance-based surrogates have been found to improve prioritizations in an island network [@r200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain or habitat type; @r474; @r497]--the very systems commonly featured in real-world conservation planning exercises [eg. @r197]. Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates need to be verified. Otherwise, conservation planners may waste precious resources on protecting places that provide little benefit to biodiversity.

Here we investigate the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation in the context of conservation planning. First, we aim to verify if environmental and geographic variables correlate with broad-scale patterns in adaptive and neutral variation (respectively). Although these variables may correlate with genetic variation, they may not be effective surrogates if they have poor predictive ability or if these correlations are driven by spatial auto-correlation. Second, we aim to verify that environmental and geographic variables are effective surrogates for adaptive and neutral variation (respectively). We expect that prioritizations generated using surrogate-based targets will secure more genetic variation than prioritizations generated using only amount-based targets with the same number of planning units. Third, we aim to determine if the addition of surrogate-based targets to conservation planning exercises can result in more effective prioritizations. We expect that prioritizations generated using amount- and surrogate-based targets will sufficiently preserve the genetic variation of more species than those generated using only amount-based targets. Furthermore, we aim to test this hypothesis under several of different scenarios to see whether the additional constraints of real-world planning exercises exacerbate or nullify this effect. If these surrogates can improve the intra-specific genetic variation captured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
We used species distribution and genomic data collected by the international IntraBioDiv project in the European Alps [@r451; see @r452 and @r478 for further explanation of data collection methods]. This data set contains information for 27 alpine plant species (Figure 1a). It was collected using a $20^{\prime}$ longitude by $21^{\prime}$ latitude grid (approx. 22.3 km $\times$ 25 km; Figure S1). Project members visited every second grid cell, and if a species was detected in a cell, samples were collected from three individuals. They genotyped samples using amplified fragment length polymorphisms [AFLP; @r453], and constructed matrices denoting the presence/absence of polymorphisms at loci for each species (mean `r round(mean(sapply(spp.StructureData.LST, structurer:::n.loci)),2)` $\pm$ `r round(sd(sapply(spp.StructureData.LST, structurer:::n.loci)),1)` SD markers genotyped per species; see Table 1 in @r478 for the number markers for each species). Thus the data set contains information describing the genomic properties of individuals for each species present in every second grid cell.

We chose this data set for several reasons. First, conservation planning exercises typically consider tens to hundreds of species with different evolutionary histories and life histories. To ensure that our results were relevant for conservation planners, we required a data set with a diverse set of species and a comparable number of species. The IntraBioDiv data set contains genetic data for 27 species with different evolutionary histories and life-history strategies. Second, we required genetic data for all species to be collected using a standardized sampling scheme, and this data set used a grid to collect samples. Third, we required the study area to contain broad-scale environmental variation to test the effectiveness of environmental variables. This data set spans across large attitudinal gradients. Although this data set contains AFLPs  which not ideal as genomic representatives (cf. single nucleotide polymorphisms; SNPs), no multi-species genomic data set with similar properties currently exists. Thus this data set was well suited for our study.

This data set has previously been used to address a range of different questions. Many studies have used this data set to explore patterns of adaptive [@r484; @r451; @r475; @r455; @r485] and neutral genetic variation [@r478; @r451; @r486]. However, to our knowledge, only a single study has used this data set to explore the effectiveness of a surrogate in the context of conservation planning. Taberlet _et al._ [-@r486] investigated whether prioritizations generated using the spatial distribution of a comprehensive set species (> 300) could adequately secure the intra-specific genetic variation for 27 species. They found that the prioritization based on species distributions was substantially different to the prioritization based on genomic data. Their results suggest that prioritizations based on species distribution data alone will be unable to secure intra-specific genetic variation. However, whether or not environmental and geographic variables can augment prioritizations remains untested.

### Landscape data
We examined the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation (respectively). We utilized the sampling grid used by the IntraBioDiv project as planning units to develop prioritizations. We used only planning units that contained at least a single individual ($n = `r nrow(grid.DF)`$) to reduce computational burden. To examine the potential effects of variation in cost (see below), we calculated the total human population density inside each planning unit [obtained at 1 km$^2$ resolution from the Global Rural-Urban Mapping Project; \texttt{GRUMP V1}; @r472] and used this to represent acquisition cost (Figure 1b). 

To describe the geographic location of each planning unit, we projected the grid into an equi-distant coordinate system (Europe Equidistant Conic: $\texttt{ESRI:102031}$), calculated their centroids, and extracted their two-dimensional coordinates. Thus we constructed two-dimensional geographic space--wherein each planning unit occupies a single point--as a potential surrogate for neutral genetic variation. We expected planning units further apart from each other in this geographic space to contain individuals with increasingly different patterns in their neutral loci.

To describe the environmental characteristics of each planning unit, we used broad-scale climatic variation. We obtained data for 19 bioclimatic variables across the extent of the study area [at $1^{\prime\prime}$ resolution; @r33]. We used this data set because it is commonly to map habitat suitability in conservation planning [@r493]. These layers and the planning units were projected into an equal-area coordinate system (Europe Lambert Conformal Conic: $\texttt{ESRI:102014}$). To reduce dimensionality, the 19 bioclimatic variables were clipped to the planning units and subjected to a principal components analysis (PCA; using \texttt{ArcMap} version 10.2.2; Table S1). The first `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components (PCs) cumulatively explained `r round(pca.DF[[4]][surrogate.params.LST[[MODE]]$number.components],1)` % of the variation and were used to generate new layers. We calculated the average value for each PC layer in each planning unit, and used these values to characterize climatic variation among the planning units (Figure S2). Thus we constructed a three-dimensional environmental space--wherein each planning unit occupies a single point--as a potential surrogate for adaptive variation. We expect planning units further apart from each other in this environmental space to contain individuals with increasingly different patterns in their adaptive loci.

### Adaptive and neutral genetic data
To investigate the effectiveness of our supposed surrogates for adaptive and neutral genetic variation, we first needed to identify which of the sampled loci--if any--were adaptive. We used two outlier detection methods to achieve this [an individual- and a population-level method as recommended by @r495]. The basic premise underpinning such methods is that neutral loci are expected to exhibit a specific level of variation among populations, and loci that deviate from this expectation should be under selection [reviewed in @r480]. The advantage of these methods--in contrast with environmental association analyses [@r489]--is that they do not use environmental data to identify loci under selection, which would have biased our analysis. Loci identified as outliers in both analyses were treated as adaptive and the rest were treated as neutral. To avoid false-positives, we omitted loci from both outlier detection methods where the global frequency of the minor allele was $> 0.1$ and assumed they were neutral.

The first outlier detection method we used involved fitting multinominal-Dirichlet models implemented in \texttt{BayeScan} [version 2.1; @r456]. We adopted a similar methodology to Bothwell _et al._ [-@r455] and applied it to each of the species in the data set. Following their methodology, we first grouped conspecifics into genetic lineages to avoid falsely classifying loci as adaptive. To complete this first step, for each species, we fit the admixture and correlated alleles model implemented in \texttt{Structure} [version 2.3.4; r@461; r@462] using the number of populations for each species previously determined by [@r478; `structure.params.LST[[MODE]]$numruns` replicates per K; `r structure.params.LST[[MODE]]$admburnin` admixture burnin iterations; `r structure.params.LST[[MODE]]$burnin` burnin iterations; `r structure.params.LST[[MODE]]$numreps` iterations used for inference]. We used \texttt{ClumPP} [version 1.1.2; @r459] to combine replicate runs to group individuals into populations (greedy algorithm using the $G^{\prime}$ static; `r clumpp.params.LST[[MODE]]$repeats` iterations). We then ran $\texttt{Bayescan}$ [@r456] for each species using these populations [1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations]. Additionally, we omitted individuals from the \texttt{Bayescan} analysis if their population membership was uncertain (maximum membership probability $< `r structure.params.LST[[MODE]]$probability.threshold`$). We ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate [$q \leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

The second outlier detection method involved fitting PCAs to identify outlier loci [using methods implemented in the \texttt{pcadapt R} package; version 3.0; @r487; @r496]. To enable comparisons between the two outlier detection analyses, we used the same individuals in this analysis as used in the \texttt{BayeScan} analysis. For each of the 27 species, we first imputed missing data to permit further analysis. Specifically, the missing value for a given locus for a given individual was replaced with the average frequency of that locus among all conspecifics. Next, we ran a PCA and extracted the minimum number of components needed to secure the over-arching population level variation among the loci (`r pcadapt.params.LST[[MODE]]$min.variance.explained*10` % of the variation in loci). We then used Mahalanobis distances [@r398] to compute _P_ values and then _q_-values [using the \texttt{qvalue R} package; @r491]. Outlier loci were detected using a suitable false discovery rate ($q < `r pcadapt.params.LST[[MODE]]$alpha.level`$). 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species. For each species, we discarded the population groupings, and partitioned adaptive and neutral loci into separate matrices. We applied separate non-metric multi-dimensional scaling analyses to each matrix [NMDS; implemented in the \texttt{vegan R} package; @r458] using Gower distances [@r464; using the \texttt{cluster R} package to accommodate sparsity; @r457] to derive continuous variables that described the main gradients of adaptive and neutral genetic variation (separately) for each species (Table S2). To ensure that the ordinations described a sufficient amount of the genomic variation, we ran successive scaling analyses with increasing dimensionality until a sufficient stress value was obtained (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for each analysis). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the individuals in each grid cell to express the typical genomic characteristics of individuals in the cells.

The previous set of analyses resulted in an adaptive (if detected) and neutral genetic space for each species. For a given species, each planning unit occupied by the species was associated with a multi-dimensional point in the species' neutral genetic space. Planning units that were closer together in this space are occupied by individuals with similar polymorphisms in their neutral loci. Similarly, if the species was associated with adaptive loci, each planning unit occupied by the species was associated with a multi-dimensional point in the species' adaptive genetic space. By spreading out conservation effort across a given genetic space, and in turn securing planning units occupied by individuals with increasingly different polymorphisms, prioritizations can secure more genetic variation [see @r465 for discussion on attribute spaces].

## Prioritization method
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve both a proportion of the species' range (using amount-based targets) and a proportion of variation found across the species' range [using space-based targets and attribute spaces; _sensu_ @r465]. We treated the environmental and geographic surrogate variables as separate attribute spaces. We also treated the adaptive (if detected) and neutral variables as separate attribute spaces for each species. We set the demand points for each species and each attribute space using the values associated with the planning units. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. Prioritizations associated with negative values--because they secured only a very small amount of genetic variation--were replaced with zeros to facilitate statistical analysis. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

## Statistical analyses
To address the first aim--to determine if the environmental and geographic variables correlate with spatial patterns of adaptive and neutral genetic variation--we computed dissimilarity matrices between the planning units, and tested for correlations between matrices based on surrogate and genetic data. For each species, we computed two dissimilarity matrices between the planning units occupied by the species in terms of their environmental conditions (their coordinates in the environmental space) and geographic position (their coordinates in the geographic space). We then computed another dissimilarity matrix between the planning units using their values in the species' neutral genetic space. If the species was associated with adaptive loci, we also computed another dissimilarity matrix between the planning units using the species' adaptive genetic space. All dissimilarity matrices calculated using Euclidean distances.

We then fit maximum likelihood population-effects (MLPE) models [@r494; implemented in the \texttt{ResistanceGA R} package; @r488]. These models explicitly accommodate the structure of dissimilarity matrices using random effects. For each species, we fit a MLPE model to correlate the species' dissimilarity matrices based on geographic position and neutral genetic variation. Additionally, if the species was associated with adaptive loci, we also fit a MLPE model to correlate the species' dissimilarity matrices calculated using environmental conditions and adaptive genetic variation. All predictor variables were z-transformed prior to model fitting to improve convergence. To test if the surrogate variables explained a substantial amount of genetic variation, we conducted $\chi^2$ tests between each model and its corresponding null model, and applied Bonferroni corrections. 

To address the second aim--to determine if the environmental and geographic variables are effective surrogates for adaptive and neutral genetic variation--we generated a collection of single-species prioritizations and computed the proportion of intra-specific genetic variation they secured. For each species, we generated `r rapr.params.LST[[MODE]]$surrogacy.analysis$replicates` prioritizations by randomly selecting different combinations of planning units that were occupied by the species. We then computed the proportion of adaptive genetic variation and environmental variation secured in each prioritization. We then generated another random set of prioritizations, and computed the proportion of neutral genetic variation and geographic variation secured in each prioritization.

We fit two full generalized linear models (GLM) with a logit links. The first model was fit to the proportion of adaptive genetic variation secured in a prioritization, and contained a continuous predictor variable measuring the proportion of environmental variation also secured in the prioritization, a categorical variable indicating the species for which the prioritization was generated, and an interaction term between these predictor variables. The second model was fit to the proportion of neutral genetic variation secured in a prioritization, and contained a continuous predictor variable assessing the proportion of geographic variation also secured in the prioritization. Similar to the previous model, this model also contained a predictor variable indicating the species for which the prioritizations were generated, and an interaction term. We subjected these models to a backwards step-wise term deletion routine to assess term significance. To gauge the performance of these surrogates for each species, we fit a suite of models correlating the proportion of adaptive and neutral genetic variation held in each prioritization and the environmental and geographic variation they also secured (respectively). We then computed the Cragg and Uhler's pseudo-R$^2$ value for each model [using the \texttt{pscl} R package; @r492].

To address the third aim--to determine if the use of surrogate-based targets can actually improve prioritizations--we investigated the effectiveness of surrogate-based targets under three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritizations for each species. However, most large-scale prioritizations involve a comprehensive set of species. By forcing solutions to secure individuals in multiple communities, this may result in prioritizations that secure a greater proportion of intra-specific genetic variation for each species. To test for this effect, our second scenario involved generating multi-species prioritizations. Finally, most large-scale prioritizations also explicitly consider opportunity cost to ensure that prioritizations are cost-effective. Our third scenario involved generating multi-species prioritizations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritizations using (1) only amount-based targets, (2) both amount- and surrogate-based targets, and (3) both amount- and genetic-based targets. The purely amount-based prioritizations served as our baseline--representing the prioritizations typically generated in conservation planning exercises (eg. by \texttt{Marxan}). We used `r (rapr.params.LST[[MODE]]$scenario.analysis$amount.target*100)` % amount-based targets in all prioritizations to ensure an adequate proportion of habitat was secured for each species. Based on the results from the analyses used to address our second aim, we set surrogate-based targets as `r (rapr.params.LST[[MODE]]$scenario.analysis$surrogate.target*100)` % and genetic-based targets as `r (rapr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` % to ensure that most of the genetic variation was secured for each species. To accommodate the fact that there were a large number of optimal solutions for the single-species amount-based prioritizations, we generated `r (rapr.params.LST[[MODE]]$scenario.analysis$single.species.amount.replicates)` replicate prioritizations. We computed the proportion of adaptive (if present) and neutral genetic variation secured for each species in the prioritizations.

We investigated the benefits of using surrogate-based targets in prioritizations using generalized linear mixed-effects models fit with logit liknks [GLMMs; using the \texttt{lme4 R} package; @469]. We fit a full model to the proportion of genetic variation secured in a given prioritization. This model contained categorical predictor variables to indicate the combination of targets used to generate the prioritizations (amount-only, amount and surrogate targets, and amount and genetic targets), the scenario that was used to generate the prioritization (single-species, multi-species, and multi-species with cost), the type of genetic variation measured (adaptive and neutral), and all interactions between these variables. This model accommodated prioritizations generated for the same species using a random intercept term. We subject the full model to a backwards step-wise term deletion routine to assess term significance. We conducted a post-hoc analysis on the minimal adequate model using Tukey contrasts with a Bonferroni correction [using the \texttt{multcomp R} package; @r470].

## Results
<!-- summary of methods -->
We successfully mapped the main spatial patterns of adaptive (if present) and neutral variation for each of the 26 species. The outlier detection methods detected adaptive genetic variation in `r as.character(as.english(length(env.surrogacy.LST)))` species. Generally, the two outlier detection methods gave fairly consistent results (mean `r round(mean(spp.outlier.methods.congruence, na.rm=TRUE),2)` % loci per species $\pm$ `r round(sd(spp.outlier.methods.congruence, na.rm=TRUE),2)` SD). Of the species that were associated with adaptive genetic variation, only a few loci were detected as under selection (mean `r round(mean(spp.outlier.loci.props, na.rm=TRUE),2)` % loci per species $\pm$ `r round(sd(spp.outlier.loci.props, na.rm=TRUE),2)` SD). After identifying the loci under selection, we used NMDS analyses to identify the main gradients of adaptive (if detected) and neutral variation for each species. For all species in which we detected adaptive loci, only a couple of continuous dimensions were needed to sufficiently describe their patterns of adaptive genetic variation (Table S2). On the other hand, a few additional continuous dimensions were occasionally needed to describe their patterns of neutral genetic variation (minimum $K = `r min(neutral.k)`$; mean $K = `r round(mean(neutral.k),2)`$; max $K = `r max(neutral.k)`$; Table S2). The ordinations resulting from the NMDS analyses were used to construct an adaptive (if present) and neutral genetic space for each species. The spatial distribution of these genetic spaces (Figures S29--S54) generally show patterns of spatial auto-correlation--especially the adaptive genetic spaces (eg. _Cerastium uniflorum_ and _Dryas octopetala_). Additionally, several species show east-west patterns in their intra-specific genetic variation (eg. _Peucedanum ostruthium_ and _Phyteuma betonicifolium_). 

<!-- correlation analysis -->
The dissimilarity matrices generated using the environmental and geographic variables correlated with those matrices based on the genetic characteristics for most species (Table 1). Specifically, environmental variation was found to significantly correlate with adaptive genetic variation for `r as.character(as.english(sum(correlation.model.summary.DF$Environmental.corr.P<0.05,na.rm=TRUE)))` species (`r round((sum(correlation.model.summary.DF$Environmental.corr.P<0.05,na.rm=TRUE)/length(na.omit(correlation.model.summary.DF$Environmental.corr.P)))*100,2)` % associated with loci under selection; _P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` SD for significant models). Additionally, geographic distances between planning units were found to significantly correlate with neutral genetic variation among planning units for `r sum(correlation.model.summary.DF$Geographic.corr.P<0.05,na.rm=TRUE)` species (`r round((sum(correlation.model.summary.DF$Geographic.corr.P<0.05)/length(na.omit(correlation.model.summary.DF$Geographic.corr.P)))*100,2)` % of all the species investigated; _P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` SD for significant models). These results suggest that prioritizations with different environmental characteristics tend to have individuals with different adaptive genetic characteristics. Similarly, planning units that are further apart tend to have individuals with different neutral genetic characteristics. After verifying that the geographic and environmental variables generally correlated with genetic variation, our next step was to determine how effective these variables were as surrogates in the context of conservation planning.

<!-- surrogacy analysis -->
The environmental and geographic variables were moderately effective surrogates for genetic variation in most species (Figure 2). The relationship between the proportion of adaptive genetic variation secured in a prioritization and the proportion of environment variation it also secured was found to depend on the species for which the prioritization was generated (${\chi^2}_{`r env.surrogacy.AOV$df[[4]]`} = {`r round(env.surrogacy.AOV$Resid..Dev[[4]],2)`}; P `r pvalString(env.surrogacy.AOV$Pr..Chi.[[4]])`$). The slope for each species were all significantly different to zero (minimum \textit{z} = `r round(max(env.surrogate.spp.DF$statistic),2)`; maximum $P `r  pvalString(max(env.surrogate.spp.DF$p.value))`$; Table S3). The explanatory power of the environmental surrogates were also generally quite high for most species (mean `r round(mean(env.surrogate.spp.DF$r2CU),2)` $R^2$ $\pm$ `r round(sd(env.surrogate.spp.DF$r2CU),2)` SD; Figure S56). The nature of these relationships suggest that most of the adaptive variation for a given species can be secured in a prioritization using moderately high environmental-based targets ($> 80$ %). Similarly, the proportion of neutral genetic variation in a prioritization was affected by the interaction between species and the proportion of neutral variation secured in the prioritization (${\chi^2}_{`r geo.surrogacy.AOV$df[[4]]`} = `r round(geo.surrogacy.AOV$Resid..Dev[[4]],2)`$; $P$ `r pvalString(geo.surrogacy.AOV$Pr..Chi.[[4]])`). The slope for each species were also significantly different to zero (minimum \textit{z} = `r round(min(geo.surrogate.spp.DF$statistic),2)`; maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; Table S3). Generally, the explanatory power of geographic variables were higher than that of environmental variables (mean `r round(mean(geo.surrogate.spp.DF$r2CU),2)` $R^2 \pm$ `r round(sd(geo.surrogate.spp.DF$r2CU),2)` SD; Figure S57). Compared to the environmental variables, the slopes of these relationships suggest that very high geographic-based targets are required to generate prioritizations that secure a sufficient proportion of neutral genetic variation for most species ($> 95$ %). Additionally, all species were associated with prioritizations that secured a large proportion of environmental or geographic variation (> 70 %), yet preserved only a very small proportion of their intra-specific genetic variation (< 10 %; Figures S55 and S56; such as _Gypsophila repens_ and _Luzula alpinopilosa_). These results suggest while prioritizations generated using environmental and geographic targets tend to secure more genetic variation, prioritizations generated using only these targets can sometimes secure only a small fraction of intra-specific genetic variation.

<!-- scenario analysis -->
To determine if the addition of environmental and geographic targets could improve the effectiveness of prioritizations in a more realistic context, we generated prioritizations according to three scenarios using different combinations of targets (Figure 3). We then computed the proportion of adaptive and neutral genetic variation secured in each prioritization (Figure 4). The proportion of genetic variation secured in a prioritization was found to depend on the combination of targets used to generate the prioritization (${\chi^2}_{`r scenario.ANOVA.4.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.4.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.ANOVA.4.DF$Pr..Chisq.[[2]])`)$. Specifically, the amount-based prioritizations tended to secure less variation than the prioritizations generated using additional surrogate (`r round(plogis(scenario.GLHT.2.DF$estimate[[1]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.2.DF$std.error[[1]])*100,2)` SE ; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[1]],2)`$; $P `r pvalString(scenario.GLHT.2.DF$p.value[[1]])`$) or genetic variables (`r round(plogis(scenario.GLHT.2.DF$estimate[[2]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.2.DF$std.error[[2]])*100,2)` SE; `$Z_1 = `r round(scenario.GLHT.2.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.GLHT.2.DF$p.value[[2]])`$). No evidence was found to suggest that the prioritizations that included additional surrogate- or genetic-based targets secured a different proportion of genetic variation (`r round(plogis(scenario.GLHT.2.DF$statistic[[3]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.2.DF$std.error[[3]])*100,2)` SE; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[3]],2)`$; $P = `r pvalString(scenario.GLHT.2.DF$p.value[[3]])`$). We also found that the prioritizations tended to secure more adaptive than neutral genetic variation (`r round(plogis(scenario.GLHT.3.DF$statistic[[1]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.3.DF$std.error[[1]])*100,2)` SE; ${\chi^2}_{`r scenario.ANOVA.3.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.3.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.ANOVA.3.DF$Pr..Chisq.[[2]])`$). We also found significant differences in the average proportion of genetic variation secured by prioritizations in each of the scenarios (${\chi^2}_{`r scenario.ANOVA.2.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.2.DF$statistic[[2]],2)`$; $P$ `r pvalString(scenario.ANOVA.2.DF$Pr..Chisq.[[2]])`). Specifically, both the multi-species (`r round(plogis(scenario.GLHT.1.DF$estimate[[1]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.1.DF$std.error[[1]])*100,2)` SE; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[1]],2)`$; $P = `r pvalString(scenario.GLHT.1.DF$p.value[[1]])`$) and multi-species with cost (`r round(plogis(scenario.GLHT.1.DF$estimate[[2]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.1.DF$std.error[[2]])*100,2)` SE; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.GLHT.1.DF$p.value[[2]])`$) scenarios were found to secure more genetic variation than the single species scenario. Neither the multi-species scenarios were found to secure a different proportion of genetic variation (`r round(plogis(scenario.GLHT.1.DF$estimate[[3]])*100,2)` % mean difference $\pm$ `r round(plogis(scenario.GLHT.1.DF$std.error[[3]])*100,2)` SE; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[3]],2)`$; $P `r pvalString(scenario.GLHT.1.DF$p.value[[3]])`$). These results suggest that surrogate-based targets can substantially improve the proportion of intra-specific genetic variation secured in a prioritization for most species.

## Discussion
* introduction
We aimed  to investigate the effectiveness of broad-scale environmental and geographic variables as surrogates for adaptive and neutral genetic variation in conservation planning. Our study is the first to address this hypothesis using moderately realistic planning scenarios. Overall, we found that the spatial patterns of environmental and geographic variables correlate with those of adaptive and neutral patterns for most species. Furthermore, we found that the addition of environmental and geographic targets to the planning process resulted in prioritizations that generally secured a greater proportion of adaptive and neutral genetic variation. 

* consequences for conservation planning


* outlier detection methods


* mapping genetic variation

* limitations/future directions
We wish to alert our readers to several limitations associated with our analysis. First, the size of the planning units we used (23 $\times$ 25 km) is much larger than typically used in conservation planning scenarios (between 1--10 km^2). We used this resolution because the genetic data were collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis because this process would have artificially introduced additional spatial auto-correlation into the dataset. Second, we used geographic distances as surrogates for connectivity between planning units. Although connectivity between areas can be modeled using isolation by resistance measures [@r473] derived from landscape data [eg. @r474], they are unlikely to be of use in broad-scale conservation exercises since they need to be tuned using species-specific parameters. Third, we used AFLP data to characterize the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @r480], our methodology would still have utilized only the main gradients of the genetic variation in order to generate prioritizations in a feasible amount of time. Overall, we are confident that the limitations associated with our study are relatively minor.

* Concluding remarks
The overarching aim of our study was to investigate the effectiveness of surrogates in conservation planning. We found that environmental and geographic are generally effective surrogates for most species. Although prioritizations generated using amount-only targets tended to secure most of the adaptive variation for most species, the addition of surrogate targets to the planning process substantially improved the proportion of intra-specific neutral genetic variation. Conservation planners are therefore urged to include these surrogates in conservation planning exercises.

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References
