---
title: "Environmental and geographic variables are effective surrogates for genetic variation in conservation planning"
author:
  - name: Jeffrey O. Hanson
    affiliation: a,1
  - name: Jonathan R. Rhodes
    affiliation: b
  - name: Cynthia Riginos
    affiliation: a
  - name: Richard A. Fuller
    affiliation: a
address:
  - code: a
    address: School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  - code: b
    address: School of Earth and Environmental Sciences, The University of Queensland, Brisbane, QLD, Australia
corresponding_author:
  - code: 1
  - text: "To whom correspondence should be addressed. E-mail: jeffrey.hanson@uqconnect.edu.au"
author_contributions: |
  JOH performed the analyses. All authors developed the study and drafted the manuscript.
lead_author_surname: "Hanson"
abstract: |
  Protected areas buffer species from anthropogenic threats and provide places for the processes that generate and maintain biodiversity to continue. However, genetic variation, the raw material for evolution, is difficult to capture in conservation planning, not least because genetic data require considerable resources to obtain and analyze. Here we show that freely available environmental and geographic distance variables are highly effective surrogates in conservation planning for representing adaptive and neutral intra-specific genetic variation. We obtained occurrence and genetic data from the IntraBioDiv project for 27 alpine plant species collected over the European Alps using a gridded sampling scheme. For each species, we identified loci that were potentially under selection using outlier loci methods, and mapped their main gradients of adaptive and neutral genetic variation across the grid cells. We then used the cells as planning units to prioritize protected area acquisitions. First, we verified that the spatial patterns of environmental and geographic variation were correlated, respectively, with adaptive and neutral genetic variation. Second, we showed that these surrogates can predict the proportion of genetic variation secured in randomly generated solutions. Finally, we discovered that solutions based only on surrogate information secured substantial amounts of adaptive and neutral genetic variation. Our work paves the way for widespread integration of surrogates for genetic variation into conservation planning.
significance: |
  To protect biodiversity for the long term, nature reserves and other protected areas need to represent a broad range of different genetic types. However, genetic data are expensive and time consuming to obtain. Here we show that freely available environmental and geographic variables can be used as effective surrogates for genetic data in conservation planning. This means that conservation planners can, with some confidence, design protected area systems to represent intra-specific genetic diversity without investing in expensive programmes to obtain and analyze genetic data.
acknowledgements: |
  JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship.
keywords:
  - conservation
  - biodiversity
  - AFLP
  - genetic diversity
pnas_type: pnasresearcharticle
bibliography: references.bib
csl: reference-style.csl
output:
  rticles::pnas_article:
    includes:
      in_header: [preamble.tex]
---

<!-- NOTES -->
<!-- 250 word limit for abstract -->
<!-- 120 word limit for significance statement -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
knitr::opts_knit$set(root.dir=normalizePath('../..'))
```

```{r,include=FALSE}
# load .rda file
checkpoint::checkpoint('2016-11-26', R.version='3.3.2', scanForPackages=FALSE)
try(session::restore.session('data/final/results.rda'))

# download basemap
data(countriesHigh)
countries.FPLY <- countriesHigh[
  countriesHigh$ADMIN %in% c(
    'Italy', 'Switzerland', 'France', 'Austria',
    'Germany', 'Slovenia', 'Croatia', 'Hungary',
    'Monaco', 'Germany', 'Slovakia', 'Czech Republic'
  )
,] %>% spFortify
grid.FPLY <- spFortify(grid.PLY)
grid.sub.FPLY <- spFortify(grid.sub.PLY)

# define ggplot2 color wheel function
gg_color_hue <- function(n) {
  hues <- seq(15, 375, length=n+1)
  hcl(h=hues, l=65, c=100)[1:n]
}
```

<!-- Introduction -->
\dropcap{P}rotected areas spearhead conservation efforts ([@r440]). They buffer species from anthropogenic impacts, providing places for them to persist. Since the resources available for conservation are limited, protected areas need to be sited in places that fulfill conservation objectives for minimal cost ([@r255]). To achieve this, conservation planning exercises often generate plans for entire networks of protected areas (prioritizations) to allocate resources cost-effectively and preserve broad-scale biodiversity processes ([@r9]). Intra-specific genetic variation is fundamental to long-term species persistence ([@r165; @r504]), and as a consequence, there has been increasing interest in designing prioritizations that represent this aspect of biodiversity ([@r67; @r3; @r499; @r61; @r503]).

Although the strength of natural selection is a continuous force, genetic variation is often classified as adaptive or neutral (reviewed in [@r480]). Adaptive genetic variation is associated with loci that significantly affect fitness. Typically, such "adaptive" loci are detected due to anomalous patterns among individuals (eg. [@r456; @r487]), though phenotypic-genetic associations are more likely to identify functionally important genomic regions for individual species (eg. [@r500]). By representing the full range of adaptive variation, protected areas can help ensure that populations with particularly beneficial adaptations are not lost ([@r3; @r43]). In contrast, neutral genetic variation is associated with loci that do not significantly affect fitness, but instead reflect the evolutionary history of different populations. By representing the full range of neutral variation, protected areas can safeguard against the adverse effects of low genetic diversity ([@r67]). Thus optimally sited protected area networks would aim to represent patterns of both adaptive and neutral genetic variation ([@r67; @r3; @r43]).

Recently, the use of surrogate data has been proposed to generate prioritizations that capture intra-specific genetic variation ([@r2; @r200])---without needing to utilize genetic data to achieve this. Since adaptation is ultimately driven by selection pressures, environmental variables have been proposed as surrogates for adaptive genetic variation (eg. [@r202; @r2]). Specifically, by conserving individuals in a broad range of environmental conditions, we might expect to capture individuals with a diverse set of local adaptations, and overall capture a large proportion of the adaptive genetic variation present in the species. However, the effectiveness of this approach remains unverified. Neutral genetic variation arises from a reduction in gene flow between populations. Over the last few years, the field of phylogeography has predominantly focused on describing neutral genetic diversity, and so the landscape factors that affect neutral variation are relatively well understood (reviewed in [@r501]). Potential surrogates for neutral genetic variation have been based on variables that predict the level of connectivity between different areas. For instance, according the isolation by distance model (IBD; [@r259]), populations located further apart are predicted experience less gene flow, and in turn, share fewer neutral loci. Based on this idea, spreading conservation priorities evenly across the geographic distribution of an island network has been found to capture more neutral genetic variation ([@r200]). However this remains untested in spatially contiguous systems where connectivity is complicated by many additional factors (eg. land use; [@r497])---the typical situation in most conservation planning exercises.

Here we determine whether environmental and geographic surrogates capture adaptive and neutral genetic variation respectively in the context of conservation planning. We use distribution and genomic data for 27 alpine plant species in the European Alps that was obtained by the IntraBioDiv project ([@r451]). The data was collected following a gridded sampling scheme which we adopted as planning units for this study. We show that for most species the spatial patterns of adaptive and neutral genetic variation correlated with environmental and geographic variables. We also show that for most species the level of association is strong enough to be operational for conservation planning. Finally, we demonstrate that using these surrogates results in prioritizations that secure a substantial proportion of intra-specific genetic variation.

Results {.unnumbered}
=====================

<!-- summary of methods -->
We detected putatively adaptive genetic variation in `r as.character(as.english(length(env.surrogacy.LST)))` of the 27 plant species. We used two outlier detection methods---that solely used genetic data and did not use environmental data---to identify loci under selection. These methods returned reasonably consistent results (mean `r round(mean(spp.outlier.methods.congruence, na.rm=TRUE),2)` % loci per species assigned the same classification $\pm$ `r round(sd(spp.outlier.methods.congruence, na.rm=TRUE),2)` SD; Table S1). Of the species that were associated with adaptive genetic variation, only a small proportion of loci were classified as being adaptive (mean `r round(mean(spp.outlier.loci.props, na.rm=TRUE),2)` % loci per species $\pm$ `r round(sd(spp.outlier.loci.props, na.rm=TRUE),2)` SD). After identifying the loci showing strong signals of selection, we used non-metric multi-dimensional scaling (NMDS) analyses to identify the main gradients of adaptive (if detected) and neutral variation for each species. Generally, only a small number of continuous dimensions was needed to sufficiently describe their patterns of adaptive ($K = `r unique(na.omit(adaptive.k))`$ for all species; Table S2) and neutral genetic variation ($`r min(neutral.k)` \leq K \leq `r max(neutral.k)`$ for all species; Table S1). The resulting ordinations were used to construct an adaptive (if detected) and neutral genetic space for each species. The spatial distribution of these genetic spaces (Figures S1--S27) generally showed substantial spatial auto-correlation (eg. _Cerastium uniflorum_, Figure S34; _Dryas octopetala_, Figure S36), suggesting that the environmental and geographic variables have potential to be effective surrogates for genetic variation.

<!-- correlation analysis -->
We verified that the spatial patterns in environmental variation correlated with the broad-scale patterns in adaptive genetic variation for each species. We also verified that there was a correlation between neutral genetic variation and variation in geographic position. For each species, we constructed dissimilarity matrices expressing differences between the planning units where individuals were detected based on the units' (i) environmental characteristics, (ii) geographic position, and also the (iii) adaptive (if adaptive loci detected), and (iv) neutral genetic characteristics of the individuals found inside them. The spatial patterns of environmental variation were significantly correlated with the pattern of adaptive genetic variation for `r as.character(as.english(sum(correlation.model.summary.DF$Environmental.corr.P<0.05,na.rm=TRUE)))` of the `r as.character(as.english(sum(!is.na(correlation.model.summary.DF$Environmental.corr.P))))` species associated with adaptive variation (_P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` SD for significant models; Table S3). Similarly, geographic distance between planning units was also correlated significantly with the spatial pattern of neutral genetic variation among planning units for `r sum(correlation.model.summary.DF$Geographic.corr.P<0.05,na.rm=TRUE)` of the `r sum(!is.na(correlation.model.summary.DF$Geographic.corr.P))` species (_P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` SD for significant models; Table S3). Thus, for most species, planning units that contained different environmental conditions contained individuals with different adaptive genetic characteristics, and planning units that were further apart tended to contain individuals with different neutral genetic characteristics. After verifying these patterns were correlated, our next step was to determine whether these associations were strong enough to be effective for conservation planning.

<!-- surrogacy analysis -->
For each species, we generated a suite of `r format(raptr.params.LST[[MODE]]$surrogacy.analysis$random.replicates, big.mark=',', scientific=FALSE)` random prioritizations, and calculated the proportion of environmental variation and adaptive (if present) genetic variation that each prioritization captured. We then repeated this process, and calculated the proportion of variation in geographic position and neutral genetic variation they sampled. The environmental and geographic variables were moderately effective predictors for the genetic variation represented by the randomly generated prioritizations (Figures 1, S28 and S29). The relationship between the proportion of genetic and surrogate variation secured in the prioritizations varied among the species (species $\times$ proportion of environmental variation interaction term: ${\chi^2}_{`r env.surrogacy.AOV$df[[4]]`} = {`r round(env.surrogacy.AOV$Resid..Dev[[4]],2)`}; P `r pvalString(env.surrogacy.AOV$Pr..Chi.[[4]])`$; species $\times$ proportion of geographic variation interaction term: ${\chi^2}_{`r round(geo.surrogacy.AOV$df[[4]],2)`} = `r round(geo.surrogacy.AOV$Resid..Dev[[4]],2)`$; $P =$ `r pvalString(geo.surrogacy.AOV$Pr..Chi.[[4]])`). _Post-hoc_ analyses showed that these relationships were positive for all species (environmental: minimum \textit{z} = `r round(max(env.surrogate.spp.DF$statistic),2)`, maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; geographic: minimum \textit{z} = `r round(min(geo.surrogate.spp.DF$statistic),2)`, maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; Table S4). After establishing that the the proportion of surrogate variation secured in a prioritization also predicted the proportion of genetic variation it secured, our final step was to determine if prioritizations were more effective when generated using targets based on the surrogate variables.

```{r, echo=FALSE, message=FALSE, warning=FALSE, fig.align='center', fig.width=6, fig.height=4.5, out.width='3.42in', out.height='2.5in', fig.cap=paste0('The proportion of adaptive (a; ',sum(!is.na(correlation.model.summary.DF$Environmental.corr.P)),' species) and neutral (b; ',sum(!is.na(correlation.model.summary.DF$Geographic.corr.P)),' species) genetic variation represented in a suite of randomly generated prioritizations for different species as a function of the surrogate variation they also secured (lines). Panels (c) and (d) show the distribution of $R^2$ values for the ',sum(!is.na(correlation.model.summary.DF$Environmental.corr.P)) + sum(!is.na(correlation.model.summary.DF$Geographic.corr.P)),' models shown in panels (a) and (b).')}
# make figure
p1 <- ggplot(aes(x=surrogate.held,y=genetic.held), data=env.surrogacy.DF) +
  xlab('Environmental variation\nsecured (%)') +
  ylab('Adaptive genetic\nvariation secured (%)') +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black')) +
  coord_cartesian(xlim=c(0, 1)) +
  scale_x_continuous(limits=c(0,1), labels=function(x) {x*100}) +
  scale_y_continuous(limits=c(0,1), labels=function(x) {x*100})
for (i in seq_along(env.surrogacy.LST))
  p1 <- p1 + stat_smooth(method='glm', method.args=list(family='binomial'),
    data=filter(env.surrogacy.DF, Species==unique(Species)[i]),
    fullrange=TRUE, se=FALSE, colour='black')
p1 <- p1 + geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(a)')

p2 <- ggplot(aes(x=surrogate.held,y=genetic.held), data=geo.surrogacy.DF) +
  xlab('Geographic variation\nsecured (%)') +
  ylab('Neutral genetic\nvariation secured (%)') +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black', colour='transparent')) +
  coord_cartesian(xlim=c(0, 1)) +
  scale_x_continuous(limits=c(0,1), labels=function(x) {x*100}) +
  scale_y_continuous(limits=c(0,1), labels=function(x) {x*100})
for (i in seq_along(geo.surrogacy.LST))
  p2 <- p2 + stat_smooth(method='glm', method.args=list(family='binomial'),
    data=filter(geo.surrogacy.DF, Species==unique(Species)[i]),
    fullrange=TRUE, se=FALSE, colour='black')
p2 <- p2 + geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(b)')

p1.hist <- ggplot(aes(r2CU),
    data=filter(env.surrogate.spp.DF, !is.na(r2CU))) +
  geom_histogram(fill='grey10', bins=12) +
  ylab('\nFrequency') + xlab(expression(Environmental~variables~(R^2))) +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black'),
    plot.background=element_rect(fill='transparent', colour='transparent')) +
  geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(c)') +
  scale_y_continuous(labels=function(x) {formatC(x, digits=2, format='f')}) +
  scale_x_continuous(limits=c(0, 1), labels=function(x) {formatC(x, digits=2, format='f')})

p2.hist <- ggplot(aes(r2CU),
    data=filter(geo.surrogate.spp.DF, !is.na(r2CU))) +
  geom_histogram(fill='grey10', bins=12) +
  ylab('\nFrequency') + xlab(expression(Geographic~variables~(R^2))) +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black'),
    plot.background=element_rect(fill='transparent')) +
  geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(d)') +
  scale_y_continuous(labels=function(x) {formatC(x, digits=2, format='f')}) +
  scale_x_continuous(limits=c(0, 1), labels=function(x) {formatC(x, digits=2, format='f')})

# render figure
grid.arrange(p1,p2,p1.hist,p2.hist,nrow=2)
```

<!-- scenario analysis -->
To determine whether environmental and geographic targets could improve the effectiveness of prioritizations in representing genetic variation, we generated prioritizations using (i) "amount targets" reflecting the traditional approach of representing a certain proportion of each species' geographic distribution, (ii) "amount and surrogate targets" where we targeted representation of the environmental and geographic surrogates as well as a certain proportion of each species' geographic distribution, and (iii) "amount and genetic targets" where we targeted representation of the directly measured genetic variation as well as a certain proportion of each species' geographic distribution. To determine if these patterns were robust, for each of the three combinations of the targets, we generated prioritizations under four scenarios: (i) single-species with equal costs, (ii) single species with acquisition costs, (iii) multi-species with equal costs, and (iv) multi-species with acquisition costs (Figure S30).

```{r, echo=FALSE, warning=FALSE, message=FALSE, fig.align='center', fig.width=(5*0.9), fig.height=(7.1*0.9), out.width='3.42in', out.height='4in', fig.cap='The proportion of adaptive and neutral genetic variation secured for each species in solutions based on different targets and scenarios. Box plots show the median, and the 25th and 75th percentiles. Whiskers show the data at 1.5 times the inter-quantile range. Points show data outside the whiskers. Red lines show the target proportion of genetic variation.'}
# prepare data for plotting
scenario.SDF <- scenario.DF %>%
  group_by(Prioritisation, Context, Metric, Species) %>%
  summarize(genetic.held=mean(genetic.held, na.rm=TRUE)) %>%
  data.frame()

# make plot
suppressWarnings({
  ggplot(aes(x=Prioritisation,y=genetic.held),
    data=scenario.SDF) +
    geom_boxplot(position=position_dodge(0.9)) +
    scale_fill_manual(name='Prioritization',
      values=c('grey80','grey50','grey20')) +
    geom_abline(slope=0, color='red', linetype='dashed',
      intercept=raptr.params.LST[[MODE]]$scenario.analysis$genetic.target) +
    ylab('Genetic variation secured (%)') +
    xlab('') +
    scale_y_continuous(breaks=c(0,0.25,0.5,0.75,1)) +
    theme_classic() +
    theme(strip.background = element_rect(fill='grey20'),
      strip.text = element_text(color='white', size=9),
      panel.border = element_rect(color='black', fill=NA),
      axis.text.x=element_text(angle=45, hjust=1, vjust=1, size=8),
      axis.text.y=element_text(size=8)) +
    facet_grid(Context ~ Metric)
})
```

The proportion of genetic variation secured in a prioritization depended on the targets used to generate the prioritization (${\chi^2}_{`r scenario.ANOVA.4.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.4.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.ANOVA.4.DF$p.value[[2]])`)$. Most notably, prioritizations based on the surrogate variables represented significantly more genetic variation than amount-based prioritizations (`r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount & surrogate\ntargets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount & surrogate\ntargets')$genetic.held*100),2)` SD overall genetic variation secured versus `r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount targets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount targets')$genetic.held*100),2)` SD; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[1]],2)`$; $P `r pvalString(scenario.GLHT.2.DF$p.value[[1]])`$), and were not distinguishable from prioritizations based on measured genetic data (`r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount & genetic\ntargets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount & genetic\ntargets')$genetic.held*100),2)` SD; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[3]],2)`$; $P `r pvalString(scenario.GLHT.2.DF$p.value[[3]])`$). These results indicate that environmental and geographic surrogates in this case far outperform traditional amount-based conservation planning, and perform almost as well as a conservation plan based directly on measured genetic data.

Overall, the prioritizations tended to secure a greater proportion of adaptive genetic variation than neutral genetic variation (adaptive: mean `r round(mean(filter(scenario.sub.DF,Metric=='Adaptive variation')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Metric=='Adaptive variation')$genetic.held*100),2)` secured across all prioritizations, neutral: mean `r round(mean(filter(scenario.sub.DF,Metric=='Neutral variation')$genetic.held*100),2)` $\pm$ `r round(sd(filter(scenario.sub.DF,Metric=='Neutral variation')$genetic.held*100),2)`; ${\chi^2}_{`r scenario.ANOVA.3.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.3.DF$statistic[[2]],2)`$; _P_ `r pvalString(scenario.ANOVA.3.DF$p.value[[2]])`). Thus, regardless of the targets used to generate a prioritization, or the planning context under which the prioritization was generated, prioritizations tend to secure more adaptive than neutral genetic genetic variation.

The average proportion of genetic variation secured by prioritizations varied under different scenarios (${\chi^2}_{`r scenario.ANOVA.2.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.2.DF$statistic[[2]],2)`$; _P_ `r pvalString(scenario.ANOVA.2.DF$p.value[[2]])`). Specifically, prioritizations generated under single-species scenario (mean `r round(mean(filter(scenario.sub.DF,Context=='Single-species\n(equal costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Single-species\n(equal costs)')$genetic.held*100),2)`) secured less genetic variation than those under the single species with opportunity costs (mean `r round(mean(filter(scenario.sub.DF,Context=='Single-species\n(opportunity costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Single-species\n(opportunity costs)')$genetic.held*100),2)`; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[1]],2)`$; $P = `r pvalString(scenario.GLHT.1.DF$p.value[[1]])`$) or the multi-species with cost scenarios (mean `r round(mean(filter(scenario.sub.DF,Context=='Multi-species\n(opportunity costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Multi-species\n(opportunity costs)')$genetic.held*100),2)`; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[3]],2)`$; $P = `r pvalString(scenario.GLHT.1.DF$p.value[[3]])`$). These results suggest the amount of genetic variation secured in a prioritization may be affected by the data used to represent acquisition cost.

Discussion {.unnumbered}
========================

<!--  introduction -->
We have shown that broad-scale environmental and geographic variables can be effective as surrogates for adaptive and neutral genetic variation in conservation planning. Our study is the first to show this using field measurements of genetic variation for a broad range of species. The spatial patterns of environmental and geographic surrogates were strongly correlated with the patterns of adaptive and neutral genetic variation for most species. Moreover, for most species, prioritizations generated using environmental and geographic targets secured a greater proportion of adaptive and neutral genetic variation than traditional conservation planning. These environmental and geographic surrogates were based on freely available data, and could be applied to any study region across the world.

<!-- surrogate/genetic held correlations -->
Our results demonstrate that environmental and geographic variables are effective surrogates for most species considered here (Figure 2). Despite this, geographic distance was a suprisingly poor surrogate for neutral genetic variation for a few species (eg. _Gypsophila repens_: $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Gypsophila_repens')$Geographic.r2CU,2)`; _Luzula alpinopilosa_: $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Luzula_alpinopilosa')$Geographic.r2CU,2)`; _Gentiana nivalis_ $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Gentiana_nivalis')$Geographic.r2CU,2)`). One explanation for this result is that geographic distance will be a poor surrogate for neutral genetic variation where spatial genetic structure is complicated by additional factors ([@r505; @r497]). For example, some species can maintain relatively high connectivity between distant populations (eg. wind dispersing plants; [@r451]), and in some places can be disrupted by landscape features (eg. anthropogenically modified land [@r497]). Overall, such species were the exception in our analysis, and both surrogates performed moderately well for most species.

<!-- suitable targets for surrogates -->
Our results show that environmental and geographic surrogates can be used to capture genetic variation in prioritizations, the next question is: what percentage of surrogate variation should planners target to preserve genetic variation? To secure at least `r raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100` % of the species' neutral genetic variation, prioritizations needed to sample `r round(mean(surrogate.repr.prop.DF$geographic.held*100,na.rm=TRUE),2)` % $\pm$ `r round(sd(surrogate.repr.prop.DF$geographic.held*100,na.rm=TRUE),2)` SD of the variation in geographic position among the planning units occupied by each species. Additionally, to secure at least `r raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100` % of each species' adaptive genetic variation, prioritizations needed to sample `r round(mean(surrogate.repr.prop.DF$environmental.held*100,na.rm=TRUE),2)` % $\pm$ `r round(sd(surrogate.repr.prop.DF$environmental.held*100,na.rm=TRUE),2)` SD of the environmental variation in planning units occupied each the species. For all species, however, it was possible to generate solutions that secured a large proportion of the surrogate variation (> 90 %) and only a small proportion of genetic variation (< 20 %; Figures S56--S57). Planners may avoid such outcomes by using both amount- and surrogate-based targets. Depending on the study area and the species of conservation interest, higher targets may be needed to increase the likelihood that prioritizations will secure a large proportion of the intra-specific genetic variation for all of the species in the planning exercise.

<!-- adaptive genetic variation secured in prioritizations-->
Our results suggest that conservation planners can secure a reasonably representative sample of the intra-specific adaptive genetic variation using only amount-based targets (Figure 2). For most species, the spatial patterns of adaptive genetic variation clustered into a few distinct groups (eg. _Carex sempervirens_ and _Cerastium uniflorum_; Figures S33 and S34 respectively). As a consequence, a prioritization would only need to select a single planning unit in each cluster to secure a representative sample. This mechanism explains why adaptive variation accumulated much more quickly than neutral genetic variation in the randomly generated prioritizations. Additionally, this mechanism also explains why the single-species prioritizations generated using only amount targets were often able to achieve high representation---they were essentially a random selection and so frequently contained planning units in different parts of the species' ranges (Figure 4a). Overall---and contrary to our original hypothesis---it seems that conventional reserve selection methods might often conserve species' adaptive genetic variation.

<!--  limitations/ weaknesses -->
There are several limitations associated with our analysis. Firstly, the size of the planning units we used (approx. 23 $\times$ 25 km) is much larger than typically used in regional conservation planning (1--10 km$^2$). We used this resolution because the genetic data were collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis by artificially introducing additional spatial auto-correlation. Secondly, we used geographic distances as surrogates for neutral genetic variation, yet distances that incorporate data on topography may better describe connectivity in the study area. However, such distances often require species-specific scaling (eg. [@r497]), and so cannot easily be utilized in multi-species planning exercises. Thirdly, we used amplified fragment length polymorphism data (AFLP; [@r453]) to describe genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data ([@r480]), we know of no suitable multi-species genomic data set, and our methodology would still have utilized only the main gradients of the genetic variation to generate prioritizations in a feasible period of time. Furthermore, even with modern population genomic approaches, a survey can at best hope to identify markers that are linked to functionally adaptive variants.

<!--  conclusion -->
Broad-scale environmental and geographic variables are generally effective surrogates for representing adaptive and neutral genetic variation for most species in our study system. Moreover, data to calculate the surrogate variables are freely available for any location in the world. Careful use of such surrogates in conservation planning could vastly improve the chances of long-term biodiversity persistence for relatively little additional cost.

Materials and Methods {.unnumbered}
===================================

Study system {#study-system .unnumbered}
----------------------------------------

We used data for 27 alpine plant species in the European Alps collected by the IntraBioDiv project ([@r451]; Figure 3a). All data, code, and results are stored in an online repository to permit replication and validation of this study (\url{www.github.com/jeffreyhanson/genetic-surrogates}). This data set has been used to explore patterns of adaptive (eg. [@r455]) and neutral genetic variation (eg. [@r478]), and the potential for species richness as a surrogate for genetic diversity ([@r486]). Data were collected using a $20^{\prime}$ longitude by $21^{\prime}$ latitude grid (approx. 22.3 $\times$ 25 km; Figure S31). Project surveyors visited every second grid cell, and for each species, they collected samples from three individuals if any individuals were found. They genotyped samples using AFLPs, and constructed matrices denoting the presence of polymorphisms at loci for each species (mean `r round(mean(sapply(spp.StructureData.LST, structurer:::n.loci)),2)` $\pm$ `r round(sd(sapply(spp.StructureData.LST, structurer:::n.loci)),1)` SD markers genotyped per species; for more information see [@r478], Table 1). Thus the data set contains information describing the genomic properties of individuals for each species over a geographic grid. We used these data because they comprised comparable genetic information for a range of species with different evolutionary and life histories collected using a standardized sampling scheme.

```{r, echo=FALSE, fig.align='center', fig.width=6, fig.height=1.5, out.width='3.42in', height='2.5in', fig.cap="Map of the study area showing (a) the pattern of species richness, and (b) the distribution of acquisition cost, both plotted on a quantile-based color ramp. Squares denote planning units."}

# calculate species richness
grid.PLY$Species_richness <- grid.PLY@data %>%
  select(5:(4+n_distinct(spp.samples.DF$species))) %>% as.matrix() %>% rowSums()

panel.DF <- grid.sub.FPLY %>%
  summarize(lat=min(lat)+(diff(range(lat))*0.96),
    long=min(long)+(diff(range(long))*0.04))

# plot species richness
p1 <- ggplot() +
  geom_polygon(data=countries.FPLY, aes(x=long, y=lat, group=group),
    fill='grey85', color='grey70') +
  geom_polygon(data=spFortify(grid.PLY), aes(x=long, y=lat,
    group=group, fill=Species_richness), color=NA) +
  scale_fill_gradientn(name='Richness',
    colors=brewer.pal(5, 'YlGn'),
    guide=guide_colorbar(ticks=element_line(color='black'), border=element_line(color='black'))
  ) +
  theme_classic() +
  theme(
    axis.ticks=element_blank(), axis.text=element_blank(), axis.line=element_blank(), plot.margin = unit(c(0, 0, 0, 0), "lines"), axis.title=element_blank(),
    panel.border=element_rect(color='black', fill=NA, size=1)
  ) +
  coord_cartesian(
    xlim=buffered.range(grid.FPLY$long, 0),
    ylim=buffered.range(grid.FPLY$lat, 0)
  ) + geom_text(aes_string(x='long',y='lat'), data=panel.DF, label='(a)')

qn01 <- scales::rescale(quantile(grid.PLY@data$pop.density, seq(0,1,length.out=9), na.rm=TRUE))
p2 <- ggplot() +
  geom_polygon(data=countries.FPLY, aes(x=long, y=lat, group=group),
    fill='grey85', color='grey70') +
  geom_polygon(data=spFortify(grid.PLY), aes(x=long, y=lat,
    group=group, fill=pop.density), color=NA) +
  scale_fill_gradientn(name='Cost',
    colors=brewer.pal(5,'YlOrRd'),
    values=qn01,
    guide=guide_colorbar(ticks=element_line(color='black'), border=element_line(color='black'))
  ) +
  theme_classic() +
  theme(
    axis.ticks=element_blank(), axis.text=element_blank(), axis.line=element_blank(), plot.margin = unit(c(0, 0, 0, 0), "lines"), axis.title=element_blank(),
    panel.border=element_rect(color='black', fill=NA, size=1)
  ) +
  coord_cartesian(
    xlim=buffered.range(grid.FPLY$long, 0),
    ylim=buffered.range(grid.FPLY$lat, 0)
  )  + geom_text(aes_string(x='long',y='lat'), data=panel.DF, label='(b)')

grid.arrange(p1,p2,nrow=1)
```

Landscape data {#landscape-data .unnumbered}
--------------------------------------------

We adopted the sampling grid used to collect data as planning units to develop prioritizations. To reduce computational burden, we omitted cells that did not contain samples (`r nrow(grid.DF)` grid cells used in analysis). To examine the effect of variation in cost, we calculated the total human population density inside each planning unit (1 km$^2$ resolution from the Global Rural-Urban Mapping Project; [@r472]) and used this to represent acquisition cost (Figure 3b). All spatial and statistical analyses were conducted in \texttt{R} (version `r paste0(R.version$major, '.', R.version$minor)`; [@r192]).

We created environmental and geographic surrogate variables for each species (Figure S32). To describe the geographic location of each planning unit (Figure S33), we projected the grid into an equi-distant coordinate system (Europe Equidistant Conic; $\texttt{ESRI:102031}$), calculated the centroid of each grid cell, and extracted their two-dimensional coordinates. To describe the environmental characteristics of each planning unit (Figure S34), we obtained 19 climatic layers ($30^{\prime\prime}$ resolution; [@r33]), projected them and the planning units into an equal-area coordinate system (Europe Lambert Conformal Conic; $\texttt{ESRI:102014}$), and computed planning unit averages for each climatic variable. To reduce dimensionality, for each species, we subjected the climatic values associated with the planning units they were found in to a principal components analysis (PCA; Table S1). We used the first `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components to characterize climatic variation found across the species' geographic distributions inside the study area (capturing `r round(mean(spp.pca.DF[[2]]*100),2)` % $\pm$ `r round(sd(spp.pca.DF[[2]]*100),2)` SD of the total climatic variation; Figure S2). Thus we constructed a two-dimensional geographic space as a potential surrogate for neutral genetic variation, and a `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))`-dimensional environmental space as a potential surrogate for adaptive genetic variation for each species. In these spaces, each planning unit was associated with a single point. Planning units with more comparable environmental conditions, or higher spatial proximity in the real world, were associated with points that were closer together in these environmental or geographic spaces.

Adaptive and neutral genetic data {#adaptive-and-neutral-genetic-data .unnumbered}
----------------------------------------------------------------------------------

To investigate the effectiveness of our putative surrogates, we first needed to identify which of the sampled loci were adaptive (Figure S35). Following best practice, we used two outlier detection methods to achieve this (an individual- and a population-level method; [@r495]). The basic premise underpinning such methods is that neutral loci are expected to exhibit a certain level of variation, and loci that deviate from this expectation are likely to be under selection ([@r480]). The advantage of these methods---in contrast with environmental association analyses---is that they do not use environmental data, which would have introduced an element of circularity into our analysis and invalidated it. Loci identified by both outlier detection methods were treated as adaptive and the remainder were treated as neutral. To minimize false-positives, we omitted loci from both methods where the global frequency of the minor allele was less than 10 % and treated them as neutral.

The first outlier detection method involved fitting multinominal-Dirichlet models implemented in \texttt{BayeScan} (version 2.1; [@r456]). We adopted a similar methodology to Bothwell _et al._ ([-@r455]) and applied it to each of the species separately. Following their methodology, we initially grouped conspecifics into genetic lineages to further minimize false-positives (Figures S36--S62), by fitting admixture and correlated alleles models implemented in \texttt{Structure} (version 2.3.4; [@r462]; `r structure.params.LST[[MODE]]$numruns` replicates per species; `r format(structure.params.LST[[MODE]]$admburnin, big.mark=',', scientific=FALSE)` admixture burnin iterations; `r format(structure.params.LST[[MODE]]$burnin, big.mark=',', scientific=FALSE)` burnin iterations; `r format(structure.params.LST[[MODE]]$numreps+structure.params.LST[[MODE]]$burnin, big.mark=',', scientific=FALSE)` total iterations) using the number of lineages previously determined by Alvarez _et al._ ([@r478]) and combining replicate runs using \texttt{ClumPP} (version 1.1.2; [@r459]; greedy algorithm based on the $G^{\prime}$ statistic; `r format(clumpp.params.LST[[MODE]]$repeats, big.mark=',', scientific=FALSE)` iterations). We then ran $\texttt{Bayescan}$ for each species using these lineages (1:1 prior odds; `r bayescan.params.LST[[MODE]]$reps` replicates per species; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r format(bayescan.params.LST[[MODE]]$burn, big.mark=',', scientific=FALSE)` burn-in iterations; `r format(bayescan.params.LST[[MODE]]$n+bayescan.params.LST[[MODE]]$burn, big.mark=',', scientific=FALSE)` total iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations) using a suitable false discovery rate ($q \leq `r bayescan.params.LST[[MODE]]$fdr`$; [@r466]). We omitted individuals if their population membership was uncertain (maximum membership probability $< `r structure.params.LST[[MODE]]$probability.threshold`$).

The second outlier detection method involved fitting PCAs to identify outlier loci (implemented in the \texttt{pcadapt R} package; [@r487]). To enable comparisons between the two outlier detection methods, we used the same individuals in this analysis as in the \texttt{BayeScan} analysis. For each of the 27 species, we first imputed missing data by replacing missing values with the average frequency among conspecifics, then ran a PCA over the loci matrix, and extracted the minimum number of components needed to secure the over-arching population level variation among the loci (`r pcadapt.params.LST[[MODE]]$min.variance.explained*100` % of the variation in loci). We then computed _q_-values using Mahalanobis distances, and used the same false discovery rate used in the \texttt{BayeScan} analysis to identify loci under selection (using the \texttt{qvalue R} package; [@r491]).

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species. We discarded the population groupings, and partitioned species' adaptive and neutral loci into separate matrices. We applied NMDS (implemented in the \texttt{vegan R} package; [@r458]) using Gower distances (via the \texttt{cluster R} package; [@r457]) to derive continuous variables that described the main gradients of adaptive and neutral genetic variation separately for each species (Table S2). To ensure that the ordinations described a sufficient amount of the genetic variation, we ran successive scaling analyses with increasing dimensionality until a sufficient stress value was obtained (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for each analysis). Since each grid cell had multiple samples per species, we averaged the ordinated values for conspecifics in the same grid cell.

Thus we constructed an adaptive (if detected) and neutral genetic space for each species. For a given species, each planning unit in which it occurred was associated with a multi-dimensional point in the species' adaptive genetic space (if adaptive genetic variation was detected) and another multi-dimensional point in the species' neutral genetic space. Planning units that were closer together in these genetic spaces were occupied by individuals with more comparable AFLP polymorphisms. By spreading out conservation effort across these genetic spaces, and in turn selecting planning units occupied by individuals with increasingly different polymorphisms, prioritizations can secure more genetic variation.

Prioritization method {#prioritization-method .unnumbered}
----------------------------------------------------------

We used the \texttt{raptr R} package to generate solutions ([@r465]). This toolkit can identify the cheapest set of planning units required to preserve both a target proportion of the species' geographic range (using amount-based targets) and a broad sample of variation found across the species' range. We used the environmental and geographic surrogate spaces, and the adaptive (if detected) and neutral genetic spaces to generate and evaluate solutions. Solutions associated with negative values---because they secured very little genetic variation---were replaced with zeros to facilitate statistical analysis. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality.

Computational and statistical analyses {#statistical-analyses .unnumbered}
--------------------------------------------------------

Our first aim was to determine if the environmental and geographic variables correlate with the spatial patterns of adaptive and neutral genetic variation. To achieve this, for each species, we created dissimilarity matrices using Euclidean distances and the data in the surrogate and genetic spaces. These matrices showed differences between the planning units occupied by the species in terms of the units' (i) geographic position, (ii) environmental characteristics, and the (iii) adaptive (if detected) and (iv) neutral genetic characteristics of the individuals inside them.

We fitted maximum likelihood population-effects (MLPE) models using maximum likelihood to investigate correlations between the dissimilarity models (using the \texttt{lme4 R} package; [@r469]). These models use random effects to accommodate the structure of dissimilarity matrices. For each species, we fitted a MLPE model to the species' dissimilarity matrices based on geographic position and neutral genetic variation. If adaptive loci were detected, we also fitted a MLPE model to the species' dissimilarity matrices describing environmental and adaptive genetic variation. All data variables were _z_-transformed to improve convergence. To test if the surrogates explained the genetic variation, we compared each model to its null model using a $\chi^2$ test, and applied Bonferroni corrections.

Our second aim was to determine if the environmental and geographic variables were effective surrogates for adaptive and neutral genetic variation. For each species, we generated `r format(raptr.params.LST[[MODE]]$surrogacy.analysis$random.replicates*2, big.mark=',', scientific=FALSE)` prioritizations by randomly selecting different combinations of planning units that the species occupied. For half of these random prioritizations, we calculated the proportion of geographic variation and neutral genetic variation they secured, and for the other half, we calculated the proportion of environmental variation and adaptive genetic variation they secured. The proportion of surrogate and genetic variation captured in a prioritization was calculated using the \texttt{raptr R} package.

We fitted two full generalized linear models (GLM) with logit links. The first model was fit to the proportion of adaptive genetic variation secured in a prioritization using the proportion of environmental variation also secured in the prioritization. The second model was fit to the proportion of neutral genetic variation secured in a prioritization using the proportion of geographic variation also secured in the prioritization. Additionally, both models contained a variable indicating the species for which the prioritizations were generated, and an interaction term. They were subjected to backwards stepwise term deletion routines to assess term significance. To assess the performance of the surrogates, we refit these models separately for each species and computed the Cragg and Uhler's pseudo-$R^2$ value (using the \texttt{pscl R} package; [@r492]).

Our third aim was to determine if surrogate-based targets improved representation of genetic variation in prioritizations. As previously mentioned, we generated prioritizations using different combinations of targets and conservation planning scenarios. We used `r (raptr.params.LST[[MODE]]$scenario.analysis$amount.target*100)` % amount-based targets for each species in all prioritizations to secure an adequate proportion of the species' distributions. Based on the results from our previous analysis, we used `r (raptr.params.LST[[MODE]]$scenario.analysis$surrogate.target*100)` % surrogate-based targets and `r (raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` % genetic-based targets. We generated a single solution for each target/scenario combination, except for the "single-species (equal costs) with amount-based targets" combination for which we generated `r format(raptr.params.LST[[MODE]]$scenario.analysis$single.species.amount.replicates, big.mark=',', scientific=FALSE)` replicates because it had many optimal solutions. We computed the proportion of adaptive (if present) and neutral genetic variation secured for each species in the prioritizations.

We fitted generalized linear mixed-effects models (GLMMs) with logit links to  evaluate the prioritizations (using the \texttt{lme4 R} package). We fitted a full model to the proportion of genetic variation secured in a given prioritization. This model contained categorical variables indicating the targets (amount only, amount and surrogate targets, or amount and genetic targets), the planning scenario (single-species, multi-species, or multi-species with cost), the type of genetic variation measured (adaptive or neutral), and all interactions between them. Data for same species were accommodated using a random intercept term. The full model was subjected to a backwards stepwise term deletion routine to assess significance. A _post-hoc_ analysis was conducted on the minimal adequate model using Tukey contrasts with a Bonferroni correction (using the \texttt{multcomp R} package; [@r470]).

\showmatmethods
\showacknow
\pnasbreak
