---
title: "Environmental and geographic variables are effective surrogates for genetic variation in conservation planning"
author:
  - name: Jeffrey O. Hanson
    affiliation: a,1
  - name: Jonathan R. Rhodes
    affiliation: b
  - name: Cynthia Riginos
    affiliation: a
  - name: Hugh P. Possingham
    affiliation: a
  - name: Richard A. Fuller
    affiliation: a
address:
  - code: a
    address: School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  - code: b
    address: School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
corresponding_author:
  - code: 1
  - text: "To whom correspondence should be addressed. E-mail: jeffrey.hanson@uqconnect.edu.au"
author_contributions: |
  All authors developed the study and drafted the manuscript. JOH performed all analyses.
lead_author_surname: "Hanson"
abstract: |
  Protected areas buffer species from anthropogenic threats and provide places for processes to continue. However, evolutionary processes are not often explicitly considered when deciding where to site protected areas because genetic data require considerable resources to obtain and analyse. Here we show that environmental and geographic distance variables are effective surrogates for adaptive and neutral intra-specific variation for conservation planning. We obtained occurrence and genetic data from the IntraBioDiv project for 27 species collected over the European Alps using a gridded sampling scheme. For each species, we identified loci that were potentially under selection using outlier loci methods, and mapped their main gradients of adaptive and neutral genetic variation across the grid cells. We then used the cells as planning units to generate prioritisations. Our first analysis verified that the spatial patterns of environmental and geographic variation correlated with those adaptive and neutral genetic variation (respectively). Our second analysis showed that these surrogates can predict the proportion of genetic variation secured in randomly generated solutions. Our third analysis found that solutions generated using additional surrogate-based targets secured more neutral genetic variation. We discovered that solutions generated using traditional amount-based targets tend to secure most species' adaptive genetic variation. Overall, our work provides a rigorous examination of two potential surrogates for genetic variation.
significance: |
  Protected areas are used to preserve biodiversity patterns and processes. Since the resources available for conservation are limited, protected areas need to be sited in cost-effective locations. However, despite their importance, evolutionary processes are not often used to guide reserve selection. This is primarily because genetic data require extensive resources to obtain and process. Here we show that freely available environmental and geographic variables can be used as surrogates for genetic data in conservation planning.
acknowledgements: |
  JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Biodiversity and Conservation Science (CBCS).
keywords:
  - conservation
  - biodiversity
  - AFLP
  - genetic diversity
pnas_type: pnasresearcharticle
bibliography: references.bib
csl: reference-style.csl
output:
  rticles::pnas_article:
    includes:
      in_header: [preamble.tex]
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
knitr::opts_knit$set(root.dir=normalizePath('../..'))
```

```{r,include=FALSE}
# load .rda file
checkpoint::checkpoint('2016-08-03', R.version='3.3.1', scanForPackages=FALSE)
try(session::restore.session('data/final/results.rda'))

# set options
options(error=NULL)

# download basemap
data(countriesHigh)
countries.FPLY <- countriesHigh[
  countriesHigh$ADMIN %in% c(
    'Italy', 'Switzerland', 'France', 'Austria',
    'Germany', 'Slovenia', 'Croatia', 'Hungary',
    'Monaco', 'Germany', 'Slovakia', 'Czech Republic'
  )
,] %>% spFortify
grid.FPLY <- spFortify(grid.PLY)
grid.sub.FPLY <- spFortify(grid.sub.PLY)

# define ggplot2 color wheel function
gg_color_hue <- function(n) {
  hues <- seq(15, 375, length=n+1)
  hcl(h=hues, l=65, c=100)[1:n]
}
```

<!-- Introduction -->
\dropcap{P}rotected areas spearhead conservation efforts ([@r440]). They buffer species from anthropogenic impacts; providing places for them to persist. Since the resources available for conservation are limited, protected areas need to be sited in places that fulfill conservation objectives for minimal cost ([@r255]). To achieve this, conservation planning exercises often generate plans for entire networks of protected areas (prioritisations) to allocate resources cost-effectively and preserve broad-scale biodiversity processes. Intra-specific genetic variation is fundamental to long-term species persistence ([@r165]), and as a consequence, there has been increasing interest in designing prioritisations that capture and cultivate this aspect of biodiversity ([@r67; @r3; @r61; @r499]). 

Broadly speaking, though the strength of natural selection is a continuous force, genetic variation can be classified as either adaptive or neutral (reviewed in [@r480]). Adaptive genetic variation is associated with loci that significantly affect fitness. Typically, such "adaptive" loci are detected due to anomalous patterns among individuals (eg. [@r456; @r487]), though phenotypic-genetic associations are more likely to identify functionally important genomic regions for individual species (eg. [@r500]). By representing the full range of adaptive variation, protected areas can help ensure that populations with particularly beneficial adaptations are not lost ([@r3; @r43]). In contrast, neutral genetic variation is associated with loci that do not significantly affect fitness, but instead reflects the evolutionary history of different populations. By representing the full range of neutral variation, protected areas can safeguard against the adverse effects of low genetic diversity ([@r67]). Thus optimally sited protected area networks would aim to represent both adaptive and neutral patterns of genetic variation ([@r67; @r3; @r43]).

Recently, the use of surrogate data has been proposed to generate prioritisations that sufficiently intra-specific genetic variation--without needing to directly utilise genetic data to achieve this. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been proposed to guide reserve selection as surrogates for adaptive genetic variation (eg. [@r202; @r2]). By generating prioritisations that preserve a more representative sample of environmental conditions across species' ranges, these prioritisations could also capture more adaptive variation. However, the effectiveness of environmental surrogates in representing adaptive genetic variation remains unverified. Neutral genetic variation arises from a reduction in gene flow between individuals. Over the last few years, the field of phylogeography has predominantly focused on describing neutral genetic diversity, and so the landscape factors that affect neutral variation are relatively well understood (reviewed in [@r501]). Potential surrogates for neutral variation have been based on variables that predict the levels of connectivity between planning units. For instance, isolation by distance (IBD; [@r259]) predicts that populations further apart will experience less gene flow, and in turn, share less neutral loci. Based on this idea, geographic surrogates based on distance have been found to improve conservation prioritisations in an island network ([@r200]). However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors (eg. habitat; [@r497])--the typical situations in most conservation planning exercises.

Here we investigate the effectiveness of environmental and geographic surrogates for capturing adaptive and neutral genetic variation respectively in the context of conservation planning. We used distribution and genomic data for 27 alpine plants collected by the international IntraBioDiv project in the European Alps ([@r451]). We show that for most species the spatial patterns of adaptive and neutral genetic variation correlated with environmental and geographic variables. We also show that for most species the level of association is strong enough to be operational for conservation planning. Finally, we demonstrate the using these surrogates results in prioritisations that secure more genetic variation than expected by chance.

Results {.unnumbered}
=====================

<!-- summary of methods -->
We detected putatively adaptive genetic variation in `r as.character(as.english(length(env.surrogacy.LST)))` of the 27 plant species. We used two outlier detection methods to detect loci under selection, and they returned reasonably consistent results (mean `r round(mean(spp.outlier.methods.congruence, na.rm=TRUE),2)` % loci per species assigned the same classification $\pm$ `r round(sd(spp.outlier.methods.congruence, na.rm=TRUE),2)` SD; Table S1). Of the species that were associated with adaptive genetic variation, only a small proportion of loci were classified as being adaptive (mean `r round(mean(spp.outlier.loci.props, na.rm=TRUE),2)` % loci per species $\pm$ `r round(sd(spp.outlier.loci.props, na.rm=TRUE),2)` SD). After identifying the loci showing strong signals of selection, we used non-metric multi-dimensional scaling (NMDS) analyses to identify the main gradients of adaptive (if detected) and neutral variation for each species. Generally, only a small number of continuous dimensions was needed to sufficiently describe their patterns of adaptive (all $K = `r unique(na.omit(adaptive.k))`$; Table S2) and neutral genetic variation ($K = `r min(neutral.k)`$--$`r max(neutral.k)`$; Table S2). The resulting ordinations were used to construct an adaptive (if detected) and neutral genetic space for each species. The spatial distribution of these genetic spaces (Figures S1--S27) generally showed substantial spatial auto-correlation (eg. _Cerastium uniflorum_, Figure S6; _Dryas octopetala_, Figures S8), suggesting that the environmental and geographic variables may prove to be effective surrogates.

<!-- correlation analysis -->
We verified that the spatial patterns in environmental and geographic variation correlated with the broad-scale spatial patterns in adaptive and neutral variation for most species (respectively). For each species,we constructed dissimilarity matrices based on the planning units they were found in. These matrices expressed differences between the planning units in terms of their (i) environmental characteristics, (ii) geographic position, and also the (iii) adaptive (if detected), and (iv) neutral genetic characteristics of individuals found inside them. By correlating these matrices, the spatial patterns of environmental variation were found to be significantly correlated with that of adaptive genetic variation for `r as.character(as.english(sum(correlation.model.summary.DF$Environmental.corr.P<0.05,na.rm=TRUE)))` of the `r as.character(as.english(sum(!is.na(correlation.model.summary.DF$Environmental.corr.P))))` species associated with adaptive variation (_P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` SD for significant models; Table S3). Similarly, geographic distance between planning units was also correlated significantly with the spatial pattern of neutral genetic variation among planning units for `r sum(correlation.model.summary.DF$Geographic.corr.P<0.05,na.rm=TRUE)` of the `r sum(!is.na(correlation.model.summary.DF$Geographic.corr.P))` species (_P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` SD for significant models; Table S3). Thus, for most species, planning units that contained different environmental conditions contained individuals with different adaptive genetic characteristics, and planning units that were further apart tended to contain individuals with different neutral genetic characteristics. After verifying these patterns were correlated, our next step was to determine whether they were correlated enough to be effective for conservation planning.

<!-- surrogacy analysis -->
For each species, we generated a suite of `r  format(raptr.params.LST[[MODE]]$surrogacy.analysis$random.replicates, big.mark=',', scientific=FALSE)` random prioritisations, and calculated the proportion of environmental variation and adaptive (if present) genetic variation they sampled. We then repeated this process, and calculated the proportion of geographic variation and neutral genetic variation they sampled. The environmental and geographic variables were moderately effective predictors for the genetic variation represented by the randomly generated prioritisations (Figure 1; Figures S28--S29). The relationship between the proportion of genetic and surrogate variation secured in the prioritisations varied among the species (species $\times$ proportion of environmental variation interaction term: ${\chi^2}_{`r env.surrogacy.AOV$df[[4]]`} = {`r round(env.surrogacy.AOV$Resid..Dev[[4]],2)`}; P `r pvalString(env.surrogacy.AOV$Pr..Chi.[[4]])`$; species $\times$ proportion of geographic variation interaction term: ${\chi^2}_{`r round(geo.surrogacy.AOV$df[[4]],2)`} = `r round(geo.surrogacy.AOV$Resid..Dev[[4]],2)`$; $P$ `r pvalString(geo.surrogacy.AOV$Pr..Chi.[[4]])`). _Post-hoc_ analyses showed that these relationships were positive for all species (environmental: minimum \textit{z} = `r round(max(env.surrogate.spp.DF$statistic),2)`, maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; geographic: minimum \textit{z} = `r round(min(geo.surrogate.spp.DF$statistic),2)`, maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; Table S4). After establishing that the the proportion of surrogate variation secured in a prioritisation also predicted the proportion of genetic variation it secured, our final step was to determine if prioritisations were more effective when generated using targets based on the surrogates.

```{r, echo=FALSE, message=FALSE, warning=FALSE, fig.align='center', fig.width=6, fig.height=4.5, out.width='3.42in', out.height='2.5in', fig.cap='Adaptive (a) and neutral (b) genetic variation represented in a suite of randomly generated prioritisations as a function of the environmental and geographic surrogates they also secured (respectively). Panels (c) and (d) show the distribution of $R^2$ values for models using each surrogate to explain genetic variation.'}
# make figure
p1 <- ggplot(aes(x=surrogate.held,y=genetic.held), data=env.surrogacy.DF) +
  xlab('Environmental variation\nsecured (%)') + 
  ylab('Adaptive genetic\nvariation secured (%)') +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black')) +
  coord_cartesian(xlim=c(0, 1)) +
  scale_x_continuous(limits=c(0,1), labels=function(x) {x*100}) +
  scale_y_continuous(limits=c(0,1), labels=function(x) {x*100})
for (i in seq_along(env.surrogacy.LST))
  p1 <- p1 + stat_smooth(method='glm', method.args=list(family='binomial'),
    data=filter(env.surrogacy.DF, Species==unique(Species)[i]),
    fullrange=TRUE, se=FALSE, colour='black')
p1 <- p1 + geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(a)')

p2 <- ggplot(aes(x=surrogate.held,y=genetic.held), data=geo.surrogacy.DF) +
  xlab('Geographic variation\nsecured (%)') + 
  ylab('Neutral genetic\nvariation secured (%)') +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black', colour='transparent')) +
  coord_cartesian(xlim=c(0, 1)) +
  scale_x_continuous(limits=c(0,1), labels=function(x) {x*100}) +
  scale_y_continuous(limits=c(0,1), labels=function(x) {x*100})
for (i in seq_along(geo.surrogacy.LST))
  p2 <- p2 + stat_smooth(method='glm', method.args=list(family='binomial'),
    data=filter(geo.surrogacy.DF, Species==unique(Species)[i]),
    fullrange=TRUE, se=FALSE, colour='black')
p2 <- p2 + geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(b)')

p1.hist <- ggplot(aes(r2CU),
    data=filter(env.surrogate.spp.DF, !is.na(r2CU))) +
  geom_histogram(fill='grey10', bins=5) +
  ylab('\nFrequency') + xlab(expression(Environmental~variables~(R^2))) +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black'),
    plot.background=element_rect(fill='transparent', colour='transparent')) +
  geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(c)') +
  scale_y_continuous(labels=function(x) {formatC(x, digits=2, format='f')}) +
  scale_x_continuous(limits=c(0, 1), labels=function(x) {formatC(x, digits=2, format='f')})

p2.hist <- ggplot(aes(r2CU),
    data=filter(geo.surrogate.spp.DF, !is.na(r2CU))) +
  geom_histogram(fill='grey10', bins=10) +
  ylab('\nFrequency') + xlab(expression(Geographic~variables~(R^2))) +
  theme_classic() + theme(axis.line.x=element_line(color='black'),
    axis.line.y=element_line(color='black'), 
    plot.background=element_rect(fill='transparent')) +
  geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(d)') +
  scale_y_continuous(labels=function(x) {formatC(x, digits=2, format='f')}) +
  scale_x_continuous(limits=c(0, 1), labels=function(x) {formatC(x, digits=2, format='f')})

# render figure
grid.arrange(p1,p2,p1.hist,p2.hist,nrow=2)
```

<!-- scenario analysis -->
To determine whether using environmental and geographic targets could improve the effectiveness of prioritisations in representing genetic variation, we generated prioritisations using (i) "amount targets" reflecting the traditional approach of representing a certain proportion of each species' geographic distribution, (ii) "amount and surrogate targets" where we also targeted representation of the environmental and geographic surrogates as well as a certain proportion of each species' geographic distribution, and (iii) "amount and genetic targets" where we targeted representation of the directly measured genetic variation as well as a certain proportion of each species' geographic distribution. To determine if these patterns were robust, for each of the three combinations of the targets, we generated prioritisations under four scenarios: (i) single-species with equal costs, (ii) single species with opportunity costs, (iii) multi-species with equal costs, and (iv) multi-species with opportunity costs (Figure 3, Figure S30).

```{r, echo=FALSE, warning=FALSE, message=FALSE, fig.align='center', fig.width=(5*0.9), fig.height=(7.1*0.9), out.width='3.42in', out.height='4in', fig.cap='Box plots show the proportion of adaptive and neutral genetic variation secured for each species in solution generated from different targets and scenarios. Red lines show the target proportion of genetic variation.'}
# prepare data for plotting
scenario.SDF <- scenario.DF %>% 
  group_by(Prioritisation, Context, Metric, Species) %>% 
  summarize(genetic.held=mean(genetic.held, na.rm=TRUE)) %>%
  data.frame()

# make plot
suppressWarnings({
  ggplot(aes(x=Prioritisation,y=genetic.held),
    data=scenario.SDF) +
    geom_boxplot(position=position_dodge(0.9)) +
    scale_fill_manual(name='Prioritisation',
      values=c('grey80','grey50','grey20')) +
    geom_abline(slope=0, color='red', linetype='dashed',
      intercept=raptr.params.LST[[MODE]]$scenario.analysis$genetic.target) +
    ylab('Genetic variation secured (%)') +
    xlab('') +
    scale_y_continuous(breaks=c(0,0.25,0.5,0.75,1)) +
    theme_classic() +
    theme(strip.background = element_rect(fill='grey20'),
      strip.text = element_text(color='white', size=9),
      panel.border = element_rect(color='black', fill=NA),
      axis.text.x=element_text(angle=45, hjust=1, vjust=1, size=8),
      axis.text.y=element_text(size=8)) +
    facet_grid(Context ~ Metric)
})
```

The proportion of genetic variation secured in a prioritisation depended on the targets used to generate the prioritisation (${\chi^2}_{`r scenario.ANOVA.4.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.4.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.ANOVA.4.DF$p.value[[2]])`)$. Most notably, prioritisations based on the surrogate variables represented significantly more genetic variation than amount-based prioritisations (`r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount & surrogate\ntargets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount & surrogate\ntargets')$genetic.held*100),2)` SD overall genetic variation secured versus `r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount targets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount targets')$genetic.held*100),2)` SD; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[1]],2)`$; $P = `r pvalString(scenario.GLHT.2.DF$p.value[[1]])`$), and were not distinguishable from prioritisations based on measured genetic data (`r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount & genetic\ntargets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount & genetic\ntargets')$genetic.held*100),2)` SD; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[3]],2)`$; $P = `r pvalString(scenario.GLHT.2.DF$p.value[[3]])`$). These results indicate that environmental and geographic surrogates in this case far outperform traditional amount-based conservation planning, and perform as well as a conservation plan based directly on measured genetic data.

Overall, the prioritisations tended to secure more adaptive then neutral genetic variation (adaptive: mean `r round(mean(filter(scenario.sub.DF,Metric=='Adaptive variation')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Metric=='Adaptive variation')$genetic.held*100),2)` secured across all prioritisations, neutral: mean `r round(mean(filter(scenario.sub.DF,Metric=='Neutral variation')$genetic.held*100),2)` $\pm$ `r round(sd(filter(scenario.sub.DF,Metric=='Neutral variation')$genetic.held*100),2)`; ${\chi^2}_{`r scenario.ANOVA.3.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.3.DF$statistic[[2]],2)`$; _P_ `r pvalString(scenario.ANOVA.3.DF$p.value[[2]])`). Thus, regardless of the targets used to generate a prioritisation, or the planning context under which the prioritisation was generated, prioritisations tend to secure more adaptive than neutral genetic genetic variation.

The average proportion of genetic variation secured by prioritisations varied under different scenarios (${\chi^2}_{`r scenario.ANOVA.2.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.2.DF$statistic[[2]],2)`$; _P_ `r pvalString(scenario.ANOVA.2.DF$p.value[[2]])`). Both the multi-species (mean `r round(mean(filter(scenario.sub.DF,Context=='Multi-species\n(equal costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Multi-species\n(equal costs)')$genetic.held*100),2)`) and multi-species with cost (mean `r round(mean(filter(scenario.sub.DF,Context=='Multi-species\n(opportunity costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Multi-species\n(opportunity costs)')$genetic.held*100),2)`) were both found to secure more genetic variation than the single species scenario (mean `r round(mean(filter(scenario.sub.DF,Context=='Single-species\n(equal costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Single-species\n(equal costs)')$genetic.held*100),2)`; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.GLHT.1.DF$p.value[[2]])`$; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[1]],2)`$; $P = `r pvalString(scenario.GLHT.1.DF$p.value[[1]])`$ respectively). These results suggest that on average the multi-species prioritisations can secure more genetic variation than single-species prioritisations--regardless of targets used to generate them.

Discussion {.unnumbered}
========================

<!--  introduction -->
We aimed to investigate the effectiveness of broad-scale environmental and geographic variables as surrogates for adaptive and neutral genetic variation in conservation planning. Our study is the first to address this aim using a broad range of species and calibrated against field measurements of genetic variation. The spatial patterns of environmental and geographic surrogates were strongly correlated with the patterns of adaptive and neutral genetic variation for most species, and environmental and geographic targets resulted in prioritisations that secured a greater proportion of adaptive and neutral genetic variation than traditional conservation planning for most species. These environmental and geographic surrogates are based on freely available data, and could be calculated for any study region across the world.

<!-- surrogate/genetic held correlations -->
Our results demonstrate that environmental and geographic surrogates are strong enough to be operational (Figure 2). For most species, the environmental and geographic surrogates variables were effective surrogates for adaptive (mean `r round(mean(env.surrogate.spp.DF$r2CU),2)` $R^2$ $\pm$ `r round(sd(env.surrogate.spp.DF$r2CU),2)` SD; Figures 2c and S56) and neutral genetic variation (mean `r round(mean(geo.surrogate.spp.DF$r2CU),2)` $R^2$ $\pm$ `r round(sd(geo.surrogate.spp.DF$r2CU),2)` SD; Figures 2d and Figure S57). Despite this, for a few species, geographic distance was a poor surrogate for neutral genetic variation (eg. _Gypsophila repens_: $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Gypsophila_repens')$Geographic.r2CU,2)`; _Luzula alpinopilosa_: $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Luzula_alpinopilosa')$Geographic.r2CU,2)`; _Gentiana nivalis_ $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Gentiana_nivalis')$Geographic.r2CU,2)`). Indeed, geographic distances will be a poor surrogate for species that utilise dispersal modes that result in non-linear relationships between connectivity and geographic distance [eg. wind dispersing plants can maintain high connectivity between distant areas; @r451], and in landscapes where connectivity is most affected by landscape features ([@r497]). Overall, however, such species were the exception in our analysis, and both simple surrogates performed moderately well for a diverse set of species.

<!-- suitable targets for surrogates -->
Our results suggest that environmental and geographic surrogates can predict how much genetic variation prioritisations secure, but what targets should be used? First, our results suggest that surrogate-based targets should be used in conjunction with amount-based targets. For all species, it was possible to generate solutions that secured a high proportion of the surrogate variable (> 90 %) and only a small proportion of adaptive or neutral genetic variation (< 20 %; Figures S28--S29), and so use of amount-based targets may ameliorate this. Second, geographic surrogates may need much higher targets than environmental surrogates to capture evolutionary processes. In our analysis, geographic variation accumulated much slower for neutral genetic representation than environmental variation did with adaptive representation (Figures 2a--2b and S28--S29). To secure a representative sample of the species' neutral genetic variation ($\geq$ `r raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100` %), prioritisations needed to sample `r round(mean(surrogate.repr.prop.DF$geographic.held*100,na.rm=TRUE),2)` % $\pm$ `r round(sd(surrogate.repr.prop.DF$geographic.held*100,na.rm=TRUE),2)` SD of the variation in geographic position among the planning units occupied by the species. On the other hand, adaptive variation accumulated much faster. To secure at least `r raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100` % of each species' adaptive genetic variation, prioritisations only needed to sample on average `r round(mean(surrogate.repr.prop.DF$environmental.held*100,na.rm=TRUE),2)` % $\pm$ `r round(sd(surrogate.repr.prop.DF$environmental.held*100,na.rm=TRUE),2)` SD of the environmental conditions found across the planning units occupied by the species. Note that these proportions are based on averages across species. In many cases, higher targets may be needed to ensure that prioritisations secure a representative sample of the genetic variation for all species in a planning exercise.

<!-- adaptive genetic variation secured in prioritisations-->
Our results suggest that conservation planners can secure a reasonably representative sample of the intra-specific adaptive genetic variation using only amount-based targets. For most species, the spatial patterns of adaptive genetic variation clustered into a few distinct groups (eg. _Carex semipervens_ and _Cerastium uniflorum_; Figures S5--S6). As a consequence, a prioritisation would only need to select a planning unit in each cluster to secure a representative sample. This mechanism explains why adaptive variation accumulated much quicker than neutral genetic variation in the randomly generated prioritisations. Additionally, this mechanism also explains why the single-species prioritisations generated using only amount targets were often able to achieve high representation--they were essentially a random selection and so frequently contained planning units in different parts of the species' ranges (Figure 4a). Overall--and contrary to our original hypotheses--it seems that conventional reserve selection methods may secure a representative sample of species' adaptive genetic variation.

<!--  limitations/ weaknesses -->
There are several limitations associated with our analysis. First, the size of the planning units we used (23 $\times$ 25 km) is much larger than typically used in regional conservation planning scenarios (1--10 km$^2$). We used this resolution because the genetic data were collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis by artificially introducing additional spatial auto-correlation. Second, we used geographic distances as surrogates for neutral genetic variation, but connectivity in this landscape may be better described using distances that incorporate information on topography. However, such distances require specific-specific scaling to adequate describe their connectivity (eg. [@r497]), and so cannot easily be utilised in large-scale planning exercises. Third, we used amplified fragment length polymorphism data (AFLP; [@r453]) to characterise the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data ([@r480]), no multi-species genomic data set with similar properties currently exists, and our methodology would still have utilised only the main gradients of the genetic variation to generate prioritisations in a feasible amount of time.

<!--  conclusion -->
Broad-scale environmental and geographic variables are generally effective surrogates for representing adaptive and neutral genetic variation for most species in our study system. Moreover, data to calculate the surrogate variables are freely available for any location in the world. Careful use of such surrogates in conservation planning could vastly improve the chances of long-term biodiversity persistence for relatively additional cost.

Materials and Methods {.unnumbered}
===================================

Study system {#study-system .unnumbered}
----------------------------------------

We used data for 27 alpine plant species in the European Alps collected by the IntraBioDiv project ([@r451]; Figure 2a). All data, code, and results are stored in an online repository (\url{www.github.com/jeffreyhanson/genetic.surrogates}) to replicate and validate this study. This data set has previously been used to explore patterns of adaptive (eg. [@r475; @r455]) and neutral genetic variation (eg. [@r478]), and to show that species richness is not an effective surrogate for genetic diversity in conservation planning ([@r486]). It was collected using a $20^{\prime}$ longitude by $21^{\prime}$ latitude grid (approx. 22.3 km $\times$ 25 km; Figure S31). Project members visited every second grid cell, and if a species was detected in a cell, samples were collected from three individuals. They genotyped samples using AFLPs, and constructed matrices denoting the presence/absence of polymorphisms at loci for each species (mean `r round(mean(sapply(spp.StructureData.LST, structurer:::n.loci)),2)` $\pm$ `r round(sd(sapply(spp.StructureData.LST, structurer:::n.loci)),1)` SD markers genotyped per species; for more information see [@r478], Table 1). Thus the data set contains information describing the genomic properties of individuals for each species present in every second grid cell. We used this data set because it comprised of comparable genetic data for a range of species with different evolutionary and life histories collected using a standardised sampling scheme, and pertained to a study area with substantial environmental variation to test the effectiveness of environmental variables. 

```{r, echo=FALSE, fig.align='center', fig.width=6, fig.height=1.5, out.width='3.42in', height='2.5in', fig.cap="Map of the study area. Squares denote planning units. Panel (a) depicts patterns of species riches, and panel (b) shows the distribution of acquisition cost (plotted using a quantile-based color ramp)."}

# calculate species richness
grid.PLY$Species_richness <- grid.PLY@data %>%
  select(5:(4+n_distinct(spp.samples.DF$species))) %>% as.matrix() %>% rowSums()
  
panel.DF <- grid.sub.FPLY %>% 
  summarize(lat=min(lat)+(diff(range(lat))*0.96),
    long=min(long)+(diff(range(long))*0.04))
  
# plot species richness
p1 <- ggplot() +
  geom_polygon(data=countries.FPLY, aes(x=long, y=lat, group=group),
    fill='grey85', color='grey70') +
  geom_polygon(data=spFortify(grid.PLY), aes(x=long, y=lat, 
    group=group, fill=Species_richness), color=NA) +
  scale_fill_gradientn(name='Richness',
    colors=brewer.pal(5, 'YlGn'),
    guide=guide_colorbar(ticks=element_line(color='black'), border=element_line(color='black'))
  ) +
  theme_classic() +
  theme(
    axis.ticks=element_blank(), axis.text=element_blank(), axis.line=element_blank(), plot.margin = unit(c(0, 0, 0, 0), "lines"), axis.title=element_blank(),
    panel.border=element_rect(color='black', fill=NA, size=1)
  ) +
  coord_cartesian(
    xlim=buffered.range(grid.FPLY$long, 0),
    ylim=buffered.range(grid.FPLY$lat, 0)
  ) + geom_text(aes_string(x='long',y='lat'), data=panel.DF, label='(a)')
  
qn01 <- scales::rescale(quantile(grid.PLY@data$pop.density, seq(0,1,length.out=9), na.rm=TRUE))
p2 <- ggplot() +
  geom_polygon(data=countries.FPLY, aes(x=long, y=lat, group=group),
    fill='grey85', color='grey70') +
  geom_polygon(data=spFortify(grid.PLY), aes(x=long, y=lat, 
    group=group, fill=pop.density), color=NA) +
  scale_fill_gradientn(name='Cost',
    colors=brewer.pal(5,'YlOrRd'),
    values=qn01,
    guide=guide_colorbar(ticks=element_line(color='black'), border=element_line(color='black'))
  ) +
  theme_classic() +
  theme(
    axis.ticks=element_blank(), axis.text=element_blank(), axis.line=element_blank(), plot.margin = unit(c(0, 0, 0, 0), "lines"), axis.title=element_blank(),
    panel.border=element_rect(color='black', fill=NA, size=1)
  ) +
  coord_cartesian(
    xlim=buffered.range(grid.FPLY$long, 0),
    ylim=buffered.range(grid.FPLY$lat, 0)
  )  + geom_text(aes_string(x='long',y='lat'), data=panel.DF, label='(b)')
  
grid.arrange(p1,p2,nrow=1)
```

Landscape data {#landscape-data .unnumbered}
--------------------------------------------

We adopted the sampling grid used by the IntraBioDiv project as planning units to develop prioritisations, omitting cells that were not found to contain individuals belonging to the study species (`r nrow(grid.DF)` grid cells used in analysis). To examine the potential effects of variation in cost, we calculated the total human population density inside each planning unit (1 km$^2$ resolution from the Global Rural-Urban Mapping Project; [@r472]) and used this to represent acquisition cost (Figure 2b). All spatial and statistical analysis were conducted in \texttt{R} (version `r paste0(R.version$major, '.', R.version$minor)`; [@r192]).

We created environmental and geographic surrogate variables for each species (see Figures S32--S34 for workflow). To describe the geographic location of each planning unit, we projected the grid into an equi-distant coordinate system (Europe Equidistant Conic; $\texttt{ESRI:102031}$), calculated the centroid of each grid cell, and extracted their two-dimensional coordinates. To describe the environmental characteristics of each planning unit, we obtained 19 climatic layers ($30^{\prime\prime}$ resolution; [@r33]), projected them and the planning units into an equal-area coordinate system (Europe Lambert Conformal Conic; $\texttt{ESRI:102014}$), and computed planning unit averages for each climatic variable. To reduce dimensionality, for each species, we subject the climatic values associated with their planning units to a principal components analysis (PCA; Table S5). We used the first `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components to characterise climatic variation found across the species' distributions inside the study area (capturing `r round(mean(spp.pca.DF[[2]]*100),2)` % $\pm$ `r round(sd(spp.pca.DF[[2]]*100),2)` SD of the total climatic variation; Figure S2). Thus we constructed a two-dimensional geographic space as a potential surrogate for neutral genetic variation, and a `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))`-dimensional environmental space as a potential surrogate for adaptive genetic variation for each species. In these spaces, each planning unit was associated with a single point. Planning units with more similar environmental conditions, or higher spatial proximity in the real world, were associated with points that were closer together in these spaces (respectively).

Adaptive and neutral genetic data {#adaptive-and-neutral-genetic-data .unnumbered}
----------------------------------------------------------------------------------

To investigate the effectiveness of our supposed surrogates, we quantified the main spatial patterns of adaptive and neutral genetic variation (see Figures S32 and S35 for workflow). To do this, we first needed to identify which of the sampled loci were adaptive. Following best practices, we used two outlier detection methods to achieve this (an individual- and a population-level method; [@r495]). The basic premise underpinning such methods is that neutral loci are expected to exhibit a certain level of variation, and loci that deviate from this expectation are likely to be under selection ([@r480]). The advantage of these methods--in contrast to environmental association analyses--is that they do not use environmental data, which would have introduced circularity into our analysis. Loci identified as outliers in both analyses were treated as adaptive and the remainder were treated as neutral. To minimise false-positives, we omitted loci from both methods where the global frequency of the minor allele was less than 10 % and treated them as neutral.

The first outlier detection method involved fitting multinominal-Dirichlet models implemented in \texttt{BayeScan} (version 2.1; [@r456]). We adopted a similar methodology to Bothwell _et al._ ([-@r455]) and applied it to each of the species separately. Following their methodology, we initially grouped conspecifics into genetic lineages (Figures S36--S62). We grouped conspecifics by fitting admixture and correlated alleles models implemented in \texttt{Structure} (version 2.3.4; [@r462]; `r structure.params.LST[[MODE]]$numruns` replicates per species; `r format(structure.params.LST[[MODE]]$admburnin, big.mark=',', scientific=FALSE)` admixture burnin iterations; `r format(structure.params.LST[[MODE]]$burnin, big.mark=',', scientific=FALSE)` burnin iterations; `r format(structure.params.LST[[MODE]]$numreps+structure.params.LST[[MODE]]$burnin, big.mark=',', scientific=FALSE)` total iterations) using the number of lineages previously determined by Alvarez _et al._ ([@r478]) and combining replicate runs using \texttt{ClumPP} (version 1.1.2; [@r459]; greedy algorithm based on the $G^{\prime}$ statistic; `r format(clumpp.params.LST[[MODE]]$repeats, big.mark=',', scientific=FALSE)` iterations). We then ran $\texttt{Bayescan}$ for each species using these lineages (1:1 prior odds; `r bayescan.params.LST[[MODE]]$reps` replicates per species; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r format(bayescan.params.LST[[MODE]]$burn, big.mark=',', scientific=FALSE)` burn-in iterations; `r format(bayescan.params.LST[[MODE]]$n+bayescan.params.LST[[MODE]]$burn, big.mark=',', scientific=FALSE)` total iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations) using a suitable false discovery rate ($q \leq `r bayescan.params.LST[[MODE]]$fdr`$; [@r466]). We omitted individuals if their population membership was uncertain (maximum membership probability $< `r structure.params.LST[[MODE]]$probability.threshold`$).

The second outlier detection method involved fitting PCAs to identify outlier loci (implemented in the \texttt{pcadapt R} package; [@r487]). To enable comparisons between the two outlier detection methods, we used the same individuals in this analysis as in the \texttt{BayeScan} analysis. For each of the 27 species, we first imputed missing data--replacing missing values with the average frequency among conspecifics--ran a PCA, and extracted the minimum number of components needed to secure the over-arching population level variation among the loci (`r pcadapt.params.LST[[MODE]]$min.variance.explained*100` % of the variation in loci). We then computed _q_-values using Mahalanobis distances, and used the same false discovery rate used in the \texttt{BayeScan} analysis to identify loci under selection (using the \texttt{qvalue R} package; [@r491]).

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species. We discarded the population groupings, and partitioned species' adaptive and neutral loci into separate matrices. We applied NMDS (implemented in the \texttt{vegan R} package; [@r458]) using Gower distances (via the \texttt{cluster R} package; [@r457]) to derive continuous variables that described the main gradients of adaptive and neutral genetic variation separately for each species (Table S2). To ensure that the ordinations described a sufficient amount of the genetic variation, we ran successive scaling analyses with increasing dimensionality until a sufficient stress value was obtained (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for each analysis). Since each grid cell had multiple samples per species, we averaged the ordinated values belonging to conspecifics in the same grid cell. 

Thus we constructed an an adaptive (if detected) and neutral genetic space for each species. For a given species, each planning unit that it was found in was associated with a multi-dimensional point in the species' adaptive genetic space (if adaptive genetic variation was detected) and another multi-dimensional point in the species' neutral genetic space. Planning units that were closer together in these genetic spaces were occupied by individuals with more similar polymorphisms. By spreading out conservation effort across these genetic spaces, and in turn selecting planning units occupied by individuals with increasingly different polymorphisms, prioritisations can secure more genetic variation.

Prioritisation method {#prioritisation-method .unnumbered}
----------------------------------------------------------

We used the unreliable representative and adequate prioritisation (URAP) formulation of reserve selection ([@r465]). This formulation can identify the minimum number of planning units required to preserve both a proportion of the species' geographic range (using amount-based targets) and a sample of variation found across the species' range. We used the environmental and geographic surrogate spaces, and the adaptive (if detected) and neutral genetic spaces to generate and evaluate solutions. Solutions associated with negative values--because they secured a very small amount of genetic variation--were replaced with zeros to facilitate statistical analysis. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality (using the \texttt{raptr R} package [@r465]).

Computational and statistical analyses {#statistical-analyses .unnumbered}
--------------------------------------------------------

Our first aim was to determine if the environmental and geographic variables correlate with spatial patterns of adaptive and neutral genetic variation. To achieve this, for each species, we generated two dissimilarity matrices between the planning units that it occupied based on their environmental conditions and geographic position (using their respective surrogate spaces). We then generated another dissimilarity matrix between the planning units using their values in the species' neutral genetic space. If the species was associated with adaptive loci, we also generated another dissimilarity matrix between the planning units using the species' adaptive genetic space. Dissimilarities were compured as Euclidean distances.

We then fit maximum likelihood population-effects (MLPE) models ([@r494]). These models accommodate the structure of dissimilarity matrices using random effects. For each species, we fit a MLPE model to the species' dissimilarity matrices based on geographic position and neutral genetic variation. Additionally, if adaptive loci were detected, we also fit a MLPE model to the species' dissimilarity matrices describing environmental and adaptive genetic variation. All predictor variables were _z_-transformed prior to model fitting to improve convergence. To test if the surrogates explained a substantial amount of genetic variation, we compared each model to its corresponding null model using $\chi^2$, and applied Bonferroni corrections.

Our second aim was to determine if the environmental and geographic variables were effective surrogates for adaptive and neutral genetic variation. For each species, we generated `r format(raptr.params.LST[[MODE]]$surrogacy.analysis$random.replicates, big.mark=',', scientific=FALSE)` prioritisations by randomly selecting different combinations of planning units that it occupied, and calculated the proportion of geographic variation and neutral genetic variation it secured. We repeated this processes, and calculated the proportion of environmental variation and adaptive genetic variation they secured.

We fit two full generalised linear models (GLM) with logit links. The first model was fit to the proportion of adaptive genetic variation secured in a prioritisation using the proportion of environmental variation also secured in the prioritisation. The second model was fit to the proportion of neutral genetic variation secured in a prioritisation using proportion of geographic variation also secured in the prioritisation. Both models also contained a variable indicating the species for which the prioritisations were generated, and an interaction term. They were subject to backwards step-wise term deletion routines to assess term significance. To assess the performance of the surrogates, we refit these models seperately for each species and computed the Cragg and Uhler's pseudo-R value (using the \texttt{pscl R} package; [@r492]).

Our third aim was to determine whether surrogate-based targets improved prioritisations. As previously mentioned, we generated prioritisations using different combinations of targets and conservation planning scenarios. We used `r (raptr.params.LST[[MODE]]$scenario.analysis$amount.target*100)` % amount-based targets in all prioritisations. Based on the results from our previous aim, we used `r (raptr.params.LST[[MODE]]$scenario.analysis$surrogate.target*100)` % surrogate-based targets and `r (raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` % genetic-based targets. We generated a single solution for each target/scenario combination, except for the "single-species (equal costs) with amount-based targets" combination for which we generated `r format(raptr.params.LST[[MODE]]$scenario.analysis$single.species.amount.replicates, big.mark=',', scientific=FALSE)` replicates because it had many optimal solutions. We computed the proportion of adaptive (if present) and neutral genetic variation secured for each species in the prioritisations.

We investigated the benefits of using surrogate-based targets in prioritisations using generalised linear mixed-effects models fit with logit links (GLMMs; using the \texttt{lme4 R} package; [@r469]). We fit a full model to the proportion of genetic variation secured in a given prioritisation. This model contained categorical variables indicating the combination of targets (amount-only, amount and surrogate targets, or amount and genetic targets), the scenario used to generate the prioritisation (single-species, multi-species, or multi-species with cost), the type of genetic variation measured (adaptive or neutral), and all interactions between them. Prioritisations generated for the same species were accomodated using a random intercept term. We subject the full model to a backwards step-wise term deletion routine to assess term significance, and conducted a post-hoc analysis on the minimal adequate model using Tukey contrasts with a Bonferroni correction (using the \texttt{multcomp R} package; [@r470]).

\showmatmethods
\showacknow
\pnasbreak
