---
title: "Are environmental and geographic variables effective surrogates for genetic variation in conservation planning?"
author: |
  | Jeffrey O. Hanson$^1$, Jonathan R. Rhodes$^2$, Cynthia Riginos$^1$, Hugh P. Possingham$^1$, Richard A. Fuller$^1$
  | $^1$School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  | $^2$School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
  | Correspondance should be addressed to jeffrey.hanson@uqconnect.edu.au
date: "`r format(Sys.time(), '%d %B %Y')`"
tags: ["conservation", "biodiversity", "AFLP", "reserve-selection"]
abstract: "Insert abstract here."
output:
  rmarkdown::pdf_document:
    toc: false
    toc_depth: 5
    keep_tex: yes
    fig_caption: yes
    includes:
      in_header: preamble-latex.tex
      after_body: postamble-latex.tex
fontsize: 11pt
documentclass: article
bibliography: Endnote_lib.bib
csl: reference-style.csl
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
setwd('..')
setwd('..')

# load .rda file
try(session::restore.session('data/results/results.rda'))

# set options
options(error=NULL)

```

<!--  title page -->
Short running title: Genetic surrogates in conservation \newline
Word count: XXXXX \newline
Number of references: XXXX \newline
Number of figures, table and text boxes: 4 \newline
\newpage


## Introduction
<!--  main text -->
The act of setting aside areas for preservation spearheads modern conservation efforts [@r440]. Protected area networks buffer species from anthropogenic impacts and provide places for populations of species to persist. Intra-specific genetic variation underpins species persistence, and there has been increasing interest in understanding how to design protected area networks in a way that maximizes this variation [@r67; @r3; @r61]. Broadly speaking, genetic variation can be classified as either adaptive or neutral [reviewed in @r48]. Adaptive genetic variation is associated with loci that affect fitness. By preserving existing patterns of adaptive variation, protected areas can ensure that populations with particularly beneficial adaptations are not lost [@r3]. In contrast, neutral genetic variation is associated with loci that do not affect fitness. Neutral variation often reflects the evolutionary history of different populations. By preserving existing patterns of neutral variation, protected areas can secure different genetic lineages and avoid the adverse effects of low genetic diversity [eg. inbreeding depression; @r67]. However, genetic data requires considerable investment to obtain and analyze. As a consequence, conservation planners do not typically use genetic data to inform prioritizations [@r61]. Thus prioritizations may fail to capture species' intra-specific genetic variation, and ultimately fail to deliver long-term conservation outcomes.

Recently, conservation scientists have been investigating surrogates for genetic variation to generate better prioritizations without the overheads associated with genetic data. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been used to guide reserve selection [eg. habitat conditions; @r247; @r2]. However, their effectiveness as surrogates in conservation planning remains unverified. On the other hand, neutral genetic variation arises from a reduction in gene flow between individuals. Surrogates for neutral variation are based on variables that predict the level of connectivity between planning units. For instance, the isolation by distance theory [IBD; @r259] predicts that populations further apart will have less connectivity, exchange less genetic material, and in turn have less genetic variation in common. Based on this theory, geographic distance-based surrogates have been found to improve prioritizations in an island network [@r200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain; eg. @r474]--the very systems commonly featured in real-world conservation planning exercises [eg. @r197]. Before conservation planners can reliably use surrogates for genetic data in conservation planning, the effectiveness of these surrogates needs to be verified. Otherwise, conservation planners may waste precious resources on protecting places that provide little benefit to biodiversity.

Here we investigate the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation in the context of conservation planning. Conventionally, prioritizations are generated by setting targets on the total amount of habitat that needs to be preserved for each species (amount-based targets), and finding the combination of planning units that fulfills all the targets for minimal cost [eg. using \texttt{Marxan}; @r22]. First, we aim to verify that environmental and geographic variables are effective surrogates for adaptive and neutral variation (respectively). We expect that prioritizations generated using surrogate-based targets will secure more genetic variation than prioritizations generated using only amount-based targets with the same number of planning units. Second, we aim to determine if the addition of surrogate-based targets to conservation planning exercises can result in more effective prioritizations. We expect that prioritizations generated using amount- and surrogate-based targets will sufficiently preserve the genetic variation of more species than those generated using only amount-based targets. If these surrogates can improve the intra-specific genetic variation captured in a prioritization, then conservation planners may use these surrogates when genetic data is not available, and in turn deliver more effective protected areas.

## Methods
### Study system
We used species distribution and genomic data collected by the IntraBioDiv project in the European Alps [@r451; see @r452 and @r478 for further explanation of data collection methods]. This data set was particularly well suited for our study because it contains genetic data for a diverse range of species that were collected using the same sampling regime. Additionally, the study area covers a broad geographic domain that encompasses attitudinal gradients. As part of this project, data were collected for 27 alpine plant species using a 20' longitude by 21' latitude grid (approx. 22.3 km $\times$ 25 km; Figure S1). Project members visited every second grid cell, and if a species was detected in a cell, samples were collected from three individuals. They genotyped samples using amplified fragment length polymorphisms [AFLP; @r453], and constructed matrices denoting the presence/absence of polymorphisms at loci for each species. Thus the data set contains information describing the genomic variation of individuals for each species present in every second grid cell. We used these grid cells as planning units to develop prioritizations. To reduce computational effort, we used only grid cells that contained at least a single individual ($n = `r nrow(grid.DF)`$). The majority of our analyses were performed using the \texttt{R} software package [version `r paste0(version$major, '.', version$minor)`; @r192].

### Landscape data
We examined the effectiveness of environmental and geographic surrogates for adaptive and neutral genetic variation (respectively). To describe the geographic location of each grid cell, we projected the grid into an equi-distant coordinate system ($\texttt{ESRI:102031}$), calculated their centroids, and extracted their two-dimensional coordinates. To describe the environmental characteristics of each grid cell, we obtained 19 bioclimatic variables [approx. 1 km resolution; @r33]. These layers and the grid cells were projected into an equal-area coordinate system ($\texttt{ESRI:102014}$). The bioclimatic variables were clipped to the extent of the grid and subject to a principal components analysis (PCA; using \texttt{ArcMap} version 10.2.2; Table S1). The first `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components (PCs) cumulatively explained `r round(pca.DF[[4]][surrogate.params.LST[[MODE]]$number.components],1)` % of the variation and were used to generate two new layers. We calculated the average value for each PC layer in each grid cell, and used these values to characterize the main themes of climatic variation among the grid cells (Figure S2). The cost of purchasing land for preservation often exerts a strong influence on which planning units are recommended for prioritization [reviewed in @r482]. To examine the potential effects of cost, we calculated the total human population density inside each planning unit [obtained at 1 km resolution from the Global Rural-Urban Mapping Project; \texttt{GRUMP V1}; @r472] and used this to represent acquisition cost.

### Adaptive and neutral genetic data
To investigate the effectiveness of the surrogates for adaptive and neutral genetic variation, we first needed to identify which of the sampled loci--if any--were under selection. We used outlier detection methods [compared in @r454; reviewed in @r480] to achieve this. The basic premise underpinning these analyses is that neutral loci are expected to exhibit a specific level of variation among populations ($F_{st}$), and loci that deviate from this expectation should be under selection. The advantage of these methods--in contrast with environmental association analyses [eg. @r475]--is that they do not use environmental data to identify loci under selection, which would have biased our analysis. In an earlier study, Bothwell _et al._ [-@r455] applied such outlier detection methods to one of the species in this data set (_Gentiana nivalis_). Here, we adopted a similar methodology and applied it to each of the 27 species in the data set.

First, we grouped individuals into genetic lineages for each species [using a methodology similar to that implemented in \texttt{prabclust R} package; @r479]. The purpose of this step was to avoid falsely classifying loci as adaptive. For each species, we ordinated the genomic data, and then fit models to the ordinated data to group individuals into populations. Specifically, we used non-metric multi-dimensional scaling [NMDS; implemented in the $\texttt{vegan R}$ package; @r458] using Gower distances [@r464; using the $\texttt{cluster R}$ package to accommodate sparsity; @r457] to transform the binary loci data into continuous variables (Table S2). To ensure that the ordinations described a sufficient amount of the genomic variation, we ran successive scaling analyses with increasing dimensionality until this was achieved (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$population.clustering$max.stress`; `r nmds.params.LST[[MODE]]$population.clustering$trymax` random starts for each analysis). We then fit a suite of Gaussian mixture models to the ordinations to estimate the probability that individuals belonged to different populations [assuming different covariance structures implemented in the \texttt{mclust R} package; @r477; @r476]. For each parametrization, we fit a set of models assuming varying numbers of total populations (\textit{G}= 1--`r max(mclust.params.LST[[MODE]]$G)`). We identified the most parsimonious model for each species (lowest Bayesian Information Criterion $BIC$ statistic; Figures S4), and used them to group individuals to populations (Figure S5).

After grouping individuals into populations, we identified the loci under selection (Table S3). Since outlier detection methods compare allele frequencies between populations, we were unable to examine species that contained a single population. These species were omitted from subsequent analysis ($n = `r sum(sapply(spp.Mclust.LST, function(x) return(ncol(x$mclust$z)==1)))`$). For the remaining species, we ran $\texttt{Bayescan}$ [version 2.1; 1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations; @r456]. To avoid false-positives, we omitted loci where the global frequency of the minor allele was $\geq 0.05$ (following guidelines in the user manual) and assumed they were neutral. Additionally, we omitted individuals from this analysis when their population membership was uncertain (maximum membership probability $< `r mclust.params.LST[[MODE]]$probability.threshold`$). We ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate to ensure convergence [FDR $\leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species (Figures S6--S32). For each species, we partitioned adaptive and neutral loci into separate data sets, and reran the successive nonmetric multidimensional scaling analyses (using parameters described above but with maximum stress values $\leq$ `r nmds.params.LST[[MODE]]$genetic.space$max.stress`) to derive continuous variables that described the main gradients of adaptive and neutral genetic variation separately (Table S2). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the individuals in each grid cell to express the typical genomic characteristics of individuals in the cells.

### Prioritizations
We used the unreliable representative and adequate prioritization (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve both a proportion of the species' range (using amount-based targets) and a proportion of the species' intra-specific variation [using space-based targets and attribute spaces; _sensu_ @r465]. We treated the environmental and geographic surrogate variables as separate attribute spaces. We also treated the adaptive (if detected) and neutral variables as separate attribute spaces for each species. We set the demand points for each species and each attribute space using the values associated with the planning units. For every prioritization we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritization secured to assess its performance. Prioritizations associated with negative values--because they secured only a very small amount of genetic variation--were replaced with zeros to facilitate statistical analysis. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the $\texttt{rapr}$ R package [@r465] and $\texttt{Gurobi}$ [@r463].

To address the first aim--to determine if the environmental and geographic variables are effective surrogates for adaptive and neutral genetic variation--we generated a collection of single-species prioritizations. For each species, we first generated two sets of prioritizations using increasing environmental and geographic-based targets (separately). Prioritizations were generated using the following surrogate-based targets: `r paste(rapr.params.LST[[MODE]]$surrogate.target,collapse=', ')`. We then create a set of null expectations with which to compare the surrogate-based prioritizations. These null expectations represented possible solutions obtained using conventional conservation planning methods that just use amount (eg. Marxan). For each species, for each target, we generated `r ` prioritizations by randomly selecting planning units occupied by the species. The number of randomly selected planning units was set to equal the number of planning units in the optimal prioritizations.

To address the second aim--to determine if the use of surrogate-based targets can actually improve prioritizations in real-world conservation planning scenarios--we investigated the effectiveness of surrogate-based targets using three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritizations for each species. However, most large-scale prioritizations involve a comprehensive set of species. By forcing solutions to secure individuals in multiple communities, this may result in prioritizations that secure a greater proportion of intra-specific genetic variation for each species. To test for this effect, our second scenario involved generating multi-species prioritizations. Finally, most large-scale prioritizations also explicitly consider opportunity cost to ensure that prioritizations are cost-effective. Our third scenario involved generating multi-species prioritizations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritizations using (1) only amount-based targets, (2) both amount- and surrogate-based targets, and (3) both amount- and genetic-based targets. The purely amount-based prioritizations served as our baseline--representing the prioritizations typically generated in conservation planning exercises. We used `r (rapr.params.LST[[MODE]]$scenario.analysis$amount.target*100)` % amount-based targets in all prioritizations to ensure an adequate proportion of habitat was secured for each species. Based on the previous analysis, we set surrogate-based targets as `r (rapr.params.LST[[MODE]]$scenario.analysis$surrogate.target*100)` % and genetic-based targets as `r (rapr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` % to ensure that most of the variation was secured for each species. To accomodate variation in the high number of optimal solutions for the single-species amount-based prioritizations, we generated `r (rapr.params.LST[[MODE]]$scenario.analysis$single.species.amount.replicates)` replicate prioritizations.

We then counted the number of species that were sufficiently represented in terms of their adaptive and neutral genetic variation (genetic variation secured $\geq$ `r (rapr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` %) for each prioritization. We used a global $G$ test to assess whether the proportion of species that met their genetic target in a given prioritization depended on the combination of targets used to generate the planning units (amount-based targets, amount and surrogate-based targets, amount and genetic-based targets), the planning scenario (single-species, multi-species, multi-species with cost), or the type of genetic variation measured (adaptive or neutral). We also conducted a post-hoc analysis involving pairwise $G$ tests (using the \texttt{RVAideMemoire R} package; r@483).

## Results
<!-- Surrogacy analysis -->
The environmental and geographic-based variables were effective surrogates for adaptive and genetic variation (Figure 1). The prioritizations generated using greater surrogate-based targets secured more genetic variation (XXXX). In addition, prioritizations generated using environmental targets often secured more adaptive genetic variation than prioritizations generated using geographic-based targets did neutral genetic variation (XXXX). Only particularly high geographic targets ($\geq 0.95$ %) were able to secure a moderate proportion of neutral variation ($\geq XXXX$ %). Overall, there was a non-linear relationship between the surrogate-based targets used to generate the prioritizations and the proportion of genetic variation they secured.

<!-- Scenario analysis -->
To determine if the addition of environmental and geographic targets could improve the effectiveness of prioritizations in a more realistic context, we generated prioritizations according to three different scenarios (Figure 2). The prioritizations generated under the single-species scenario showed similar selection frequencies across the study area--regardless of the combination of targets used to generate them. On the other hand, the prioritizations generated under the multi-species scenario showed different selection frequencies depending on the targets used to generate them. The addition of surrogate- or genetic-based targets resulted in prioritizations that contained substantially more planning units. The prioritizations generated under the multi-species with cost scenario had different spatial distributions to the previous two scenarios. The prioritizations generated under this third scenario concentrated conservation effort towards areas with higher elevation.

The inclusion of surrogate-based targets caused an improvement in the proportion of neutral variation secured in a prioritization (Figure 3). Specifically, the proportion of genetic variation secured by the prioritizations differed depending on the type of genetic variation measured (XXXX) and combination of targets used to generate the prioritizations (XXXX). No significant differences were found among the single-species, multi-species, and multi-species with cost contexts (XXXX).

## Discussion
<!-- introduction -->
We aimed to investigate the effectiveness of broad-scale environmental and geographic variables as surrogates for adaptive and neutral genetic variation in conservation planning. Our study is the first to address this hypothesis using moderately realistic planning scenarios. Overall, we found that environmental and geographic variables are effective surrogates for a few species when developing prioritizations. Furthermore, when developing prioritizations under various planning scenarios, we found that the addition of environmental and geographic targets resulted in prioritizations that sufficiently represented the genetic variation of a greater number of species. Our results suggest that while the environmental and geographic variables have little benefit for most species, they can make a large difference for some species. 

<!-- consequences for conservation planning -->
+ Typically, conservation planning exercises do not consider intra-specific variation when designing prioritizations. 
+ our results show that for most species, a sufficient proportion of their intra-specific variation can be secured using suitable amount-based targets alone.
+ in particular, we found that using suitable amount-based targets in a multi-species context with comprehensive set of target species resulted in a prioritization that adequately represented the adaptive genetic variation for nearly all the target species (panel c; Figure 3).
+ however, for some species, such a prioritization secures only a small proportion of their genetic variation.
+ we found that for some species, prioritizations generated using amount-based targets only secured a small amount of their neutral genetic variation (panels b, d, e; Figure 3).
+ in planning exercises where the decision maker does not have access to genetic data on their target species across the extent of the study area--ie. nearly all real world planning scenarios--the decision maker will not know which species are poorly represented in terms of their inta-specific variation.
+ the poorly represented species may be those of particular significant or they may not.
+ the decision maker has to weigh the increased costs of a prioritization that includes environmental and geographic surrogates against the risk that a prioritization that does not include these surrogates fails to represent species of conservation significance.
+ here we found that prioritizations generated using environmental and geographic surrogates have only minor increased costs (compare panels c and d; panels e and f; Figure 3).
+ overall, these results suggest that although surrogates are unnecessary for most species, they can be highly beneficial for a few species, the increased costs associated with using these surrogates is small, so generally using surrogates is a good idea.

<!-- comparison to previous studies that have investigated genetic studies -->
+ Previous studies on this data set have found correlations between environmental and genetic variation.
+ Yet, for most species, prioritizations generated using environmental surrogates did not secure substantially more genetic variation than randomly generated prioritizations with the same number of planning units.
+ Here we discuss some potential reasons for this disagreement.
+ 

+ This apparent disagreement can be attributed to multiple reasons.


+ These results highlight the importance of using prioritizations to assess the effectiveness of potential surrogates for conservation and not simply measuring the strength of correlation between potential surrogates and genetic variation.






<!-- limitations/future directions -->
We wish to alert our readers to several limitations associated with our analysis. First, the size of the planning units we used (23 $\times$ 25 km) is much larger than typically used in conservation planning scenarios (between 1--10 km^2). We used this resolution because the genetic data was collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis because this process would have  artificially introduced additional spatial auto-correlation into the dataset. Second, we used geographic distances as surrogates for connectivity between planning units. Although connectivity between areas can be modeled using isolation by resistance measures [@r473] derived from landscape data [eg. @r474], they are unlikely to be of use in broad-scale conservation exercises since they need to be tuned using species-specific parameters. Third, we used AFLP data to characterize the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @r480], our methodology would still have utilized only the main gradients of the genetic variation in order to generate prioritizations in a feasible amount of time. Overall, we are confident that the limitations associated with our study are relatively minor.

### Concluding remarks
The overarching aim of our study was to investigate the effectiveness of surrogates in conservation planning. We found that environmental and geographic surrogates are effective. However, they seem to be largely unnecessary in conservation planning scenarios because a comprehensive set of species can be used to achieve a similar result. Conservation planners are therefore urged to use a comprehensive set of species in prioritizations--one of the fundamental principals of conservation planning.

## Acknowledgements
JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.

## References
