---
title: "Environmental and geographic variables are effective surrogates for genetic variation in conservation planning"
author:
  - name: Jeffrey O. Hanson
    affiliation: a,1
  - name: Jonathan R. Rhodes
    affiliation: b
  - name: Cynthia Riginos
    affiliation: a
  - name: Hugh P. Possingham
    affiliation: a
  - name: Richard A. Fuller
    affiliation: a
address:
  - code: a
    address: School of Biological Sciences, The University of Queensland, Brisbane, QLD, Australia
  - code: b
    address: School of Geography, Planning and Environmental Management, The University of Queensland, Brisbane, QLD, Australia
corresponding_author:
  - code: 1
  - text: "To whom correspondence should be addressed. E-mail: jeffrey.hanson@uqconnect.edu.au"
author_contributions: |
  All authors developed the study and drafted the manuscript. JOH performed all analyses.
lead_author_surname: "Hanson"
abstract: |
  Protected areas buffer species from anthropogenic threats and provide places for processes to continue. However, evolutionary processes are not often explicitly considered when deciding where to site protected areas because genetic data require considerable resources to obtain and analyse. Here we show that environmental and geographic distance variables are effective surrogates for adaptive and neutral intra-specific variation for conservation planning, and that these surrogates can be used to deliver more effective solutions. We obtained distribution and genetic data from the international IntraBioDiv project for 27 species collected over the European Alps. Samples were collected using a gridded sampling scheme. We used these cells as planning units to generate solutions. For each species, we identified loci that were potentially under selection using outlier loci methods, and mapped their main gradients of adaptive and neutral genetic variation across the planning units. Our first analysis confirms that the the spatial patterns of adaptive and neutral genetic variation correlate with those of environmental and geographic variation (respectively). Our second analysis verifies that solutions that secure more adaptive genetic variation also tend to secure more adaptive variation, and similar patterns were found for neutral genetic variation and geographic variables. Our third analysis found that solutions generated using additional targets to explicitly secure a representative sample of the environmental and geographic variation across a species range resulted in solutions that secure more neutral genetic variation for most species. Additionally, we found that most solutions--even those generated without surrogate or genetic data--tended to secure most of the adaptive genetic variation for most species. Overall, our analysis provides a rigorous examination of two potential surrogates for genetic variation. 
significance: |
  Protected areas are used to preserve biodiversity patterns and processes. Since the resources available for conservation are limited, protected areas need to be sited in cost-effective locations. However, despite the stated emphasis on biodiversity processes, evolutionary processes are not often used to guide reserve selection. This is due to the fact that genetic data is often too costly to obtain and process. Here we found that environmental and geographic variables are cost-effective surrogates for genetic data. These variable can be used to deliver candidate reserve systems that secure the adaptive and neutral evolutionary processes for a greater number of species with minimal additional cost. This research highlights the importance of explicitly considering biodiversity processes during conservation planning exercises.
acknowledgements: |
  JOH is funded by an Australian Postgraduate Award (APA) scholarship. RAF has an Australian Research Council Future Fellowship. This work was supported by the Centre of Excellence for Environmental Decisions (CEED) and the Landscape Ecology and Conservation Group (LEC) at The University of Queensland.
keywords:
  - conservation
  - biodiversity
  - AFLP
  - genetic diversity
pnas_type: pnasresearcharticle
bibliography: references.bib
csl: reference-style.csl
output:
  rticles::pnas_article:
    includes:
      in_header: [preamble.tex]
---

<!-- NOTES -->
<!-- 150 word limit for abstract -->
<!-- 5000 word length limit -->

```{r, include=FALSE}
# set wd
knitr::opts_knit$set(root.dir=normalizePath('../..'))
```

```{r,include=FALSE}
# load .rda file
checkpoint::checkpoint('2016-08-03', R.version='3.3.1', scanForPackages=FALSE)
try(session::restore.session('data/final/results.rda'))

# set options
options(error=NULL)

# download basemap
data(countriesHigh)
countries.FPLY <- countriesHigh[
	countriesHigh$ADMIN %in% c(
		'Italy', 'Switzerland', 'France', 'Austria',
		'Germany', 'Slovenia', 'Croatia', 'Hungary',
		'Monaco', 'Germany', 'Slovakia', 'Czech Republic'
	)
,] %>% spFortify
grid.FPLY <- spFortify(grid.PLY)
grid.sub.FPLY <- spFortify(grid.sub.PLY)

# define ggplot2 color wheel function
gg_color_hue <- function(n) {
	hues <- seq(15, 375, length=n+1)
	hcl(h=hues, l=65, c=100)[1:n]
}
```

<!-- Introduction -->
Protected areas spearhead conservation efforts [@r440]. They buffer species from anthropogenic impacts, providing places for species to persist. Since the resources available for conservation are limited, protected areas need to be sited in places that fulfill conservation objectives for minimal cost [@r255]. To achieve this, conservation planning exercises often generate plans for entire networks of protected areas (prioritisations) to allocate resources cost-effectively and preserve broad-scale biodiversity processes. Intra-specific genetic variation is fundamental to long-term species persistence [@r165], and as a consequence, there has been increasing interest in designing prioritisations that capture and cultivate this aspect of biodiversity [@r67; @r3; @r61; @r499]. 

Broadly speaking, though the strength of natural selection is a continuous force, genetic variation can be classified as either adaptive or neutral [reviewed in @r480]. Adaptive genetic variation is associated with loci that significantly affect fitness. Typically, such "adaptive" loci are detected due to anomalous patterns among individuals [eg. @r456; @r487; @r480; @r490], though phenotypic-genetic associations are more likely to identify functionally important genomic regions for individual species [eg. @r500]. If they represent the full range of adaptive variation, protected areas can help ensure that populations with particularly beneficial adaptations are not lost [@r3; @r43]. In contrast, neutral genetic variation is associated with loci that do not significantly affect fitness, but instead reflects the evolutionary history of different populations. By representing the full range of neutral variation, protected areas can safeguard against the adverse effects of low genetic diversity [@rXXX]. Thus optimally sited protected area networks would aim to represent both adaptive and neutral patterns of genetic variation for species of conservation interest [@r67; @r3; @r43].

Recently, the use of surrogate data has been proposed to generate prioritisations that adequately preserve intra-specific genetic variation--without needing to directly utilise genetic data to achieve this. Since adaptive genetic variation is ultimately driven by extrinsic selection pressures, environmental variables have been proposed to guide reserve selection as surrogates for adaptive genetic variation [eg. environmental conditions; @r202; @r247; @r2]. By preserving a more representative sample of environmental conditions across the species' ranges, prioritisations could be generated that also capture more adaptive variation. However, the effectiveness of environmental surrogates in representing adaptive genetic variation remains unverified. Neutral genetic variation arises from a reduction in gene flow between individuals. Over the last few years, the field of phylogeography has predominantly focused on describing neutral genetic diversity, and so the landscape factors that affect neutral variation are relatively well understood [@r501], and potential surrogates for neutral variation have been based on variables that predict the levels of connectivity between planning units. For instance, isolation by distance [IBD; @r259] predicts that populations further apart will be more weakly connected, exchange less genetic material, and in turn have less genetic variation in common. Based on this idea, geographic surrogates based on distance have been found to improve conservation prioritisations in an island network [@r200]. However this remains untested in spatially contiguous systems where connectivity is complicated by additional factors [eg. terrain or habitat type; @r474; @r497]--the typical situations in most conservation planning exercises [eg. @r197].

Here we investigate the effectiveness of environmental and geographic surrogates for capturing adaptive and neutral genetic variation respectively in the context of conservation planning. We used distribution and genomic data for 27 alpine plants collected by the international IntraBioDiv project in the European Alps [@r451]. We show that for most species the spatial patterns of adaptive and neutral genetic variation correlated with environmental and geographic variables. We also show that for most species the level of association is strong enough to be operational for conservation planning. Finally, we demonstrate the using these surrogates results in prioritisations that secure more genetic variation than expected by chance.

Results {.unnumbered}
=====================

<!-- summary of methods -->
We detected putatively adaptive genetic variation in `r as.character(as.english(length(env.surrogacy.LST)))` of the 27 plant species. We used two outlier detection methods to detect loci under selection, and they returned reasonably consistent results (mean `r round(mean(spp.outlier.methods.congruence, na.rm=TRUE),2)` % loci per species assigned the same classification $\pm$ `r round(sd(spp.outlier.methods.congruence, na.rm=TRUE),2)` SD). Of the species that were associated with adaptive genetic variation, only a small proportion of loci were classified as being adaptive (mean `r round(mean(spp.outlier.loci.props, na.rm=TRUE),2)` % loci per species $\pm$ `r round(sd(spp.outlier.loci.props, na.rm=TRUE),2)` SD). After identifying the loci showing strong signals of selection, we used non-metric multi-dimensional scaling (NMDS) analyses to identify the main gradients of adaptive (if detected) and neutral variation for each species. Generally, only a small number of continuous dimensions was needed to sufficiently describe their patterns of adaptive (all $K = `r unique(na.omit(adaptive.k))`$; Table S2) and neutral genetic variation (minimum $K = `r min(neutral.k)`$; mean $K = `r round(mean(neutral.k),2)`$; maximum $K = `r max(neutral.k)`$; Table S2). The resulting ordinations were used to construct an adaptive (if detected) and neutral genetic space for each species. The spatial distribution of these genetic spaces (Figures S29--S54) generally showed substiantial spatial auto-correlation (eg. _Cerastium uniflorum_, Figure S34; _Dryas octopetala_, Figures S36), suggesting that the environmental and geographic variables may prove to be effective surrogates.

<!-- correlation analysis -->
We verified that the spatial patterns in environmental and geographic variation correlated with the broad-scale spatial patterns in adaptive and neutral variation for most species (respectively). We constructed dissimilarity matrices between the planning units. For each species, we constructed matrices expressing the differences between planning units as a function of their environmental characteristics and geographic position, and the adaptive (if detected) and neutral genetic characteristics between individuals. The spatial patterns of environmental variation was significantly correlated with that of adaptive genetic variation for `r as.character(as.english(sum(correlation.model.summary.DF$Environmental.corr.P<0.05,na.rm=TRUE)))` of the `r as.character(as.english(sum(!is.na(correlation.model.summary.DF$Environmental.corr.P))))` species associated with adaptive variation (_P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Environmental.corr.P<0.05)$Environmental.Marginal.R2),2)` SD for significant models). Geographic distance between planning units also correlated significantly with the spatial pattern of neutral genetic variation among planning units for `r sum(correlation.model.summary.DF$Geographic.corr.P<0.05,na.rm=TRUE)` of the `r sum(!is.na(correlation.model.summary.DF$Geographic.corr.P))` species (_P_ < 0.05; mean `r round(mean(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` marginal $R^2$ $\pm$ `r round(sd(filter(correlation.model.summary.DF, Geographic.corr.P<0.05)$Geographic.Marginal.R2),2)` SD for significant models). Thus, for most species, planning units that contained different environmental conditions contained individuals with different adaptive genetic characteristics, and planning units that were further apart tended to contain individuals with different neutral genetic characteristics. After verifying these patterns, our next step was to determine whether these variables are effective surrogates for conservation planning.
```{r, echo=FALSE, results='asis'}
curr.file <- tempfile(fileext='.tex')
x <- correlation.model.summary.DF %>%
	select(-contains('raw')) %>%
	mutate(
		Environmental.corr.P=pvalString(Environmental.corr.P),
		Geographic.corr.P=pvalString(Geographic.corr.P),
		Geographic.Marginal.R2=pvalString(Geographic.Marginal.R2),
		Geographic.Conditional.R2=pvalString(Geographic.Conditional.R2),
		Environmental.Marginal.R2=pvalString(Environmental.Marginal.R2),
		Environmental.Conditional.R2=pvalString(Environmental.Conditional.R2),
		blank=rep('', length(Species)),
		Species=paste0('\\textit{', gsub('\\ ', ' ', Species), '}')) %>%
	mutate(Environmental.Chisq=sapply(round(Environmental.Chisq,2), as.character)) %>%
	mutate(Geographic.Chisq=sapply(round(Geographic.Chisq,2), as.character)) %>%
	select(Species:Environmental.corr.P, Geographic.Marginal.R2:Geographic.corr.P) %>%
	latex(file=curr.file, 
		cgroup=c('', 'Environmental', 'Geographic'), n.cgroup=c(1, 4, 4), col.just=rep('c', 9),
		digits=2, rowname=NULL, first.hline.double=FALSE, booktabs=TRUE, here=TRUE,
		colheads=c('\\bfseries{Species}', '$MR^2$', '$CR^2$', '$\\chi^2$', '\\textit{P}', '$MR^2$', '$CR^2$', '$\\chi^2$', '\\textit{P}'),
		caption='Summary of maximum likelihood population-effects (MLPE) models. Briefly, for each species associated with potentially adaptive loci, a model was fit to correlate dissimilarities among planning units where the species in terms of the adaptive genetic characteristics of individuals in them with dissimilarities in terms of the environmental conditions inside the planning units (denoted in the environmental column group). Similarly, for each species, a model was fit to correlate dissimilarities among the planning units in terms of the neutral genetic characteristics of indiviudals in them with dissimilarities in terms of the geographic position of the planning units (denoted in the geographic column group). Data shows the results from comparing each model to its corresponding null model using $\\mathbf{\\chi^2}$ tests (1 degree of freedom), and the $\\mathbf{R^2}$ ($\\mathbf{MR^2}$) and conditional $\\mathbf{R^2}$ ($\\mathbf{CR^2}$) for each model.'
	)
readLines(curr.file)[-1] %>% paste0(collapse='\n') %>% 
	gsub(pattern='\\cline{1-', replacement='\\cline{2-6} \\cline{8-', fixed=TRUE) %>% 
	gsub(pattern='{c}{\\textit{P}}\\tabularnewline', replacement='{c}{\\textit{P}}\\T\\B\\tabularnewline', fixed=TRUE) %>% 
	gsub(pattern='\\toprule', replacement='', fixed=TRUE) %>%
	gsub(pattern='\\midrule', replacement='\\midrule', fixed=TRUE) %>%
	gsub(pattern='\\bottomrule', replacement='\\midrule', fixed=TRUE) %>%
	cat()
```

<!-- surrogacy analysis -->
For each species, we generated a suite of `r raptr.params.LST[[MODE]]$surrogacy.analysis$random.replicates*2` random prioritisations, and calcuated the proportion of environmental variation and adaptive (if present) genetic variation they sampled. We then repeated this process, and calculated the proportion of geographic variation and neutral genetic variation they sampled. Environmental and geographic variables were moderately effective predictors for genetic variation for the genetic variation represented by the randomly generated prioritisations (Figure 1). The relationship between the proportion of genetic variation and surrogate variation secured in the prioritisations varied among species for environmental surrogates of adaptive genetic variation (species $\times$ proportion of environmental variation interaction term; ${\chi^2}_{`r env.surrogacy.AOV$df[[4]]`} = {`r round(env.surrogacy.AOV$Resid..Dev[[4]],2)`}; P `r pvalString(env.surrogacy.AOV$Pr..Chi.[[4]])`$), and geographic surrogates of neutral variation (${\chi^2}_{`r round(geo.surrogacy.AOV$df[[4]],2)`} = `r round(geo.surrogacy.AOV$Resid..Dev[[4]],2)`$; $P$ `r pvalString(geo.surrogacy.AOV$Pr..Chi.[[4]])`). _Post-hoc_ analyses showed that the relationships for all species were significantly different to zero (environmental: minimum \textit{z} = `r round(max(env.surrogate.spp.DF$statistic),2)`, maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; geographic: minimum \textit{z} = `r round(min(geo.surrogate.spp.DF$statistic),2)`, maximum $P `r pvalString(max(env.surrogate.spp.DF$p.value))`$; Table S3). After establishing that the surrogate variables predicted how well prioritisations secured genetic variation, our final step was to determine if prioritisations were more effective when generated using surrogate-based targets.

```{r, echo=FALSE, message=FALSE, warning=FALSE, fig.align='center', fig.width=6, fig.height=4.5, out.width='3.42in', out.height='2.5in', fig.cap='Adaptive (a) and neutral (b) genetic variation represented in a suite of randomly generated prioritisations as a function of the environmental and geographic surrogates they also secured (respectively). Panels (c) and (d) show the distribution of $R^2$ values for models using each surrogate to explain genetic variation.'}
# make figure
p1 <- ggplot(aes(x=surrogate.held,y=genetic.held), data=env.surrogacy.DF) +
	xlab('Environmental variation\nsecured (%)') + 
	ylab('Adaptive genetic\nvariation secured (%)') +
	theme_classic() + theme(axis.line.x=element_line(color='black'),
		axis.line.y=element_line(color='black')) +
	coord_cartesian(xlim=c(0, 1)) +
	scale_x_continuous(limits=c(0,1), labels=function(x) {x*100}) +
	scale_y_continuous(limits=c(0,1), labels=function(x) {x*100})
for (i in seq_along(env.surrogacy.LST))
	p1 <- p1 + stat_smooth(method='glm', method.args=list(family='binomial'),
		data=filter(env.surrogacy.DF, Species==unique(Species)[i]),
		fullrange=TRUE, se=FALSE, colour='black')
p1 <- p1 + geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(a)')

p2 <- ggplot(aes(x=surrogate.held,y=genetic.held), data=geo.surrogacy.DF) +
	xlab('Geographic variation\nsecured (%)') + 
	ylab('Neutral genetic\nvariation secured (%)') +
	theme_classic() + theme(axis.line.x=element_line(color='black'),
		axis.line.y=element_line(color='black', colour='transparent')) +
	coord_cartesian(xlim=c(0, 1)) +
	scale_x_continuous(limits=c(0,1), labels=function(x) {x*100}) +
	scale_y_continuous(limits=c(0,1), labels=function(x) {x*100})
for (i in seq_along(geo.surrogacy.LST))
	p2 <- p2 + stat_smooth(method='glm', method.args=list(family='binomial'),
		data=filter(geo.surrogacy.DF, Species==unique(Species)[i]),
		fullrange=TRUE, se=FALSE, colour='black')
p2 <- p2 + geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(b)')

p1.hist <- ggplot(aes(r2CU),
		data=filter(env.surrogate.spp.DF, !is.na(r2CU))) +
	geom_histogram(fill='grey10', bins=5) +
	ylab('\nFrequency') + xlab(expression(Environmental~variables~(R^2))) +
	theme_classic() + theme(axis.line.x=element_line(color='black'),
		axis.line.y=element_line(color='black'),
		plot.background=element_rect(fill='transparent', colour='transparent')) +
	geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(c)') +
	scale_y_continuous(labels=function(x) {formatC(x, digits=2, format='f')}) +
	scale_x_continuous(limits=c(0, 1), labels=function(x) {formatC(x, digits=2, format='f')})

p2.hist <- ggplot(aes(r2CU),
		data=filter(geo.surrogate.spp.DF, !is.na(r2CU))) +
	geom_histogram(fill='grey10', bins=10) +
	ylab('\nFrequency') + xlab(expression(Geographic~variables~(R^2))) +
	theme_classic() + theme(axis.line.x=element_line(color='black'),
		axis.line.y=element_line(color='black'), 
		plot.background=element_rect(fill='transparent')) +
	geom_text(x=-Inf, y=Inf, hjust=-0.25, vjust=1.5, label='(d)') +
	scale_y_continuous(labels=function(x) {formatC(x, digits=2, format='f')}) +
	scale_x_continuous(limits=c(0, 1), labels=function(x) {formatC(x, digits=2, format='f')})

# render figure
grid.arrange(p1,p2,p1.hist,p2.hist,nrow=2)
```

<!-- scenario analysis -->
To determine whether using environmental and geographic targets could improve the effectiveness of prioritisations in representing genetic variation, we generated prioritisations using (i) "amount targets" reflecting the traditional approach of representing a certain proportion of each species' goegraphic distribution, (ii) "amount and surrogate targets" where we also targeted representation of the environmental and geographic surrogates as well as a certain proportion of each species' geographic distribution, and (iii) "amount and genetic targets" where we targeted representation of the directly measured genetic variation as well as a certain proportion of each species' geographic distribution (Figure 3). To determine if these patterns were robust, for each of the three combinations of the targets, we generated prioritsations under four scenarios: (i) single-species with equal costs, (ii) single species with opportunity costs, (iii) multi-species with equal costs, and (iv) multi-species with opportunity costs.

```{r, echo=FALSE, warning=FALSE, message=FALSE, fig.align='center', fig.width=(5*0.9), fig.height=(7.1*0.9), out.width='3.42in', out.height='4in', fig.cap='The effectiveness of prioritizations at securing genetic variation. Box plots show the proportion of adaptive and neutral genetic variation secured for each species using different targets and under different scenarios. Red lines show the target proportion of genetic variation to secure a representative sample.'}
# prepare data for plotting
scenario.SDF <- scenario.DF %>% 
	group_by(Prioritisation, Context, Metric, Species) %>% 
	summarize(genetic.held=mean(genetic.held, na.rm=TRUE)) %>%
	data.frame()

# make plot
suppressWarnings({
	ggplot(aes(x=Prioritisation,y=genetic.held),
		data=scenario.SDF) +
		geom_boxplot(position=position_dodge(0.9)) +
		scale_fill_manual(name='Prioritisation',
			values=c('grey80','grey50','grey20')) +
		geom_abline(slope=0, color='red', linetype='dashed',
			intercept=raptr.params.LST[[MODE]]$scenario.analysis$genetic.target) +
		ylab('Genetic variation secured (%)') +
		xlab('') +
		scale_y_continuous(breaks=c(0,0.25,0.5,0.75,1)) +
		theme_classic() +
		theme(strip.background = element_rect(fill='grey20'),
			strip.text = element_text(color='white', size=9),
			panel.border = element_rect(color='black', fill=NA),
			axis.text.x=element_text(angle=45, hjust=1, vjust=1, size=8),
			axis.text.y=element_text(size=8)) +
		facet_grid(Context ~ Metric)
})
```

The proportion of genetic variation secured in a prioritisation depended on the targets used to generate the prioritisation (${\chi^2}_{`r scenario.ANOVA.4.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.4.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.ANOVA.4.DF$p.value[[2]])`)$. Most notably, prioritisations based on the surrogate variables represented significantly more genetic variation than amount-based prioritisations (`r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount & surrogate\ntargets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount & surrogate\ntargets')$genetic.held*100),2)` overall genetic variation secured versus `r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount targets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount targets')$genetic.held*100),2)`; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[1]],2)`$; $P = `r pvalString(scenario.GLHT.2.DF$p.value[[1]])`$), and were not distinguishable from prioritisations based on measured genetic data (`r round(mean(filter(scenario.sub.DF,Prioritisation=='Amount & genetic\ntargets')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Prioritisation=='Amount & genetic\ntargets')$genetic.held*100),2)`; $Z_1 = `r round(scenario.GLHT.2.DF$statistic[[3]],2)`$; $P = `r pvalString(scenario.GLHT.2.DF$p.value[[3]])`$). These results indicate that environmental and geographic surrogates in this case far outperform traditional amount-based conservation planning, and perform as well as a conservation plan based directly on measured genetic data.

Overall, the prioritisations tended to secure more adaptive then neutral genetic variation (adaptive: mean `r round(mean(filter(scenario.sub.DF,Metric=='Adaptive variation')$genetic.held*100),2)` % $\pm$ `r round(sd(filter(scenario.sub.DF,Metric=='Adaptive variation')$genetic.held*100),2)` secured across all prioritisations, neutral: mean `r round(mean(filter(scenario.sub.DF,Metric=='Neutral variation')$genetic.held*100),2)` $\pm$ `r round(sd(filter(scenario.sub.DF,Metric=='Neutral variation')$genetic.held*100),2)`; ${\chi^2}_{`r scenario.ANOVA.3.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.3.DF$statistic[[2]],2)`$; _P_ `r pvalString(scenario.ANOVA.3.DF$p.value[[2]])`). Thus, regardless of the targets used to generate a prioritisation, or the planning context under which the prioritisation was generated, prioritisations tend to secure more adaptive than neutral genetic genetic variation.

The average proportion of genetic variation secured by prioritisations varied under different scenarios (${\chi^2}_{`r scenario.ANOVA.2.DF$Chi.Df[[2]]`} = `r round(scenario.ANOVA.2.DF$statistic[[2]],2)`$; _P_ `r pvalString(scenario.ANOVA.2.DF$p.value[[2]])`). Both the multi-species (mean `r round(mean(filter(scenario.sub.DF,Context=='Multi-species\n(equal costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Multi-species\n(equal costs)')$genetic.held*100),2)`) and multi-species with cost (mean `r round(mean(filter(scenario.sub.DF,Context=='Multi-species\n(opportunity costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Multi-species\n(opportunity costs)')$genetic.held*100),2)`) were both found to secure more genetic variation than the single species scenario (mean `r round(mean(filter(scenario.sub.DF,Context=='Single-species\n(equal costs)')$genetic.held*100),2)` % genetic variation secured $\pm$ `r round(sd(filter(scenario.sub.DF,Context=='Single-species\n(equal costs)')$genetic.held*100),2)`; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[2]],2)`$; $P `r pvalString(scenario.GLHT.1.DF$p.value[[2]])`$; $Z_1 = `r round(scenario.GLHT.1.DF$statistic[[1]],2)`$; $P = `r pvalString(scenario.GLHT.1.DF$p.value[[1]])`$ respectively). These results suggest that on average the multi-species prioritisations can secure more genetic variation than single-species prioritisations--regardless of targets used to generate them.

Discussion {.unnumbered}
========================

<!--  introduction -->
We aimed to investigate the effectiveness of broad-scale environmental and geographic variables as surrogates for adaptive and neutral genetic variation in conservation planning. Our study is the first to address this aim using a diverse set of species and calibrated against field measurements of genetic variation. The spatial patterns of environmental and geographic surrogates was strongly correlated with the patterns of adaptive and neutral genetic variation for most species, and environmental and geographic targets resulted in prioritisations that secured a greater proportion of adaptive and neutral genetic variation than traditional conservation planning for most species. These environmental and geographic surrogates are based on freely available data, and could be calculated for any study region across the world.

<!-- surrogate/genetic dissimilarity matrix correlations -->
Our analyses showed that differences in the environmental conditions and geographic position among the planning units tended to correlate with genetic differences among the individuals in the planning units (Table 1). For both surrogates and their corresponding models, the conditional $R^2$ values (environmental vs. adaptive genetic variation: mean `r round(mean(correlation.model.summary.DF$Environmental.Conditional.R2,na.rm=TRUE),2)` conditional $R^2$ `r round(sd(correlation.model.summary.DF$Environmental.Conditional.R2,na.rm=TRUE),2)` $\pm$ SD; geographic vs. neutral genetic variation: mean `r round(mean(correlation.model.summary.DF$Geographic.Conditional.R2,na.rm=TRUE),2)` conditional $R^2$ `r round(sd(correlation.model.summary.DF$Geographic.Conditional.R2,na.rm=TRUE),2)` $\pm$ SD) were generally slightly higher than the marginal $R^2$ values (environmental vs. adaptive genetic variation: mean `r round(mean(correlation.model.summary.DF$Environmental.Marginal.R2,na.rm=TRUE),2)` marginal $R^2$ `r round(sd(correlation.model.summary.DF$Environmental.Marginal.R2,na.rm=TRUE),2)` $\pm$ SD; geographic vs. neutral genetic variation: mean `r round(mean(correlation.model.summary.DF$Geographic.Marginal.R2,na.rm=TRUE),2)` marginal $R^2$ `r round(sd(correlation.model.summary.DF$Geographic.Marginal.R2,na.rm=TRUE),2)` $\pm$ SD). This result suggests that while variation in the surrogate variables correlates with genetic variation, both forms of genetic variation contain additional spatially auto-correlated variation that is not explained by these surrogates. In particular, the differences between the conditional and marginal $R^2$ values for the models examining the effectiveness of geographic variables suggest that this unexplained spatial auto-correlation may be non-linear or driven by a few neighboring planning units with genetically similar individuals. Similarly, other studies have also reported strong patterns of spatial auto-correlation in patterns of genetic variation [eg. @r502]. 

<!-- surrogate/genetic held correlations -->
Our results demonstrate that environmental and geographic surrogates are strong enough to be operational (Figure 2). Generally, the environmental and geographic surrogates variables were equally effective surrogates for adaptive (mean `r round(mean(env.surrogate.spp.DF$r2CU),2)` $R^2$ $\pm$ `r round(sd(env.surrogate.spp.DF$r2CU),2)` SD; Figures 2c and S56) and neutral genetic variation (mean `r round(mean(geo.surrogate.spp.DF$r2CU),2)` $R^2$ $\pm$ `r round(sd(geo.surrogate.spp.DF$r2CU),2)` SD; Figures 2d and Figure S57), although for a few species, the geograsphic surrogate performed poorly (eg. _Gypsophila repens_: $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Gypsophila_repens')$Geographic.r2CU,2)`; _Luzula alpinopilosa_: $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Luzula_alpinopilosa')$Geographic.r2CU,2)`; _Gentiana nivalis_ $R^2$ = `r round(filter(surrogate.spp.DF, Species=='Gentiana_nivalis')$Geographic.r2CU,2)`). The bimodal distribution of explained variation (Figure 2d) suggests that geographic position alone may not be a suitable surrogate for neutral variation in conservation planning for all species. Indeed, geographic distances will be a poor surrogate for genetic lineages in the process of sympatric speciation, connectivity between areas can be strongly affected by local topology, and some of species have dispersal modes that are non-linearly related to geographic distance (eg. wind dispersing plants can have high connectivity between nearby and distant areas; @r451). Overall, however, such species were the exception in our analysis, and both simple surrogates performed moderately well.

<!-- suitable targets for surrogates -->
In general, adaptive variation accumulated more quickly with environmental representation than neutral variation did with geographic representation (Figures 2a--2b and S56--S57). To secure at least `r raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100` % of each species' adaptive genetic variation, prioritisations needed to sample on average `r round(mean(surrogate.repr.prop.DF$environmental.held*100,na.rm=TRUE),2)` % $\pm$ `r round(sd(surrogate.repr.prop.DF$environmental.held*100,na.rm=TRUE),2)` SD of the environmental conditions found across the planning units occupied by the species. On the other hand, to secure a representative sample of the species' neutral genetic variation ($\geq$ `r raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100` %), prioritisations needed to sample `r round(mean(surrogate.repr.prop.DF$geographic.held*100,na.rm=TRUE),2)` % $\pm$ `r round(sd(surrogate.repr.prop.DF$geographic.held*100,na.rm=TRUE),2)` SD of the variation in geographic position among the planning units occupied by the species. These results suggest that geographic surrogates may require higher targets than environmental surrogates to generate prioritisations that secure representative sample of genetic variation. Additionally, these results suggest using surrogate-based targets alone to generate prioritisations may be insufficient to secure a representative sample of species' genetic variation. For all species, it was possible to generate solution that secured a high proportion of the surrogate variable (> 90 %) but only a small proportion of adaptive or neutral genetic variation (< 20 %; Figures S56--S57). Thus prioritisations should be generated using a combination of amount-based targets--as is conventionally used in reserve selection processes--and also surrogate-based targets.

<!-- adaptive genetic variation secured in prioritisations-->
Our results suggest that conservation planners can secure a reasonably representative sample of the intra-specific adaptive genetic variation using only amount-based targets. Since the spatial patterns of adaptive genetic variation clustered into a few distinct groups for most species (eg. _Carex semipervens_ and _Cerastium uniflorum_; Figures S33 and S34 respectively), a prioritisation would only need to select a planning unit in each cluster to secure a representative sample. As a consequence, the single-species prioritisations generated using amount targets and without opportunity cost data were often able to achieve high representation because they were essentially random selection (Figure 4a). To a lesser extent, this is also true for the single-species prioritisations generated with amount targets and consideration to opportunity cost (Figure 4c), since the costs were fairly homogenous among planning units with the exception of a few planning units in lowland areas. Overall--and contrary to our original hypotheses--it seems that conventional reserve selection methods may secure a representative sample of species' adaptive genetic variation if a comprehensive set of species is used.

<!--  limitations/ weaknesses -->
There are several limitations associated with our analysis. First, the size of the planning units we used (23 $\times$ 25 km) is much larger than typically used in regional conservation planning scenarios (1--10 km$^2$). We used this resolution because the genetic data were collected at this scale. Whilst we could have interpolated the genetic data to a finer resolution and used smaller planning units, this would have biased our analysis by artificially introducing additional spatial auto-correlation. Second, we used geographic distances as surrogates for neutral genetic variation, but connectivity in this landscape may be better described using distances that incorporate information on topography. However, such distances require specific-specific scaling to adequate describe their connectivity [eg. due to different dispersal modes; @r497], and so cannot easily be utilised in large-scale planning exercises. Third, we used amplified fragment length polymorphism (AFLP) data to characterise the main gradients of genetic variation. Whilst next-generation sequencing provides higher resolution genetic information than AFLP data [reviewed in @r480], no multi-species genomic data set with similar properties currently exists, and our methodology would still have utilised only the main gradients of the genetic variation to generate prioritisations in a feasible amount of time.

<!--  conclusion -->
Broad-scale environmental and geographic variables are generally effective surrogates for representing adaptive and neutral genetic variation for most species in our study system. Moreover, data to calculate the surrogate variables are freely available for any location in the world. Careful use of such surrogates in conservation planning could vastly improve the chances of long-term biodiversity persistence for relatively additional cost.

Materials and Methods {.unnumbered}
===================================

Study system {#study-system .unnumbered}
----------------------------------------

We used species distribution and genomic data collected by the international IntraBioDiv project in the European Alps [@r451; see @r452 and @r478 for further explanation of data collection methods]. This data set contains information for 27 alpine plant species (Figure 2a). This data set has previously been used to explore patterns of adaptive [@r484; @r451; @r475; @r455; @r485] and neutral genetic variation [@r478; @r451; @r486], and to show that species richness is not an effective surrogate for genetic diversity in conservation planning [@r486]. It was collected using a $20^{\prime}$ longitude by $21^{\prime}$ latitude grid (approx. 22.3 km $\times$ 25 km; Figure S1). Project members visited every second grid cell, and if a species was detected in a cell, samples were collected from three individuals. They genotyped samples using amplified fragment length polymorphisms [AFLP; @r453], and constructed matrices denoting the presence/absence of polymorphisms at loci for each species (mean `r round(mean(sapply(spp.StructureData.LST, structurer:::n.loci)),2)` $\pm$ `r round(sd(sapply(spp.StructureData.LST, structurer:::n.loci)),1)` SD markers genotyped per species; see Table 1 in Alvarez _et al._ [@r478] for the number markers for each species). Thus the data set contains information describing the genomic properties of individuals for each species present in every second grid cell. We used this data set because it comprised of comparable genetic data for a range of species with different evolutionary and life histories collected using a standardised sampling scheme, and pertained to a study area with substantial environmental variation to test the effectiveness of environmental variables. 

```{r, echo=FALSE, fig.align='center', fig.width=6, fig.height=1.5, out.width='3.42in', height='2.5in', fig.cap="Map of the study area. Squares denote planning units. Panel (a) depicts patterns of species riches. Planning units with a brighter color are inhabited by more species. Panel (b) shows the distribution of opportunity cost across the study area (computed as total population density) plotted using a quantile-based color ramp."}

# calculate species richness
grid.PLY$Species_richness <- grid.PLY@data %>%
	select(5:(4+n_distinct(spp.samples.DF$species))) %>% as.matrix() %>% rowSums()
	
panel.DF <- grid.sub.FPLY %>% 
	summarize(lat=min(lat)+(diff(range(lat))*0.96),
		long=min(long)+(diff(range(long))*0.04))
	
# plot species richness
p1 <- ggplot() +
	geom_polygon(data=countries.FPLY, aes(x=long, y=lat, group=group),
		fill='grey85', color='grey70') +
	geom_polygon(data=spFortify(grid.PLY), aes(x=long, y=lat, 
		group=group, fill=Species_richness), color=NA) +
	scale_fill_gradientn(name='Richness',
		colors=brewer.pal(5, 'YlGn'),
		guide=guide_colorbar(ticks=element_line(color='black'), border=element_line(color='black'))
	) +
	theme_classic() +
	theme(
		axis.ticks=element_blank(), axis.text=element_blank(), axis.line=element_blank(), plot.margin = unit(c(0, 0, 0, 0), "lines"), axis.title=element_blank(),
		panel.border=element_rect(color='black', fill=NA, size=1)
	) +
	coord_cartesian(
		xlim=buffered.range(grid.FPLY$long, 0),
		ylim=buffered.range(grid.FPLY$lat, 0)
	) + geom_text(aes_string(x='long',y='lat'), data=panel.DF, label='(a)')
	
qn01 <- scales::rescale(quantile(grid.PLY@data$pop.density, seq(0,1,length.out=9), na.rm=TRUE))
p2 <- ggplot() +
	geom_polygon(data=countries.FPLY, aes(x=long, y=lat, group=group),
		fill='grey85', color='grey70') +
	geom_polygon(data=spFortify(grid.PLY), aes(x=long, y=lat, 
		group=group, fill=pop.density), color=NA) +
	scale_fill_gradientn(name='Cost',
		colors=brewer.pal(5,'YlOrRd'),
		values=qn01,
		guide=guide_colorbar(ticks=element_line(color='black'), border=element_line(color='black'))
	) +
	theme_classic() +
	theme(
		axis.ticks=element_blank(), axis.text=element_blank(), axis.line=element_blank(), plot.margin = unit(c(0, 0, 0, 0), "lines"), axis.title=element_blank(),
		panel.border=element_rect(color='black', fill=NA, size=1)
	) +
	coord_cartesian(
		xlim=buffered.range(grid.FPLY$long, 0),
		ylim=buffered.range(grid.FPLY$lat, 0)
	)  + geom_text(aes_string(x='long',y='lat'), data=panel.DF, label='(b)')
	
grid.arrange(p1,p2,nrow=1)
```

Landscape data {#landscape-data .unnumbered}
--------------------------------------------

We adopted the sampling grid used by the IntraBioDiv project as planning units to develop prioritisations, omitting cells that were surveyed and not found to contain individuals belonging to any of the study species to reduce computational burden (`r nrow(grid.DF)` grid cells used in analysis). To examine the potential effects of variation in cost (see below), we calculated the total human population density inside each planning unit [obtained at 1 km$^2$ resolution from the Global Rural-Urban Mapping Project; \texttt{GRUMP V1}; @r472] and used this to represent acquisition cost (Figure 2b). Unless otherwise specified, all computational, spatial, and statistical analysis were conducted in R [version `r paste0(R.version$major, '.', R.version$minor)`; @r192].

To describe the geographic location of each planning unit, we projected the grid into an equi-distant coordinate system (Europe Equidistant Conic; $\texttt{ESRI:102031}$), calculated the centroid of each grid cell, and extracted their two-dimensional coordinates. Thus we constructed two-dimensional geographic space--wherein each planning unit occupies a single point--as a potential surrogate for neutral genetic variation. We expected planning units further apart from each other in this geographic space to contain individuals with increasingly different patterns in their neutral loci.

To describe the environmental characteristics of each planning unit, we obtained data for 19 bioclimatic variables across the extent of the study area [at $30^{\prime\prime}$ resolution; @r33]. We used this data set because it is commonly to map habitat suitability in conservation planning [@r493]. These layers and the planning units were projected into an equal-area coordinate system (Europe Lambert Conformal Conic; $\texttt{ESRI:102014}$). To reduce dimensionality, the 19 bioclimatic variables were clipped to the planning units and subjected to a principal components analysis (PCA; using \texttt{ArcMap} version 10.2.2; Table S1). The first `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components (PCs) cumulatively explained `r round(pca.DF[[4]][surrogate.params.LST[[MODE]]$number.components],1)` % of the variation and were used to generate new layers. We calculated the average value for each of the `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))` principal components in each planning unit, and used these values to characterise climatic variation among the planning units (Figure S2). Thus we constructed a `r as.character(as.english(surrogate.params.LST[[MODE]]$number.components))`-dimensional environmental space--wherein each planning unit occupies a single point--as a potential surrogate for adaptive variation. We expect planning units further apart from each other in this environmental space to contain individuals with increasingly different patterns in their adaptive loci.

Adaptive and neutral genetic data {#adaptive-and-neutral-genetic-data .unnumbered}
----------------------------------------------------------------------------------

To investigate the effectiveness of our supposed surrogates for adaptive and neutral genetic variation, we first needed to identify which of the sampled loci are adaptive. We used two outlier detection methods to achieve this [an individual- and a population-level method as recommended by @r495]. The basic premise underpinning such methods is that neutral loci are expected to exhibit a specific level of variation among populations, and loci that deviate from this expectation are likely to be under selection [reviewed in @r480]. The advantage of these methods--in contrast with environmental association analyses [@r489]--is that they do not use environmental data to identify loci under selection, which would have introduced circularity into our analysis. Loci identified as outliers in both analyses were treated as adaptive and the remainder were treated as neutral. To minimise false-positives, we omitted loci from both outlier detection methods where the global frequency of the minor allele was less than 10 % and assumed they were neutral.

The first outlier detection method we used involved fitting multinominal-Dirichlet models implemented in \texttt{BayeScan} [version 2.1; @r456]. We adopted a similar methodology to Bothwell _et al._ [-@r455] and applied it to each of the species in the data set. Following their methodology, we first grouped conspecifics into genetic lineages to avoid falsely classifying loci as adaptive. To complete this first step, for each species, we fit the admixture and correlated alleles model implemented in \texttt{Structure} [version 2.3.4; @r461; @r462; `r structure.params.LST[[MODE]]$numruns` replicates per K; `r structure.params.LST[[MODE]]$admburnin` admixture burnin iterations; `r structure.params.LST[[MODE]]$burnin` burnin iterations; `r structure.params.LST[[MODE]]$numreps` iterations used for inference] using the number of populations for each species previously determined by Alvarez _et al._ [-@r478]. We used \texttt{ClumPP} [version 1.1.2, @r459] to combine replicate runs to group individuals into populations (greedy algorithm using the $G^{\prime}$ static; `r clumpp.params.LST[[MODE]]$repeats` iterations). We then ran $\texttt{Bayescan}$ [@r456] for each species using these clusters (1:1 prior odds; `r bayescan.params.LST[[MODE]]$nbp` pilot runs; `r bayescan.params.LST[[MODE]]$burn` burn-in iterations; `r bayescan.params.LST[[MODE]]$n` post-burn-in iterations thinned by `r bayescan.params.LST[[MODE]]$thin` iterations). Additionally, we omitted individuals from the \texttt{Bayescan} analysis if their population membership was uncertain (maximum membership probability $< `r structure.params.LST[[MODE]]$probability.threshold`$). We ran `r bayescan.params.LST[[MODE]]$reps` replicates per species using a suitable false discovery rate [$q \leq `r bayescan.params.LST[[MODE]]$fdr`$; @r466]. 

The second outlier detection method involved fitting PCAs to identify outlier loci [using methods implemented in the \texttt{pcadapt R} package; version 3.0; @r487; @r496]. To enable comparisons between the two outlier detection analyses, we used the same individuals in this analysis as used in the \texttt{BayeScan} analysis. For each of the 27 species, we first imputed missing data to permit further analysis. Specifically, the missing value for a given locus for a given individual was replaced with the average frequency of that locus among all conspecifics. Next, we ran a PCA and extracted the minimum number of components needed to secure the over-arching population level variation among the loci (`r pcadapt.params.LST[[MODE]]$min.variance.explained*10` % of the variation in loci). We then used Mahalanobis distances [@r398] to compute _P_ values and then _q_-values [using the \texttt{qvalue R} package; @r491]. Outlier loci were detected using a suitable false discovery rate ($q < `r pcadapt.params.LST[[MODE]]$alpha.level`$). 

After classifying loci as adaptive or neutral, we mapped the main gradients of the adaptive (if detected) and neutral genetic variation for each species. For each species, we discarded the population groupings, and partitioned adaptive and neutral loci into separate matrices. We applied separate non-metric multi-dimensional scaling analyses to each matrix [NMDS; implemented in the \texttt{vegan R} package; @r458] using Gower distances [@r464; using the \texttt{cluster R} package to accommodate sparsity; @r457] to derive continuous variables that described the main gradients of adaptive and neutral genetic variation (separately) for each species (Table S2). To ensure that the ordinations described a sufficient amount of the genomic variation, we ran successive scaling analyses with increasing dimensionality until a sufficient stress value was obtained (maximum stress value $\leq$ `r nmds.params.LST[[MODE]]$max.stress`; `r nmds.params.LST[[MODE]]$trymax` random starts for each analysis). Since each grid cell had up to three samples per species, we used the average of the ordinated values associated with the individuals in each grid cell to express the typical genomic characteristics of individuals in the cells.

The previous set of analyses resulted in an adaptive (if detected) and neutral genetic space for each species. For a given species, each planning unit occupied by the species was associated with a multi-dimensional point in the species' neutral genetic space. Planning units that were closer together in this space are occupied by individuals with more similar polymorphisms in their neutral loci. Similarly, if the species was associated with adaptive loci, each planning unit occupied by the species was associated with a multi-dimensional point in the species' adaptive genetic space. By spreading out conservation effort across a given genetic space, and in turn securing planning units occupied by individuals with increasingly different polymorphisms, prioritisations can secure more genetic variation [see @r465 for discussion on attribute spaces].

Prioritisation method {#prioritisation-method .unnumbered}
----------------------------------------------------------

We used the unreliable representative and adequate prioritisation (URAP) formulation of the reserve selection problem [@r465]. This formulation can identify the minimum number of planning units required to preserve both a proportion of the species' range (using amount-based targets) and a proportion of variation found across the species' range [using space-based targets and attribute spaces; _sensu_ @r465]. We treated the environmental and geographic surrogate variables as separate attribute spaces. We also treated the adaptive (if detected) and neutral variables as separate attribute spaces for each species. We set the demand points for each species and each attribute space using the values associated with the planning units. For every prioritisation we generated, we calculated the proportion of the adaptive (if detected) and neutral genetic variation that every prioritisation secured to assess its performance. Prioritisations associated with negative values--because they secured only a very small amount of genetic variation--were replaced with zeros to facilitate statistical analysis. We solved all reserve selection problems to within `r (gurobi.params.LST[[MODE]]$MIPGap*100)` % of optimality using the \texttt{raptr R} package [@r465] and $\texttt{Gurobi}$ [@r463].

Statistical and computational analyses {#statistical-analyses .unnumbered}
--------------------------------------------------------

To determine if the environmental and geographic variables correlate with spatial patterns of adaptive and neutral genetic variation, we computed dissimilarity matrices between the planning units, and tested for correlations between matrices based on surrogate and genetic data. For each species, we computed two dissimilarity matrices between the planning units occupied by the species in terms of their environmental conditions (their coordinates in the environmental space) and geographic position (their coordinates in the geographic space). We then computed another dissimilarity matrix between the planning units using their values in the species' neutral genetic space. If the species was associated with adaptive loci, we also computed another dissimilarity matrix between the planning units using the species' adaptive genetic space. All dissimilarity matrices calculated using Euclidean distances.

We then fit maximum likelihood population-effects (MLPE) models [@r494; implemented in the \texttt{ResistanceGA R} package; @r488]. These models explicitly accommodate the structure of dissimilarity matrices using random effects. For each species, we fit a MLPE model to correlate the species' dissimilarity matrices based on geographic position and neutral genetic variation. Additionally, if the species was associated with adaptive loci, we also fit a MLPE model to correlate the species' dissimilarity matrices calculated using environmental conditions and adaptive genetic variation. All predictor variables were z-transformed prior to model fitting to improve convergence. To test if the surrogate variables explained a substantial amount of genetic variation, we conducted $\chi^2$ tests between each model and its corresponding null model, and applied Bonferroni corrections. 

To address the second aim--to determine if the environmental and geographic variables are effective surrogates for adaptive and neutral genetic variation--we generated a collection of single-species prioritisations and computed the proportion of intra-specific genetic variation they secured. For each species, we generated `r raptr.params.LST[[MODE]]$surrogacy.analysis$random.replicates` prioritisations by randomly selecting different combinations of planning units that were occupied by the species. We then computed the proportion of adaptive genetic variation and environmental variation secured in each prioritisation. We then generated another random set of prioritisations, and computed the proportion of neutral genetic variation and geographic variation secured in each prioritisation.

We fit two full generalised linear models (GLM) with logit links. The first model was fit to the proportion of adaptive genetic variation secured in a prioritisation, and contained a continuous predictor variable measuring the proportion of environmental variation also secured in the prioritisation, a categorical variable indicating the species for which the prioritisation was generated, and an interaction term between these predictor variables. The second model was fit to the proportion of neutral genetic variation secured in a prioritisation, and contained a continuous predictor variable assessing the proportion of geographic variation also secured in the prioritisation. Similar to the previous model, this model also contained a predictor variable indicating the species for which the prioritisations were generated, and an interaction term. We subjected these models to a backwards step-wise term deletion routine to assess term significance. To gauge the performance of these surrogates for each species, we fit a suite of models correlating the proportion of adaptive and neutral genetic variation held in each prioritisation and the environmental and geographic variation they also secured (respectively). We then computed the Cragg and Uhler's pseudo-R$^2$ value for each model [using the \texttt{pscl} R package; @r492].

To determine whether the use of surrogate-based targets improves prioritisations, we investigated the effectiveness of surrogate-based targets under three scenarios with increasing levels of complexity. To assess the benefits of surrogate-based targets under the simplest of circumstances, our first scenario involved generating single-species prioritisations for each species. However, most large-scale prioritisations involve a comprehensive set of species. By forcing solutions to secure individuals in multiple communities, this may result in prioritisations that secure a greater proportion of intra-specific genetic variation for each species. To test for this effect, our second scenario involved generating multi-species prioritisations. Finally, most large-scale prioritisations also explicitly consider opportunity cost to ensure that prioritisations are cost-effective. Our third scenario involved generating multi-species prioritisations that satisfied targets for minimal opportunity cost.

For each scenario, we generated three types of prioritisations using (1) only amount-based targets, (2) both amount- and surrogate-based targets, and (3) both amount- and genetic-based targets. The purely amount-based prioritisations served as our baseline--representing the prioritisations typically generated in conservation planning exercises (eg. by \texttt{Marxan}). We used `r (raptr.params.LST[[MODE]]$scenario.analysis$amount.target*100)` % amount-based targets in all prioritisations to ensure an adequate proportion of habitat was secured for each species. Based on the results from the analyses used to address our second aim, we set surrogate-based targets as `r (raptr.params.LST[[MODE]]$scenario.analysis$surrogate.target*100)` % and genetic-based targets as `r (raptr.params.LST[[MODE]]$scenario.analysis$genetic.target*100)` % to ensure that most of the genetic variation was secured for each species. To accommodate the fact that there were a large number of optimal solutions for the single-species amount-based prioritisations, we generated `r (raptr.params.LST[[MODE]]$scenario.analysis$single.species.amount.replicates)` replicate prioritisations. We computed the proportion of adaptive (if present) and neutral genetic variation secured for each species in the prioritisations.

We investigated the benefits of using surrogate-based targets in prioritisations using generalised linear mixed-effects models fit with logit links [GLMMs; using the \texttt{lme4 R} package; @r469]. We fit a full model to the proportion of genetic variation secured in a given prioritisation. This model contained categorical predictor variables to indicate the combination of targets used to generate the prioritisations (amount-only, amount and surrogate targets, and amount and genetic targets), the scenario that was used to generate the prioritisation (single-species, multi-species, and multi-species with cost), the type of genetic variation measured (adaptive and neutral), and all interactions between these variables. This model accommodated prioritisations generated for the same species using a random intercept term. We subject the full model to a backwards step-wise term deletion routine to assess term significance. We conducted a post-hoc analysis on the minimal adequate model using Tukey contrasts with a Bonferroni correction [using the \texttt{multcomp R} package; @r470].

\showmatmethods
\showacknow
\pnasbreak

